ABAQUS
------ 



**Description** 


Finite Element Analysis software for modeling, visualization and best-in-class implicit and explicit dynamics FEA. 


**More Information** 


https://www.simulia.com/products/abaqus_fea.html 


**Available Versions**:: 


    2021
    2020
    6.19-1



ABINIT
------ 



**Description** 


ABINIT is a package whose main program allows one to find the total energy, charge density and electronic structure of systems made of electrons and nuclei (molecules and periodic solids) within Density Functional Theory (DFT), using pseudopotentials and a planewave or wavelet basis. 


**More Information** 


https://www.abinit.org/ 


**Available Versions**:: 


    9.2.1-intel-2020a



ABySS
----- 



**Description** 


Assembly By Short Sequences - a de novo, parallel, paired-end sequence assembler 


**More Information** 


https://www.bcgsc.ca/platform/bioinfo/software/abyss 


**Available Versions**:: 


    2.1.5-foss-2019b



ACTC
---- 



**Description** 


ACTC converts independent triangles into triangle strips or fans. 


**More Information** 


https://sourceforge.net/projects/actc 


**Available Versions**:: 


    1.1-GCCcore-8.3.0
    1.1-GCCcore-10.2.0



ADMIXTURE
--------- 



**Description** 


ADMIXTURE is a software tool for maximum likelihood estimation of individual ancestries from multilocus SNP genotype datasets. It uses the same statistical model as STRUCTURE but calculates estimates much more rapidly using a fast numerical optimization algorithm. 


**More Information** 


https://dalexander.github.io/admixture 


**Available Versions**:: 


    1.3.0



ADOL-C
------ 



**Description** 


The package ADOL-C (Automatic Differentiation by OverLoading in C++) facilitatesthe evaluation of first and higher derivatives of vector functions that are definedby computer programs written in C or C++. The resulting derivative evaluationroutines may be called from C/C++, Fortran, or any other language that can be linkedwith C.  


**More Information** 


https://projects.coin-or.org/ADOL-C 


**Available Versions**:: 


    2.7.2-gompi-2020a



AMOS
---- 



**Description** 


The AMOS consortium is committed to the development of open-source whole genome assembly software 


**More Information** 


http://amos.sourceforge.net 


**Available Versions**:: 


    3.1.0-foss-2018b



ANSYS
----- 



**Available Versions**:: 


    2022R1
    2021R2



ANTLR
----- 



**Description** 


ANTLR, ANother Tool for Language Recognition, (formerly PCCTS) is a language tool that provides a framework for constructing recognizers, compilers, and translators from grammatical descriptions containing Java, C#, C++, or Python actions. 


**More Information** 


https://www.antlr2.org/ 


**Available Versions**:: 


    2.7.7-GCCcore-10.2.0-Java-11
    2.7.7-GCCcore-10.3.0-Java-11



APR
--- 



**Description** 


Apache Portable Runtime (APR) libraries. 


**More Information** 


https://apr.apache.org/ 


**Available Versions**:: 


    1.7.0-GCCcore-10.2.0



APR-util
-------- 



**Description** 


Apache Portable Runtime (APR) util libraries. 


**More Information** 


https://apr.apache.org/ 


**Available Versions**:: 


    1.6.1-GCCcore-10.2.0



ASE
--- 



**Description** 


ASE is a python package providing an open source Atomic Simulation Environment in the Python scripting language.From version 3.20.1 we also include the ase-ext package, it contains optional reimplementationsin C of functions in ASE.  ASE uses it automatically when installed. 


**More Information** 


https://wiki.fysik.dtu.dk/ase 


**Available Versions**:: 


    3.22.0-foss-2021a
    3.20.1-foss-2020b



ATK
--- 



**Description** 


ATK provides the set of accessibility interfaces that are implemented by other toolkits and applications. Using the ATK interfaces, accessibility tools have full access to view and control running applications. 


**More Information** 


https://developer.gnome.org/ATK/stable/ 


**Available Versions**:: 


    2.28.1-foss-2018b
    2.32.0-GCCcore-8.2.0
    2.36.0-GCCcore-10.2.0
    2.34.1-GCCcore-8.3.0
    2.36.0-GCCcore-9.3.0



AUGUSTUS
-------- 



**Description** 


AUGUSTUS is a program that predicts genes in eukaryotic genomic sequences 


**More Information** 


https://bioinf.uni-greifswald.de/augustus/ 


**Available Versions**:: 


    3.4.0-foss-2020b
    3.3.3-foss-2019b



Advisor
------- 



**Description** 


Vectorization Optimization and Thread Prototyping - Vectorize & thread code or performance “dies” - Easy workflow + data + tips = faster code faster - Prioritize, Prototype & Predict performance gain  


**More Information** 


https://software.intel.com/intel-advisor-xe 


**Available Versions**:: 


    2019_update5



AlphaFold
--------- 



**Description** 


AlphaFold can predict protein structures with atomic accuracy even where no similar structure is known 


**More Information** 


https://deepmind.com/research/case-studies/alphafold 


**Available Versions**:: 


    2.0.0-fosscuda-2020b
    2.1.1-fosscuda-2020b



Amber
----- 



**Description** 


Amber (originally Assisted Model Building with Energy Refinement) is software for performing molecular dynamics and structure prediction. 


**More Information** 


http://ambermd.org/amber.html 


**Available Versions**:: 


    18-fosscuda-2018b-AmberTools-18-patchlevel-10-8
    18-foss-2019b-AmberTools-19-patchlevel-12-17-Python-2.7.16
    18-foss-2018b-AmberTools-18-patchlevel-10-8
    18-fosscuda-2019b-AmberTools-19-patchlevel-12-17-Python-2.7.16



AmberTools
---------- 



**Description** 


AmberTools consists of several independently developed packages that work well by themselves, and with Amber itself. The suite can also be used to carry out complete molecular dynamics simulations, with either explicit water or generalized Born solvent models. 


**More Information** 


https://ambermd.org/ 


**Available Versions**:: 


    20-gompi-2019b-dba-Python-3.7.4
    20-intel-2020a-Python-3.8.2



Anaconda2
--------- 



**Description** 


Built to complement the rich, open source Python community,the Anaconda platform provides an enterprise-ready data analytics platform that empowers companies to adopt a modern open data science analytics architecture. 


**More Information** 


https://www.anaconda.com 


**Available Versions**:: 


    4.2.0
    2019.10
    2019.03



Anaconda3
--------- 



**Description** 


Built to complement the rich, open source Python community,the Anaconda platform provides an enterprise-ready data analytics platform that empowers companies to adopt a modern open data science analytics architecture. 


**More Information** 


https://www.anaconda.com 


**Available Versions**:: 


    2021.11
    2022.05
    2020.02
    2020.11
    2021.05



Armadillo
--------- 



**Description** 


Armadillo is an open-source C++ linear algebra library (matrix maths) aiming towards a good balance between speed and ease of use. Integer, floating point and complex numbers are supported, as well as a subset of trigonometric and statistics functions. 


**More Information** 


https://arma.sourceforge.net/ 


**Available Versions**:: 


    9.900.1-foss-2020a



Arriba
------ 



**Description** 


Arriba is a command-line tool for the detection of gene fusions from RNA-Seq data. It was developed for the use in a clinical research setting. Therefore, short runtimes and high sensitivity were important design criteria. 


**More Information** 


https://github.com/suhrig/arriba 


**Available Versions**:: 


    2.3.0-GCC-11.2.0



Arrow
----- 



**Description** 


Apache Arrow (incl. PyArrow Python bindings), a cross-language development platform for in-memory data. 


**More Information** 


https://arrow.apache.org 


**Available Versions**:: 


    6.0.0-foss-2021b
    0.16.0-foss-2019b-Python-3.7.4



Aspera-CLI
---------- 



**Description** 


IBM Aspera Command-Line Interface (the Aspera CLI) isa collection of Aspera tools for performing high-speed, secure datatransfers from the command line. The Aspera CLI is for users andorganizations who want to automate their transfer workflows. 


**More Information** 


https://asperasoft.com 


**Available Versions**:: 


    3.9.6.1467.159c5b1



Aspera-Connect
-------------- 



**Description** 


Connect is an install-on-demand Web browser plug-in that facilitates high-speed uploads and downloads with an Aspera transfer server. 


**More Information** 


http://downloads.asperasoft.com/connect2/ 


**Available Versions**:: 


    3.9.6



AutoDock
-------- 



**Description** 


AutoDock is a suite of automated docking tools. It is designed to  predict how small molecules, such as substrates or drug candidates, bind to  a receptor of known 3D structure. 


**More Information** 


http://autodock.scripps.edu/ 


**Available Versions**:: 


    4.2.6-GCC-9.3.0



AutoDock_Vina
------------- 



**Description** 


AutoDock Vina is an open-source program for doing molecular docking.  


**More Information** 


http://vina.scripps.edu/index.html 


**Available Versions**:: 


    1.1.2_linux_x86



Autoconf
-------- 



**Description** 


Autoconf is an extensible package of M4 macros that produce shell scripts to automatically configure software source code packages. These scripts can adapt the packages to many kinds of UNIX-like systems without manual user intervention. Autoconf creates a configuration script for a package from a template file that lists the operating system features that the package can use, in the form of M4 macro calls. 


**More Information** 


https://www.gnu.org/software/autoconf/ 


**Available Versions**:: 


    2.69-GCCcore-7.3.0
    2.69-GCCcore-8.1.0
    2.71-GCCcore-11.3.0
    2.71-GCCcore-11.2.0
    2.69-GCCcore-10.2.0
    2.69
    2.69-GCCcore-8.2.0
    2.71-GCCcore-10.3.0
    2.69-GCCcore-8.3.0
    2.69-GCCcore-9.3.0



Automake
-------- 



**Description** 


Automake: GNU Standards-compliant Makefile generator 


**More Information** 


https://www.gnu.org/software/automake/automake.html 


**Available Versions**:: 


    1.16.3-GCCcore-10.3.0
    1.16.2-GCCcore-10.2.0
    1.16.1-GCCcore-7.3.0
    1.16.1-GCCcore-10.2.0
    1.16.1-GCCcore-8.1.0
    1.16.5-GCCcore-11.3.0
    1.16.4-GCCcore-11.2.0
    1.16.1-GCCcore-9.3.0
    1.16.1-GCCcore-8.3.0
    1.16.1-GCCcore-8.2.0



Autotools
--------- 



**Description** 


This bundle collect the standard GNU build tools: Autoconf, Automake and libtool 


**More Information** 


https://autotools.io 


**Available Versions**:: 


    20210726-GCCcore-11.2.0
    20200321-GCCcore-10.2.0
    20210128-GCCcore-10.3.0
    20180311-GCCcore-8.2.0
    20220317-GCCcore-11.3.0
    20180311-GCCcore-8.3.0
    20180311-GCCcore-8.1.0
    20180311-GCCcore-10.2.0
    20180311-GCCcore-9.3.0
    20180311-GCCcore-7.3.0



BAGEL
----- 



**Description** 


BAGEL (Brilliantly Advanced General Electronic-structure Library)is a parallel electronic-structure program. 


**More Information** 


http://www.nubakery.org 


**Available Versions**:: 


    1.2.2-foss-2019a



BCFtools
-------- 



**Description** 


Samtools is a suite of programs for interacting with high-throughput sequencing data. BCFtools - Reading/writing BCF2/VCF/gVCF files and calling/filtering/summarising SNP and short indel sequence variants 


**More Information** 


http://www.htslib.org/ 


**Available Versions**:: 


    1.9-foss-2018b
    1.11-GCC-10.2.0
    1.10.2-GCC-9.3.0
    1.10.2-GCC-8.3.0



BEDTools
-------- 



**Description** 


BEDTools: a powerful toolset for genome arithmetic.The BEDTools utilities allow one to address common genomics tasks such as finding feature overlaps andcomputing coverage.The utilities are largely based on four widely-used file formats: BED, GFF/GTF, VCF, and SAM/BAM. 


**More Information** 


https://bedtools.readthedocs.io/ 


**Available Versions**:: 


    2.29.2-GCC-9.3.0
    2.29.2-GCC-8.3.0



BLAST
----- 



**Description** 


Basic Local Alignment Search Tool, or BLAST, is an algorithm for comparing primary biological sequence information, such as the amino-acid sequences of different proteins or the nucleotides of DNA sequences. 


**More Information** 


https://blast.ncbi.nlm.nih.gov/ 


**Available Versions**:: 


    2.10.1-Linux_x86_64



BLAST+
------ 



**Description** 


Basic Local Alignment Search Tool, or BLAST, is an algorithm for comparing primary biological sequence information, such as the amino-acid sequences of different proteins or the nucleotides of DNA sequences. 


**More Information** 


https://blast.ncbi.nlm.nih.gov/ 


**Available Versions**:: 


    2.9.0-gompi-2019b
    2.10.1-iimpi-2020a
    2.7.1-foss-2018b
    2.9.0-iimpi-2019b
    2.11.0-gompi-2021a
    2.11.0-gompi-2020b
    2.9.0-gompi-2021a



BLAT
---- 



**Description** 


BLAT on DNA is designed to quickly find sequences of 95% andgreater similarity of length 25 bases or more. 


**More Information** 


https://genome.ucsc.edu/FAQ/FAQblat.html 


**Available Versions**:: 


    3.5-GCC-9.3.0



BLIS
---- 



**Description** 


BLIS is a portable software framework for instantiating high-performanceBLAS-like dense linear algebra libraries. 


**More Information** 


https://github.com/flame/blis/ 


**Available Versions**:: 


    0.8.1-GCC-11.2.0
    0.9.0-GCC-11.3.0



BWA
--- 



**Description** 


Burrows-Wheeler Aligner (BWA) is an efficient program that aligns relatively short nucleotide sequences against a long reference sequence such as the human genome. 


**More Information** 


http://bio-bwa.sourceforge.net/ 


**Available Versions**:: 


    0.7.17-foss-2018b
    0.7.17-GCCcore-11.2.0
    0.7.17-GCC-9.3.0



BamTools
-------- 



**Description** 


BamTools provides both a programmer's API and an end-user's toolkit for handling BAM files. 


**More Information** 


https://github.com/pezmaster31/bamtools 


**Available Versions**:: 


    2.5.1-GCC-8.3.0
    2.5.1-GCC-9.3.0
    2.5.1-GCC-10.2.0



BayeScEnv
--------- 



**Description** 


BayeScEnv is a Fst-based, genome-scan method that uses environmental variables to detectlocal adaptation. 


**More Information** 


https://github.com/devillemereuil/bayescenv 


**Available Versions**:: 


    1.1-GCC-8.3.0



BayesTraits
----------- 



**Description** 


BayesTraits is a computer package for performing analyses of trait  evolution among groups of species for which a phylogeny or sample of phylogenies is  available. This new package incoporates our earlier and separate programes Multistate,  Discrete and Continuous. BayesTraits can be applied to the analysis of traits that adopt  a finite number of discrete states, or to the analysis of continuously varying traits.  Hypotheses can be tested about models of evolution, about ancestral states and about  correlations among pairs of traits.  


**More Information** 


http://www.evolution.reading.ac.uk/BayesTraitsV1.html 


**Available Versions**:: 


    2.0-Beta-Linux64



Bazel
----- 



**Description** 


Bazel is a build tool that builds code quickly and reliably.It is used to build the majority of Google's software. 


**More Information** 


https://bazel.io/ 


**Available Versions**:: 


    3.7.2-GCCcore-10.2.0
    3.6.0-GCCcore-9.3.0
    3.7.2-GCCcore-10.3.0
    0.29.1-GCCcore-8.3.0
    0.26.1-GCCcore-8.3.0
    3.7.2-GCCcore-11.2.0



Beast
----- 



**Description** 


BEAST is a cross-platform program for Bayesian MCMC analysis of molecular  sequences. It is entirely orientated towards rooted, time-measured phylogenies inferred using  strict or relaxed molecular clock models. It can be used as a method of reconstructing phylogenies  but is also a framework for testing evolutionary hypotheses without conditioning on a single  tree topology. BEAST uses MCMC to average over tree space, so that each tree is weighted  proportional to its posterior probability.  


**More Information** 


http://beast2.org/ 


**Available Versions**:: 


    2.5.2-GCC-7.3.0-2.30
    2.5.2-GCC-8.2.0-2.31.1



Biopython
--------- 



**Description** 


Biopython is a set of freely available tools for biological computation written in Python by an international team of developers. It is a distributed collaborative effort to develop Python libraries and applications which address the needs of current and future work in bioinformatics.  


**More Information** 


https://www.biopython.org 


**Available Versions**:: 


    1.75-foss-2019b-Python-3.7.4
    1.78-fosscuda-2020b
    1.72-foss-2018b-Python-2.7.15



Bison
----- 



**Description** 


Bison is a general-purpose parser generator that converts an annotated context-free grammar into a deterministic LR or generalized LR (GLR) parser employing LALR(1) parser tables. 


**More Information** 


http://www.gnu.org/software/bison 


**Available Versions**:: 


    3.0.4-GCCcore-8.1.0
    3.5.3
    3.0.4
    3.7.1
    3.0.4-GCCcore-7.3.0
    3.0.5-GCCcore-8.3.0
    3.3.2
    3.5.3-GCCcore-9.3.0
    3.8.2
    3.7.6-GCCcore-10.3.0
    3.3.2-GCCcore-9.3.0
    3.0.5
    3.7.6
    3.8.2-GCCcore-11.3.0
    3.0.5-GCCcore-8.1.0
    3.7.6-GCCcore-11.2.0
    3.3.2-GCCcore-8.3.0
    3.0.5-GCCcore-7.3.0
    3.0.5-GCCcore-8.2.0
    3.7.1-GCCcore-10.2.0



Blender
------- 



**Description** 


Blender is the free and open source 3D creation suite. It supports the entirety of the 3D pipeline-modeling, rigging, animation, simulation, rendering, compositing and motion tracking, even video editing and game creation. 


**More Information** 


https://www.blender.org/ 


**Available Versions**:: 


    2.81-foss-2019b-Python-3.7.4



Boost
----- 



**Description** 


Boost provides free peer-reviewed portable C++ source libraries. 


**More Information** 


http://www.boost.org/ 


**Available Versions**:: 


    1.67.0-fosscuda-2018b
    1.71.0-gompic-2019b
    1.74.0-GCC-10.2.0
    1.70.0-gompi-2019a
    1.76.0-GCC-10.3.0
    1.72.0-gompi-2020a
    1.71.0-iimpi-2019b
    1.72.0-iimpi-2020a
    1.71.0-gompi-2019b
    1.77.0-GCC-11.2.0
    1.67.0-foss-2018b
    1.74.0-iccifort-2020.4.304



Boost.Python
------------ 



**Description** 


Boost.Python is a C++ library which enables seamless interoperability between C++ and the Python programming language. 


**More Information** 


http://boostorg.github.io/python 


**Available Versions**:: 


    1.70.0-gompi-2019a
    1.67.0-fosscuda-2018b-Python-2.7.15
    1.71.0-gompic-2019b
    1.71.0-gompi-2019b
    1.72.0-gompi-2020a
    1.67.0-foss-2018b-Python-2.7.15



Bowtie
------ 



**Description** 


Bowtie is an ultrafast, memory-efficient short read aligner. It aligns short DNA sequences (reads) to the human genome. 


**More Information** 


http://bowtie-bio.sourceforge.net/index.shtml 


**Available Versions**:: 


    1.3.1-GCC-11.2.0
    1.2.3-GCC-9.3.0



Bowtie2
------- 



**Description** 


Bowtie 2 is an ultrafast and memory-efficient tool for aligning sequencing reads to long reference sequences. It is particularly good at aligning reads of about 50 up to 100s or 1,000s of characters, and particularly good at aligning to relatively long (e.g. mammalian) genomes. Bowtie 2 indexes the genome with an FM Index to keep its memory footprint small: for the human genome, its memory footprint is typically around 3.2 GB. Bowtie 2 supports gapped, local, and paired-end alignment modes. 


**More Information** 


http://bowtie-bio.sourceforge.net/bowtie2/index.shtml 


**Available Versions**:: 


    2.4.5-GCC-11.2.0
    2.3.4.2-foss-2018b
    2.4.1-GCC-9.3.0



Brotli
------ 



**Description** 


Brotli is a generic-purpose lossless compression algorithm that compresses data using a combination of a modern variant of the LZ77 algorithm, Huffman coding and 2nd order context modeling, with a compression ratio comparable to the best currently available general-purpose compression methods. It is similar in speed with deflate but offers more dense compression.The specification of the Brotli Compressed Data Format is defined in RFC 7932. 


**More Information** 


https://github.com/google/brotli 


**Available Versions**:: 


    1.0.9-GCCcore-11.2.0



CASTEP
------ 



**Description** 


CASTEP is a leading code for calculating the properties of materials from first principles. Using density functional theory, it can simulate a wide range of properties of materialsproprieties including energetics, structure at the atomic level, vibrational properties, electronic response properties etc. In particular it has a wide range of spectroscopic features that link directly to experiment, such as infra-red and Raman spectroscopies, NMR, and core level spectra. 


**More Information** 


http://www.castep.org 


**Available Versions**:: 


    21.11-info
    16.11-intel-2020a
    16.11-info



CD-HIT
------ 



**Description** 


CD-HIT is a very widely used program for clustering and  comparing protein or nucleotide sequences. 


**More Information** 


http://weizhongli-lab.org/cd-hit/ 


**Available Versions**:: 


    4.8.1-GCC-10.3.0
    4.8.1-GCC-10.2.0



CDO
--- 



**Description** 


CDO is a collection of command line Operators to manipulate and analyse Climate and NWP model Data. 


**More Information** 


https://code.zmaw.de/projects/cdo 


**Available Versions**:: 


    1.9.10-gompi-2020b



CFITSIO
------- 



**Description** 


CFITSIO is a library of C and Fortran subroutines for reading and writing data files inFITS (Flexible Image Transport System) data format. 


**More Information** 


https://heasarc.gsfc.nasa.gov/fitsio/ 


**Available Versions**:: 


    3.47-GCCcore-8.3.0
    3.49-GCCcore-11.2.0



CGAL
---- 



**Description** 


The goal of the CGAL Open Source Project is to provide easy access to efficient and reliable geometric algorithms in the form of a C++ library. 


**More Information** 


https://www.cgal.org/ 


**Available Versions**:: 


    4.14.3-gompi-2021a
    4.14.1-foss-2019b-Python-3.7.4
    5.2-gompi-2020b
    4.14.3-gompi-2020a-Python-3.8.2
    4.14.3-iimpi-2020a-Python-3.8.2



CGNS
---- 



**Description** 


The CGNS system is designed to facilitate the exchange of data between sites and applications, and to help stabilize the archiving of aerodynamic data. 


**More Information** 


https://cgns.github.io/ 


**Available Versions**:: 


    4.1.0-intelcuda-2019b



CIF2Cell
-------- 



**Description** 


CIF2Cell is a tool to generate the geometrical setupfor various electronic structure codes from a CIF (CrystallographicInformation Framework) file. The program currently supports output for anumber of popular electronic structure programs, including ABINIT, ASE,CASTEP, CP2K, CPMD, CRYSTAL09, Elk, EMTO, Exciting, Fleur, FHI-aims,Hutsepot, MOPAC, Quantum Espresso, RSPt, Siesta, SPR-KKR, VASP. Alsoexports some related formats like .coo, .cfg and .xyz-files. 


**More Information** 


https://sourceforge.net/projects/cif2cell 


**Available Versions**:: 


    2.0.0a3-GCCcore-9.3.0-Python-3.8.2



CLHEP
----- 



**Description** 


The CLHEP project is intended to be a set of HEP-specific foundation and utility classes such as random generators, physics vectors, geometry and linear algebra. CLHEP is structured in a set of packages independent of any external package. 


**More Information** 


https://proj-clhep.web.cern.ch/proj-clhep/ 


**Available Versions**:: 


    2.4.5.1-GCC-11.2.0
    2.4.4.0-GCC-10.2.0



CMake
----- 



**Description** 


CMake, the cross-platform, open-source build system.  CMake is a family of tools designed to build, test and package software. 


**More Information** 


https://www.cmake.org 


**Available Versions**:: 


    3.11.4-GCCcore-7.3.0
    3.23.1-GCCcore-11.3.0
    3.13.3-GCCcore-8.2.0
    3.12.1-GCCcore-10.2.0
    3.22.1-GCCcore-11.2.0
    3.15.3-GCCcore-7.3.0
    3.9.4-GCCcore-9.3.0
    3.20.1-GCCcore-10.3.0
    3.12.1
    3.12.1-GCCcore-7.3.0
    3.21.1-GCCcore-11.2.0
    3.18.4-GCCcore-10.2.0
    3.16.4-GCCcore-9.3.0
    3.15.3-GCCcore-8.3.0



CP2K
---- 



**Description** 


CP2K is a freely available (GPL) program, written in Fortran 95, to perform atomistic and molecular simulations of solid state, liquid, molecular and biological systems. It provides a general framework for different methods such as e.g. density functional theory (DFT) using a mixed Gaussian and plane waves approach (GPW), and classical pair and many-body potentials.  


**More Information** 


https://www.cp2k.org/ 


**Available Versions**:: 


    7.1-intel-2020a
    7.1-intel-2020b
    6.1-foss-2020a
    6.1-intel-2020a
    6.1-foss-2019b
    8.1-foss-2020a
    6.1-foss-2019a



CPLEX
----- 



**Description** 


IBM ILOG CPLEX Optimizer's mathematical programming technology enables analytical decision support for improving efficiency, reducing costs, and increasing profitability. 


**More Information** 


https://www.ibm.com/analytics/cplex-optimizer 


**Available Versions**:: 


    20.1.0-GCCcore-8.3.0
    12.9.0



CREST
----- 



**Description** 


CREST is an utility/driver program for the xtb program. Originally it was designed as conformer sampling program, hence the abbreviation Conformer–Rotamer Ensemble Sampling Tool, but now offers also some utility functions for calculations with the GFNn–xTB methods. Generally the program functions as an IO based OMP scheduler (i.e., calculations are performed by the xtb program) and tool for the creation and analysation of structure ensembles. 


**More Information** 


https://xtb-docs.readthedocs.io/en/latest/crest.html 


**Available Versions**:: 


    2.11-intel-2021a



CUDA
---- 



**Description** 


CUDA (formerly Compute Unified Device Architecture) is a parallel computing platform and programming model created by NVIDIA and implemented by the graphics processing units (GPUs) that they produce. CUDA gives developers access to the virtual instruction set and memory of the parallel computational elements in CUDA GPUs. 


**More Information** 


https://developer.nvidia.com/cuda-toolkit 


**Available Versions**:: 


    11.6.0
    9.2.88-GCC-7.3.0-2.30
    10.1.243-iccifort-2019.5.281
    11.4.1-GCC-10.3.0
    11.2.2-GCC-10.3.0
    11.1.1-GCC-10.2.0
    10.1.243-GCC-8.3.0
    11.2.2
    11.3.1
    11.0.2-GCC-9.3.0
    11.3.1-GCC-10.3.0
    11.4.1
    11.5.0
    11.1.1-iccifort-2020.4.304
    11.5.1



CUDAcore
-------- 



**Description** 


CUDA (formerly Compute Unified Device Architecture) is a parallel computing platform and programming model created by NVIDIA and implemented by the graphics processing units (GPUs) that they produce. CUDA gives developers access to the virtual instruction set and memory of the parallel computational elements in CUDA GPUs. 


**More Information** 


https://developer.nvidia.com/cuda-toolkit 


**Available Versions**:: 


    11.2.2
    11.0.2
    11.1.1



CVXOPT
------ 



**Description** 


CVXOPT is a free software package for convex optimization based on the Python programming language. Its main purpose is to make the development of software for convex optimization applications straightforward by building on Python's extensive standard library and on the strengths of Python as a high-level programming language. 


**More Information** 


http://cvxopt.org 


**Available Versions**:: 


    1.2.4-foss-2020a
    1.2.6-foss-2021a
    1.2.3-foss-2019a
    1.2.4-foss-2020a-Python-3.8.2



CapnProto
--------- 



**Description** 


Cap’n Proto is an insanely fast data interchange format and capability-based RPC system. 


**More Information** 


https://capnproto.org 


**Available Versions**:: 


    0.7.0-GCCcore-7.3.0



CellRanger
---------- 



**Description** 


Cell Ranger is a set of analysis pipelines that process Chromium# single-cell RNA-seq output to align reads, generate gene-cell matrices and perform# clustering and gene expression analysis. 


**More Information** 


https://support.10xgenomics.com/single-cell-gene-expression/software/pipelines/latest/what-is-cell-ranger 


**Available Versions**:: 


    5.0.0
    6.0.2



CheMPS2
------- 



**Description** 


CheMPS2 is a scientific library which contains a spin-adapted implementation of thedensity matrix renormalization group (DMRG) for ab initio quantum chemistry. 


**More Information** 


https://github.com/SebWouters/CheMPS2 


**Available Versions**:: 


    1.8.9-foss-2019a
    1.8.11-foss-2021b
    1.8.11-intel-2021a



Check
----- 



**Description** 


Check is a unit testing framework for C. It features a simple interface fordefining unit tests, putting little in the way of the developer. Tests arerun in a separate address space, so both assertion failures and code errorsthat cause segmentation faults or other signals can be caught. Test resultsare reportable in the following: Subunit, TAP, XML, and a generic loggingformat. 


**More Information** 


https://libcheck.github.io/check/ 


**Available Versions**:: 


    0.15.2-GCCcore-10.3.0
    0.15.2-GCCcore-9.3.0
    0.15.2-GCCcore-10.2.0



Clang
----- 



**Description** 


C, C++, Objective-C compiler, based on LLVM.  Does not include C++ standard library -- use libstdc++ from GCC. 


**More Information** 


https://clang.llvm.org/ 


**Available Versions**:: 


    11.0.1-GCCcore-10.2.0
    11.0.1-gcccuda-2020b



CppUnit
------- 



**Description** 


CppUnit is the C++ port of the famous JUnit framework for unit testing. 


**More Information** 


https://freedesktop.org/wiki/Software/cppunit/ 


**Available Versions**:: 


    1.15.1-GCCcore-10.3.0



CubeGUI
------- 



**Description** 


Cube, which is used as performance report explorer for Scalasca and Score-P, is a generic tool for displaying a multi-dimensional performance space consisting of the dimensions (i) performance metric, (ii) call path, and (iii) system resource. Each dimension can be represented as a tree, where non-leaf nodes of the tree can be collapsed or expanded to achieve the desired level of granularity. This module provides the Cube graphical report explorer. 


**More Information** 


https://www.scalasca.org/software/cube-4.x/download.html 


**Available Versions**:: 


    4.4.4-GCCcore-9.3.0



CubeLib
------- 



**Description** 


Cube, which is used as performance report explorer for Scalasca and Score-P, is a generic tool for displaying a multi-dimensional performance space consisting of the dimensions (i) performance metric, (ii) call path, and (iii) system resource. Each dimension can be represented as a tree, where non-leaf nodes of the tree can be collapsed or expanded to achieve the desired level of granularity. This module provides the Cube general purpose C++ library component and command-line tools. 


**More Information** 


https://www.scalasca.org/software/cube-4.x/download.html 


**Available Versions**:: 


    4.4.4-GCCcore-9.3.0



CubeWriter
---------- 



**Description** 


Cube, which is used as performance report explorer for Scalasca and Score-P, is a generic tool for displaying a multi-dimensional performance space consisting of the dimensions (i) performance metric, (ii) call path, and (iii) system resource. Each dimension can be represented as a tree, where non-leaf nodes of the tree can be collapsed or expanded to achieve the desired level of granularity. This module provides the Cube high-performance C writer library component. 


**More Information** 


https://www.scalasca.org/software/cube-4.x/download.html 


**Available Versions**:: 


    4.4.3-GCCcore-9.3.0



Cufflinks
--------- 



**Description** 


Transcript assembly, differential expression, and differential regulation for RNA-Seq 


**More Information** 


http://cole-trapnell-lab.github.io/cufflinks/ 


**Available Versions**:: 


    2.2.1-foss-2020a



Cython
------ 



**Description** 


Cython is an optimising static compiler for both the Python programminglanguage and the extended Cython programming language (based on Pyrex). 


**More Information** 


https://cython.org/ 


**Available Versions**:: 


    0.29.22-GCCcore-10.2.0



DB
-- 



**Description** 


Berkeley DB enables the development of custom data management solutions, without the overhead traditionally associated with such custom projects. 


**More Information** 


https://www.oracle.com/technetwork/products/berkeleydb 


**Available Versions**:: 


    18.1.40-GCCcore-10.3.0
    18.1.32-GCCcore-7.3.0
    18.1.32-GCCcore-8.3.0
    18.1.32-GCCcore-8.2.0
    18.1.40-GCCcore-11.2.0
    18.1.32-GCCcore-9.3.0
    18.1.40-GCCcore-11.3.0
    18.1.40-GCCcore-10.2.0



DBus
---- 



**Description** 


D-Bus is a message bus system, a simple way for applications to talk to one another.  In addition to interprocess communication, D-Bus helps coordinate process lifecycle; it makes it simple and reliable to code a "single instance" application or daemon, and to launch applications and daemons on demand when their services are needed. 


**More Information** 


https://dbus.freedesktop.org/ 


**Available Versions**:: 


    1.13.18-GCCcore-10.3.0
    1.13.8-GCCcore-8.2.0
    1.13.12-GCCcore-8.3.0
    1.13.6-GCCcore-7.3.0
    1.13.12-GCCcore-9.3.0
    1.13.18-GCCcore-10.2.0



DFT-D3
------ 



**Description** 


DFT-D3 implements a dispersion correction for density functionals, Hartree-Fock and semi-empirical quantum chemical methods. 


**More Information** 


http://www.thch.uni-bonn.de/tc/index.php?section=downloads&subsection=DFT-D3&lang=english 


**Available Versions**:: 


    3.2.0-intel-2019a
    3.2.0-intel-compilers-2021.2.0



DIAMOND
------- 



**Description** 


DIAMOND is a sequence aligner for protein and translated DNA searches, designed for high performance analysis of big sequence data. 


**More Information** 


https://github.com/bbuchfink/diamond 


**Available Versions**:: 


    0.9.30-iccifort-2019.5.281
    0.9.30-GCC-8.3.0



DL_POLY_4
--------- 



**Description** 


DL_POLY is a general purpose classical molecular dynamics (MD) simulation software 


**More Information** 


https://www.scd.stfc.ac.uk/Pages/DL_POLY.aspx 


**Available Versions**:: 


    5.0.0-intel-2020b



DL_POLY_Classic
--------------- 



**Description** 


DL_POLY Classic is a general purpose (parallel and serial)molecular dynamics simulation package. 


**More Information** 


https://gitlab.com/DL_POLY_Classic/dl_poly 


**Available Versions**:: 


    1.10-foss-2019b



DOLFIN
------ 



**Description** 


DOLFIN is the C++/Python interface of FEniCS, providing a consistent PSE  (Problem Solving Environment) for ordinary and partial differential equations. 


**More Information** 


https://bitbucket.org/fenics-project/dolfin 


**Available Versions**:: 


    2019.1.0.post0-foss-2019b-Python-3.7.4



Doxygen
------- 



**Description** 


Doxygen is a documentation system for C++, C, Java, Objective-C, Python, IDL (Corba and Microsoft flavors), Fortran, VHDL, PHP, C#, and to some extent D. 


**More Information** 


https://www.doxygen.org 


**Available Versions**:: 


    1.8.14-GCCcore-7.3.0
    1.8.15-GCCcore-8.2.0
    1.8.17-GCCcore-9.3.0
    1.9.1-GCCcore-11.2.0
    1.8.16-GCCcore-8.3.0
    1.8.20-GCCcore-10.2.0
    1.9.1-GCCcore-10.3.0



EIGENSOFT
--------- 



**Description** 


The EIGENSOFT package combines functionality from our population genetics methods (Patterson et al. 2006)  and our EIGENSTRAT stratification correction method (Price et al. 2006). The EIGENSTRAT method uses principal components  analysis to explicitly model ancestry differences between cases and controls along continuous axes of variation;  the resulting correction is specific to a candidate marker’s variation in frequency across ancestral populations,  minimizing spurious associations while maximizing power to detect true associations. The EIGENSOFT package has a built-in plotting script and supports multiple file formats and quantitative phenotypes. 


**More Information** 


https://www.hsph.harvard.edu/alkes-price/software/ 


**Available Versions**:: 


    7.2.1-foss-2019b



ELPA
---- 



**Description** 


Eigenvalue SoLvers for Petaflop-Applications . 


**More Information** 


https://elpa.rzg.mpg.de 


**Available Versions**:: 


    2018.11.001-intel-2019a
    2020.11.001-foss-2020b
    2019.11.001-intel-2020a
    2020.11.001-intel-2020b
    2019.11.001-foss-2020a
    2021.05.001-intel-2021a



ESMF
---- 



**Description** 


The Earth System Modeling Framework (ESMF) is a suite of software tools for developing high-performance, multi-component Earth science modeling applications. 


**More Information** 


https://www.earthsystemcog.org/projects/esmf/ 


**Available Versions**:: 


    8.1.1-foss-2021a
    8.0.1-foss-2020b



EasyBuild
--------- 



**Description** 


EasyBuild is a software build and installation framework written in Python that allows you to install software in a structured, repeatable and robust way. 


**More Information** 


https://easybuilders.github.io/easybuild 


**Available Versions**:: 


    4.4.0
    4.5.5
    4.4.1
    4.3.4
    4.5.3
    4.5.2
    4.4.2
    4.6.0
    4.5.4
    4.3.2
    4.3.3
    4.5.0
    4.5.1



Eigen
----- 



**Description** 


Eigen is a C++ template library for linear algebra: matrices, vectors, numerical solvers, and related algorithms. 


**More Information** 


https://eigen.tuxfamily.org 


**Available Versions**:: 


    3.3.7-GCCcore-9.3.0
    3.3.7
    3.3.9-GCCcore-10.3.0
    3.3.9-GCCcore-11.2.0
    3.3.4
    3.3.8-GCCcore-10.2.0



Elk
--- 



**Description** 


An all-electron full-potential linearisedaugmented-plane wave (FP-LAPW) code with many advanced features. Writtenoriginally at Karl-Franzens-Universität Graz as a milestone of theEXCITING EU Research and Training Network, the code is designed to be assimple as possible so that new developments in the field of densityfunctional theory (DFT) can be added quickly and reliably. 


**More Information** 


http://elk.sourceforge.net/ 


**Available Versions**:: 


    7.0.12-foss-2020b



Emacs
----- 



**Description** 


GNU Emacs is an extensible, customizable text editor--and more. At its core is an interpreter for Emacs Lisp, a dialect of the Lisp programming language with extensions to support text editing. 


**More Information** 


https://www.gnu.org/software/emacs/ 


**Available Versions**:: 


    27.1-GCCcore-10.2.0



Extrae
------ 



**Description** 


Extrae is the core instrumentation package developed bythe Performance Tools group at BSC. Extrae is capable of instrumentingapplications based on MPI, OpenMP, pthreads, CUDA1, OpenCL1, and StarSs1using different instrumentation approaches. The information gathered byExtrae typically includes timestamped events of runtime calls,performance counters and source code references. Besides, Extraeprovides its own API to allow the user to manually instrument his or herapplication. 


**More Information** 


https://www.bsc.es/computer-sciences/performance-tools 


**Available Versions**:: 


    3.8.0-gompi-2020b



FCM
--- 



**Description** 


FCM is a set of tools for managing and building source code. 


**More Information** 


http://www.metoffice.gov.uk/research/collaboration/fcm 


**Available Versions**:: 


    2019.09.0
    2.3.1



FEniCS
------ 



**Description** 


FEniCS is a computing platform for solving partial differential equations (PDEs). 


**More Information** 


https://fenicsproject.org/ 


**Available Versions**:: 


    2019.1.0-foss-2019b-Python-3.7.4



FFC
--- 



**Description** 


The FEniCS Form Compiler (FFC) is a compiler for finite element variational forms. 


**More Information** 


https://bitbucket.org/fenics-project/ffc 


**Available Versions**:: 


    2019.1.0.post0-foss-2019b-Python-3.7.4



FFTW
---- 



**Description** 


FFTW is a C subroutine library for computing the discrete Fourier transform (DFT)in one or more dimensions, of arbitrary input size, and of both real and complex data. 


**More Information** 


https://www.fftw.org 


**Available Versions**:: 


    3.3.10-GCC-11.3.0
    3.3.9-gompi-2021a
    3.3.10-gompi-2021b
    3.3.8-gompi-2020a
    3.3.8-gompic-2020b
    3.3.9-intel-2021a
    3.3.8-gompi-2019b
    3.3.8-gompi-2020b
    3.3.8-gompic-2019b
    3.3.8-gompi-2018b
    3.3.8-intel-2020b
    3.3.8-intel-2019a
    3.3.8-gompi-2019a
    3.3.8-gompic-2018b
    3.3.8-intel-2020a
    3.3.8-gompic-2020a



FFTW.MPI
-------- 



**Description** 


FFTW is a C subroutine library for computing the discrete Fourier transform (DFT)in one or more dimensions, of arbitrary input size, and of both real and complex data. 


**More Information** 


https://www.fftw.org 


**Available Versions**:: 


    3.3.10-gompi-2022a



FFmpeg
------ 



**Description** 


A complete, cross-platform solution to record, convert and stream audio and video. 


**More Information** 


https://www.ffmpeg.org/ 


**Available Versions**:: 


    4.3.2-GCCcore-10.3.0
    4.2.1-GCCcore-8.3.0
    4.1.3-GCCcore-8.2.0
    4.2.2-GCCcore-9.3.0
    4.3.1-GCCcore-10.2.0



FIAT
---- 



**Description** 


The FInite element Automatic Tabulator (FIAT) supportsgeneration of arbitrary order instances of the Lagrange elements onlines, triangles, and tetrahedra. It is also capable of generatingarbitrary order instances of Jacobi-type quadrature rules on the sameelement shapes. 


**More Information** 


https://bitbucket.org/fenics-project/fiat 


**Available Versions**:: 


    2019.1.0-foss-2019b-Python-3.7.4



FLAC
---- 



**Description** 


FLAC stands for Free Lossless Audio Codec, an audio format similar to MP3, but lossless, meaningthat audio is compressed in FLAC without any loss in quality. 


**More Information** 


https://xiph.org/flac/ 


**Available Versions**:: 


    1.3.3-GCCcore-11.2.0
    1.3.3-GCCcore-10.2.0
    1.3.3-GCCcore-10.3.0



FLAIR
----- 



**Description** 


FLAIR (Full-Length Alternative Isoform analysis of RNA) for the correction, isoform definition, and alternative splicing analysis of noisy reads. FLAIR has primarily been used for nanopore cDNA, native RNA, and PacBio sequencing reads. 


**More Information** 


https://github.com/BrooksLabUCSC/flair 


**Available Versions**:: 


    1.5.1-20200630-foss-2019b-Python-3.7.4



FLASH
----- 



**Description** 


FLASH (Fast Length Adjustment of SHort reads) is a very fast and accurate software  tool to merge paired-end reads from next-generation sequencing experiments. FLASH is designed to  merge pairs of reads when the original DNA fragments are shorter than twice the length of reads.  The resulting longer reads can significantly improve genome assemblies. They can also improve  transcriptome assembly when FLASH is used to merge RNA-seq data. 


**More Information** 


https://ccb.jhu.edu/software/FLASH/ 


**Available Versions**:: 


    1.2.11-foss-2018b
    2.2.00-foss-2018b



FLINT
----- 



**Description** 


FLINT (Fast Library for Number Theory) is a C library in support of computations in number theory. Operations that can be performed include conversions, arithmetic, computing GCDs, factoring, solving linear systems, and evaluating special functions. In addition, FLINT provides various low-level routines for fast arithmetic. FLINT is extensively documented and tested. 


**More Information** 


https://www.flintlib.org/ 


**Available Versions**:: 


    2.7.1-GCC-10.3.0



FLTK
---- 



**Description** 


FLTK is a cross-platform C++ GUI toolkit for UNIX/Linux (X11), Microsoft Windows, and MacOS X. FLTK provides modern GUI functionality without the bloat and supports 3D graphics via OpenGL and its built-in GLUT emulation. 


**More Information** 


https://www.fltk.org 


**Available Versions**:: 


    1.3.5-GCC-8.3.0
    1.3.5-GCCcore-10.2.0



FSL
--- 



**Description** 


FSL is a comprehensive library of analysis tools for FMRI, MRI and DTI brain imaging data. 


**More Information** 


https://www.fmrib.ox.ac.uk/fsl/ 


**Available Versions**:: 


    6.0.3-foss-2019b-Python-3.7.4



FastME
------ 



**Description** 


FastME: a comprehensive, accurate and fast distance-based phylogeny inference program. 


**More Information** 


http://www.atgc-montpellier.fr/fastme/ 


**Available Versions**:: 


    2.1.6.2-GCC-8.3.0



FastQC
------ 



**Description** 


FastQC is a quality control application for high throughputsequence data. It reads in sequence data in a variety of formats and can eitherprovide an interactive application to review the results of several differentQC checks, or create an HTML based report which can be integrated into apipeline. 


**More Information** 


http://www.bioinformatics.babraham.ac.uk/projects/fastqc/ 


**Available Versions**:: 


    0.11.8-Java-1.8
    0.11.9-Java-11



FastTree
-------- 



**Description** 


FastTree infers approximately-maximum-likelihood phylogenetic trees from alignments of nucleotide or protein sequences. FastTree can handle alignments with up to a million of sequences in a reasonable amount of time and memory.  


**More Information** 


http://www.microbesonline.org/fasttree/ 


**Available Versions**:: 


    2.1.11-GCCcore-9.3.0



Fiji
---- 



**Description** 


Fiji is an image processing package—a 'batteries-included' distribution of ImageJ, bundling a lot of plugins which facilitate scientific image analysis.This release is based on ImageJ-2.1.0 and Fiji-2.1.1 


**More Information** 


https://fiji.sc/ 


**Available Versions**:: 


    20201104-1356



Flask
----- 



**Description** 


" Flask is a lightweight WSGI web application framework. It is designed to make getting started quick and easy, with the ability to scale up to complex applications.  


**More Information** 


https://www.palletsprojects.com/p/flask/ 


**Available Versions**:: 


    1.1.2-GCCcore-8.3.0-Python-3.7.4
    1.1.4-GCCcore-10.3.0
    1.1.2-GCCcore-10.2.0



FlexiBLAS
--------- 



**Description** 


FlexiBLAS is a wrapper library that enables the exchange of the BLAS and LAPACK implementationused by a program without recompiling or relinking it. 


**More Information** 


https://gitlab.mpi-magdeburg.mpg.de/software/flexiblas-release 


**Available Versions**:: 


    3.0.4-GCC-11.2.0
    3.2.0-GCC-11.3.0
    3.0.4-GCC-10.3.0



FreeXL
------ 



**Description** 


FreeXL is an open source library to extract valid data from within an Excel (.xls) spreadsheet. 


**More Information** 


https://www.gaia-gis.it/fossil/freexl/index 


**Available Versions**:: 


    1.0.5-GCCcore-8.3.0



FriBidi
------- 



**Description** 


The Free Implementation of the Unicode Bidirectional Algorithm. 


**More Information** 


https://github.com/fribidi/fribidi 


**Available Versions**:: 


    1.0.5-GCCcore-8.2.0
    1.0.5-GCCcore-7.3.0
    1.0.9-GCCcore-9.3.0
    1.0.10-GCCcore-10.2.0
    1.0.5-GCCcore-8.3.0
    1.0.10-GCCcore-10.3.0



GATK
---- 



**Description** 


The Genome Analysis Toolkit or GATK is a software package developed at the Broad Institute to analyse next-generation resequencing data. The toolkit offers a wide variety of tools, with a primary focus on variant discovery and genotyping as well as strong emphasis on data quality assurance. Its robust architecture, powerful processing engine and high-performance computing features make it capable of taking on projects of any size. 


**More Information** 


https://www.broadinstitute.org/gatk/ 


**Available Versions**:: 


    4.1.5.0-GCCcore-9.3.0-Java-1.8
    4.1.8.1-GCCcore-9.3.0-Java-1.8
    3.8-1-Java-1.8.0_241



GCC
--- 



**Description** 


The GNU Compiler Collection includes front ends for C, C++, Objective-C, Fortran, Java, and Ada, as well as libraries for these languages (libstdc++, libgcj,...). 


**More Information** 


http://gcc.gnu.org/ 


**Available Versions**:: 


    8.2.0-2.31.1
    10.2.0
    11.2.0
    8.3.0
    8.1.0-2.30
    10.3.0
    11.3.0
    7.3.0-2.30
    9.3.0



GCCcore
------- 



**Description** 


The GNU Compiler Collection includes front ends for C, C++, Objective-C, Fortran, Java, and Ada, as well as libraries for these languages (libstdc++, libgcj,...). 


**More Information** 


https://gcc.gnu.org/ 


**Available Versions**:: 


    8.3.0
    10.3.0
    8.2.0
    7.3.0
    11.2.0
    11.3.0
    9.3.0
    8.1.0
    10.2.0



GConf
----- 



**Description** 


GConf is a system for storing application preferences. It is intended for user preferences; not configuration of something like Apache, or arbitrary data storage. 


**More Information** 


https://developer.gnome.org/gconf/ 


**Available Versions**:: 


    3.2.6-GCCcore-8.3.0



GDAL
---- 



**Description** 


GDAL is a translator library for raster geospatial data formats that is released under an X/MIT style Open Source license by the Open Source Geospatial Foundation. As a library, it presents a single abstract data model to the calling application for all supported formats. It also comes with a variety of useful commandline utilities for data translation and processing. 


**More Information** 


https://www.gdal.org 


**Available Versions**:: 


    3.3.2-foss-2021b
    3.0.4-foss-2020a-Python-3.8.2
    3.2.1-fosscuda-2020b
    3.0.0-foss-2019a-Python-2.7.15
    3.0.4-intel-2020a-Python-3.8.2
    3.3.0-foss-2021a



GDRCopy
------- 



**Description** 


A low-latency GPU memory copy library based on NVIDIA GPUDirect RDMA technology. 


**More Information** 


https://github.com/NVIDIA/gdrcopy 


**Available Versions**:: 


    2.2-GCCcore-10.3.0
    2.1-GCCcore-10.2.0-CUDA-11.1.1
    2.3-GCCcore-11.2.0
    2.1-GCCcore-10.3.0-CUDA-11.1.1
    2.1-GCCcore-9.3.0-CUDA-11.0.2



GEOS
---- 



**Description** 


GEOS (Geometry Engine - Open Source) is a C++ port of the Java Topology Suite (JTS) 


**More Information** 


https://trac.osgeo.org/geos 


**Available Versions**:: 


    3.8.1-GCC-9.3.0-Python-3.8.2
    3.8.1-iccifort-2020.1.217-Python-3.8.2
    3.7.2-foss-2019a-Python-2.7.15
    3.6.2-foss-2018b-Python-2.7.15
    3.9.1-GCC-10.3.0
    3.9.1-GCC-11.2.0
    3.9.1-GCC-10.2.0
    3.8.0-GCC-8.3.0-Python-3.7.4



GL2PS
----- 



**Description** 


GL2PS: an OpenGL to PostScript printing library 


**More Information** 


https://www.geuz.org/gl2ps/ 


**Available Versions**:: 


    1.4.0-GCCcore-8.3.0
    1.4.2-GCCcore-11.2.0



GLM
--- 



**Description** 


OpenGL Mathematics (GLM) is a header only C++ mathematics library for graphics software based on the OpenGL Shading Language (GLSL) specifications. 


**More Information** 


https://github.com/g-truc/glm 


**Available Versions**:: 


    0.9.9.8-GCCcore-9.3.0
    0.9.9.8-GCCcore-8.3.0



GLPK
---- 



**Description** 


The GLPK (GNU Linear Programming Kit) package is intended for solving large-scale linear programming (LP), mixed integer programming (MIP), and other related problems. It is a set of routines written in ANSI C  and organized in the form of a callable library. 


**More Information** 


https://www.gnu.org/software/glpk/ 


**Available Versions**:: 


    4.65-GCCcore-8.3.0
    5.0-GCCcore-10.3.0
    4.65-GCCcore-10.2.0
    4.65-GCCcore-9.3.0
    5.0-GCCcore-11.2.0



GLib
---- 



**Description** 


GLib is one of the base libraries of the GTK+ project 


**More Information** 


http://www.gtk.org/ 


**Available Versions**:: 


    2.54.3-GCCcore-7.3.0
    2.66.1-GCCcore-10.2.0
    2.64.1-GCCcore-9.3.0
    2.68.2-GCCcore-10.3.0
    2.69.1-GCCcore-11.2.0
    2.62.0-GCCcore-8.3.0
    2.60.1-GCCcore-8.2.0



GLibmm
------ 



**Description** 


C++ bindings for Glib 


**More Information** 


https://www.gtk.org/ 


**Available Versions**:: 


    2.49.7-GCCcore-8.3.0



GMAP-GSNAP
---------- 



**Description** 


GMAP: A Genomic Mapping and Alignment Program for mRNA and EST Sequences GSNAP: Genomic Short-read Nucleotide Alignment Program 


**More Information** 


http://research-pub.gene.com/gmap/ 


**Available Versions**:: 


    2019-09-12-GCC-8.3.0



GMP
--- 



**Description** 


GMP is a free library for arbitrary precision arithmetic, operating on signed integers, rational numbers, and floating point numbers. 


**More Information** 


https://gmplib.org/ 


**Available Versions**:: 


    6.1.2-GCCcore-10.2.0
    6.2.0-GCCcore-9.3.0
    6.1.2-GCCcore-8.3.0
    6.1.2-GCCcore-7.3.0
    6.2.1-GCCcore-11.2.0
    6.2.1-GCCcore-10.3.0
    6.1.2-GCCcore-8.2.0
    6.2.0-GCCcore-10.2.0



GObject-Introspection
--------------------- 



**Description** 


GObject introspection is a middleware layer between C libraries (using GObject) and language bindings. The C library can be scanned at compile time and generate a metadata file, in addition to the actual native C library. Then at runtime, language bindings can read this metadata and automatically provide bindings to call into the C library. 


**More Information** 


https://wiki.gnome.org/GObjectIntrospection/ 


**Available Versions**:: 


    1.63.1-GCCcore-8.3.0-Python-3.7.4
    1.64.0-GCCcore-9.3.0-Python-3.8.2
    1.68.0-GCCcore-10.3.0
    1.66.1-GCCcore-10.2.0
    1.54.1-foss-2018b-Python-2.7.15
    1.60.1-GCCcore-8.2.0-Python-3.7.2



GPAW
---- 



**Description** 


GPAW is a density-functional theory (DFT) Python code based on the projector-augmented wave (PAW) method and the atomic simulation environment (ASE). It uses real-space uniform grids and multigrid methods or atom-centered basis-functions. 


**More Information** 


https://wiki.fysik.dtu.dk/gpaw/ 


**Available Versions**:: 


    20.10.0-foss-2020b
    21.6.0-foss-2021a



GPAW-setups
----------- 



**Description** 


PAW setup for the GPAW Density Functional Theory package.  Users can install setups manually using 'gpaw install-data' or use setups from this package.  The versions of GPAW and GPAW-setups can be intermixed. 


**More Information** 


https://wiki.fysik.dtu.dk/gpaw/ 


**Available Versions**:: 


    0.9.20000



GROMACS
------- 



**Description** 


GROMACS is a versatile package to perform molecular dynamics, i.e. simulate the Newtonian equations of motion for systems with hundreds to millions of particles.This is a CPU only build, containing both MPI and threadMPI builds. 


**More Information** 


http://www.gromacs.org 


**Available Versions**:: 


    2021-foss-2021a-PLUMED-2.7.2
    2022.2-foss-2021a
    2020.4-foss-2020a
    2020.4-foss-2020a-PLUMED-2.6.2
    2021.3-foss-2021a
    2020-fosscuda-2019b
    2021-foss-2020b



GSL
--- 



**Description** 


The GNU Scientific Library (GSL) is a numerical library for C and C++ programmers. The library provides a wide range of mathematical routines such as random number generators, special functions and least-squares fitting. 


**More Information** 


https://www.gnu.org/software/gsl/ 


**Available Versions**:: 


    2.7-GCC-10.3.0
    2.6-GCC-8.3.0
    2.6-GCC-10.2.0
    2.7-GCC-11.2.0
    2.5-GCC-8.2.0-2.31.1
    2.6-iccifort-2020.1.217
    2.6-GCC-9.3.0
    2.6-iccifort-2020.4.304
    2.5-GCC-7.3.0-2.30



GST-plugins-base
---------------- 



**Description** 


GStreamer is a library for constructing graphs of media-handling components. The applications it supports range from simple Ogg/Vorbis playback, audio/video streaming to complex audio (mixing) and video (non-linear editing) processing. 


**More Information** 


https://gstreamer.freedesktop.org/ 


**Available Versions**:: 


    1.16.2-GCC-8.3.0



GStreamer
--------- 



**Description** 


GStreamer is a library for constructing graphs of media-handling components. The applications it supports range from simple Ogg/Vorbis playback, audio/video streaming to complex audio (mixing) and video (non-linear editing) processing. 


**More Information** 


https://gstreamer.freedesktop.org/ 


**Available Versions**:: 


    1.16.2-GCC-8.3.0



GTK+
---- 



**Description** 


The GTK+ 2 package contains libraries used for creating graphical user interfaces for applications. 


**More Information** 


https://developer.gnome.org/gtk+/stable/ 


**Available Versions**:: 


    3.24.8-GCCcore-8.2.0
    3.24.23-GCCcore-10.2.0
    3.24.17-GCCcore-9.3.0
    2.24.32-foss-2018b
    3.24.13-GCCcore-8.3.0



Gaussian
-------- 



**Description** 


Gaussian provides state-of-the-art capabilities for electronic structuremodeling. Gaussian 03 is licensed for a wide variety of computersystems. All versions of Gaussian 03 contain every scientific/modelingfeature, and none imposes any artificial limitations on calculationsother than your computing resources and patience.This is the build from the legacy ARCUS-B system, using the PGI compiler and Atlas. 


**More Information** 


https://www.gaussian.com/ 


**Available Versions**:: 


    03.E.01-ARCUS-B
    16.A.03-ARCUS-B
    09.D.01-ARCUS-B
    16.C.01



Gaussview
--------- 



**Available Versions**:: 


    5.0.9



Gdk-Pixbuf
---------- 



**Description** 


The Gdk Pixbuf is a toolkit for image loading and pixel buffer manipulation. It is used by GTK+ 2 and GTK+ 3 to load and manipulate images. In the past it was distributed as part of GTK+ 2 but it was split off into a separate package in preparation for the change to GTK+ 3. 


**More Information** 


https://developer.gnome.org/gdk-pixbuf/stable/ 


**Available Versions**:: 


    2.40.0-GCCcore-10.2.0
    2.38.1-GCCcore-8.2.0
    2.36.12-foss-2018b
    2.40.0-GCCcore-9.3.0
    2.38.2-GCCcore-8.3.0



Geant4
------ 



**Description** 


Geant4 is a toolkit for the simulation of the passage of particles through matter. Its areas of application include high energy, nuclear and accelerator physics,  as well as studies in medical and space science. 


**More Information** 


https://geant4.cern.ch/ 


**Available Versions**:: 


    10.7.1-GCC-10.2.0
    11.0.0-GCC-11.2.0



Geant4-data
----------- 



**Description** 


Datasets for Geant4. 


**More Information** 


https://geant4.cern.ch/ 


**Available Versions**:: 


    20210510



GenomeTools
----------- 



**Description** 


A comprehensive software library for efficient processing of structured genome annotations. 


**More Information** 


http://genometools.org 


**Available Versions**:: 


    1.6.1-GCC-8.3.0
    1.6.1-GCC-9.3.0
    1.6.2-GCC-10.3.0
    1.6.1-GCC-10.2.0



GeoMxNGSPipeline
---------------- 



**Available Versions**:: 


    2022



Ghostscript
----------- 



**Description** 


Ghostscript is a versatile processor for PostScript data with the ability to render PostScript to different targets. It used to be part of the cups printing stack, but is no longer used for that. 


**More Information** 


https://ghostscript.com 


**Available Versions**:: 


    9.53.3-GCCcore-10.2.0
    9.54.0-GCCcore-10.3.0
    9.52-GCCcore-9.3.0
    9.54.0-GCCcore-11.2.0
    9.50-GCCcore-8.3.0



GitPython
--------- 



**Description** 


GitPython is a python library used to interact with Git repositories  


**More Information** 


https://gitpython.readthedocs.org 


**Available Versions**:: 


    3.1.14-GCCcore-10.2.0
    3.1.0-GCCcore-8.3.0-Python-3.7.4



GlobalArrays
------------ 



**Description** 


Global Arrays (GA) is a Partitioned Global Address Space (PGAS) programming model 


**More Information** 


https://hpc.pnl.gov/globalarrays 


**Available Versions**:: 


    5.7.2-intel-2019b-peigs
    5.7-intel-2020b
    5.8-intel-2020a
    5.8-intel-2021a
    5.7.2-foss-2019b-peigs



Glucose
------- 



**Description** 


Glucose is based on a new scoring scheme (well, not so new now, it wasintroduced in 2009) for the clause learning mechanism of so called Modern SATsolvers (it is based on our IJCAI'09 paper). It is designed to be parallel, since v4.0. 


**More Information** 


https://www.labri.fr/perso/lsimon/glucose/ 


**Available Versions**:: 


    4.1-GCC-9.3.0



Go
-- 



**Description** 


Go is an open source programming language that makes it easy to build simple, reliable, and efficient software. 


**More Information** 


https://www.golang.org 


**Available Versions**:: 


    1.14.1



Grace
----- 



**Description** 


Grace is a WYSIWYG tool to make two-dimensional plots of numerical data. 


**More Information** 


https://plasma-gate.weizmann.ac.il/Grace/ 


**Available Versions**:: 


    5.1.25-foss-2019b-5build1



GraphicsMagick
-------------- 



**Description** 


GraphicsMagick is the swiss army knife of image processing. 


**More Information** 


https://www.graphicsmagick.org/ 


**Available Versions**:: 


    1.3.34-foss-2019b



Guile
----- 



**Description** 


Guile is a programming language, designed to help programmers create flexible applications that can be extended by users or other programmers with plug-ins, modules, or scripts. 


**More Information** 


https://www.gnu.org/software/guile/ 


**Available Versions**:: 


    1.8.8-GCCcore-8.2.0
    1.8.8-GCCcore-8.3.0
    1.8.8-GCCcore-9.3.0



Guppy
----- 



**Available Versions**:: 


    5.0.11
    3.6.0



Gurobi
------ 



**Description** 


The Gurobi Optimizer is a state-of-the-art solver for mathematical programming.The solvers in the Gurobi Optimizer were designed from the ground up to exploit modernarchitectures and multi-core processors, using the most advanced implementations of thelatest algorithms. 


**More Information** 


https://www.gurobi.com 


**Available Versions**:: 


    9.1.2-GCCcore-10.3.0



HDF
--- 



**Description** 


HDF (also known as HDF4) is a library and multi-object file format for storing and managing data between machines. 


**More Information** 


https://www.hdfgroup.org/products/hdf4/ 


**Available Versions**:: 


    4.2.15-GCCcore-10.3.0
    4.2.15-GCCcore-10.2.0
    4.2.14-GCCcore-8.3.0
    4.2.15-GCCcore-11.2.0



HDF5
---- 



**Description** 


HDF5 is a data model, library, and file format for storing and managing data. It supports an unlimited variety of datatypes, and is designed for flexible and efficient I/O and for high volume and complex data. 


**More Information** 


https://portal.hdfgroup.org/display/support 


**Available Versions**:: 


    1.10.5-gompic-2019b
    1.13.1-gompi-2022a
    1.10.2-intel-2020b
    1.10.2-foss-2018b
    1.10.5-gompi-2019b-dba
    1.10.5-iimpi-2020a
    1.10.7-iimpi-2021a
    1.10.6-gompic-2020a
    1.10.2-fosscuda-2018b
    1.12.0-gompi-2020a
    1.12.1-gompi-2021b
    1.10.7-gompi-2020b
    1.10.7-iimpi-2020b
    1.10.7-gompic-2020b
    1.10.5-gompi-2019b
    1.13.1-iimpi-2022a
    1.12.1-gompi-2021a
    1.10.5-iimpic-2019b
    1.10.7-gompi-2021a
    1.10.6-gompi-2020a
    1.10.6-iimpi-2020a
    1.10.5-gompi-2019a



HH-suite
-------- 



**Description** 


The HH-suite is an open-source software package for sensitive protein sequence searching based on the pairwise alignment of hidden Markov models (HMMs). 


**More Information** 


https://github.com/soedinglab/hh-suite 


**Available Versions**:: 


    3.3.0-gompic-2020b



HISAT2
------ 



**Description** 


HISAT2 is a fast and sensitive alignment program for mapping next-generation sequencing reads (both DNA and RNA) against the general human population (as well as against a single reference genome). 


**More Information** 


https://daehwankimlab.github.io/hisat2 


**Available Versions**:: 


    2.2.1-gompi-2020b



HMMER
----- 



**Description** 


HMMER is used for searching sequence databases for homologs of protein sequences, and for making protein sequence alignments. It implements methods using probabilistic models called profile hidden Markov models (profile HMMs).  Compared to BLAST, FASTA, and other sequence alignment and database search tools based on older scoring methodology, HMMER aims to be significantly more accurate and more able to detect remote homologs because of the strength of its underlying mathematical models. In the past, this strength came at significant computational expense, but in the new HMMER3 project, HMMER is now essentially as fast as BLAST. 


**More Information** 


http://hmmer.org/ 


**Available Versions**:: 


    3.3.1-iimpi-2020a
    3.3.2-gompi-2021a
    3.3.2-gompi-2020b
    3.3.2-gompic-2020b



HMMER2
------ 



**Description** 


HMMER is used for searching sequence databases for sequence homologs, and for making sequence alignments. 


**More Information** 


http://hmmer.org 


**Available Versions**:: 


    2.3.2-GCC-8.3.0



HTSlib
------ 



**Description** 


A C library for reading/writing high-throughput sequencing data. This package includes the utilities bgzip and tabix 


**More Information** 


http://www.htslib.org/ 


**Available Versions**:: 


    1.9-foss-2018b
    1.11-GCC-10.2.0
    1.10.2-GCC-8.3.0
    1.14-GCC-11.2.0
    1.12-GCC-10.2.0
    1.10.2-GCC-9.3.0



HarfBuzz
-------- 



**Description** 


HarfBuzz is an OpenType text shaping engine. 


**More Information** 


https://www.freedesktop.org/wiki/Software/HarfBuzz 


**Available Versions**:: 


    2.6.4-GCCcore-9.3.0
    2.4.0-GCCcore-8.2.0
    2.6.4-GCCcore-8.3.0
    2.6.7-GCCcore-10.2.0
    2.8.1-GCCcore-10.3.0
    2.2.0-foss-2018b



HyPhy
----- 



**Description** 


HyPhy (Hypothesis Testing using Phylogenies) is an open-source software package  for the analysis of genetic sequences (in particular the inference of natural selection)  using techniques in phylogenetics, molecular evolution, and machine learning 


**More Information** 


https://veg.github.io/hyphy-site/ 


**Available Versions**:: 


    2.5.1-gompi-2019a



Hypre
----- 



**Description** 


Hypre is a library for solving large, sparse linear systems of equations on massively parallel computers. The problems of interest arise in the simulation codes being developed at LLNL and elsewhere to study physical phenomena in the defense, environmental, energy, and biological sciences. 


**More Information** 


https://computation.llnl.gov/projects/hypre-scalable-linear-solvers-multigrid-methods 


**Available Versions**:: 


    2.18.2-foss-2020a
    2.21.0-foss-2021a
    2.18.2-foss-2019b
    2.20.0-foss-2020b
    2.18.2-intel-2020a



ICU
--- 



**Description** 


ICU is a mature, widely used set of C/C++ and Java libraries providing Unicode and Globalization support for software applications. 


**More Information** 


https://icu-project.org/ 


**Available Versions**:: 


    66.1-GCCcore-9.3.0
    69.1-GCCcore-10.3.0
    64.2-GCCcore-8.2.0
    67.1-GCCcore-10.2.0
    64.2-GCCcore-8.3.0
    69.1-GCCcore-11.2.0
    61.1-GCCcore-7.3.0



IDBA-UD
------- 



**Description** 


IDBA-UD is a iterative De Bruijn Graph De Novo Assembler for Short Reads Sequencing data with Highly Uneven Sequencing Depth. It is an extension of IDBA algorithm. IDBA-UD also iterates from small k to a large k. In each iteration, short and low-depth contigs are removed iteratively with cutoff threshold from low to high to reduce the errors in low-depth and high-depth regions. Paired-end reads are aligned to contigs and assembled locally to generate some missing k-mers in low-depth regions. With these technologies, IDBA-UD can iterate k value of de Bruijn graph to a very large value with less gaps and less branches to form long contigs in both low-depth and high-depth regions. 


**More Information** 


https://i.cs.hku.hk/~alse/hkubrg/projects/idba_ud/ 


**Available Versions**:: 


    1.1.3-GCC-8.3.0



IGV
--- 



**Description** 


This package contains command line utilities for preprocessing, computing feature count density (coverage),  sorting, and indexing data files. 


**More Information** 


https://www.broadinstitute.org/software/igv/ 


**Available Versions**:: 


    2.8.0-Java-11



IMPUTE2
------- 



**Description** 


IMPUTE version 2 (also known as IMPUTE2) is a genotype imputation  and haplotype phasing program based on ideas from Howie et al. 2009  


**More Information** 


http://mathgen.stats.ox.ac.uk/impute/impute_v2.html 


**Available Versions**:: 


    2.3.2_x86_64_dynamic
    2.3.2_x86_64_static



IOR
--- 



**Description** 


The IOR software is used for benchmarking parallel file systems using POSIX, MPIIO, or HDF5 interfaces.  


**More Information** 


https://github.com/IOR-LANL/ior 


**Available Versions**:: 


    3.2.1-gompi-2019b
    3.3.0-gompi-2020b



IPython
------- 



**Description** 


IPython provides a rich architecture for interactive computing with: Powerful interactive shells (terminal and Qt-based). A browser-based notebook with support for code, text, mathematical expressions, inline plots and other rich media. Support for interactive data visualization and use of GUI toolkits. Flexible, embeddable interpreters to load into your own projects. Easy to use, high performance tools for parallel computing. 


**More Information** 


https://ipython.org/index.html 


**Available Versions**:: 


    7.18.1-GCCcore-10.2.0
    7.9.0-foss-2019b-Python-3.7.4



IQ-TREE
------- 



**Description** 


Efficient phylogenomic software by maximum likelihood 


**More Information** 


http://www.iqtree.org/ 


**Available Versions**:: 


    1.6.12-foss-2018b



ISA-L
----- 



**Description** 


Intelligent Storage Acceleration Library 


**More Information** 


https://github.com/intel/isa-l 


**Available Versions**:: 


    2.30.0-GCCcore-11.2.0



ImageMagick
----------- 



**Description** 


ImageMagick is a software suite to create, edit, compose, or convert bitmap images 


**More Information** 


https://www.imagemagick.org/ 


**Available Versions**:: 


    7.0.11-14-GCCcore-10.3.0
    7.0.10-35-GCCcore-10.2.0
    7.0.9-5-GCCcore-8.3.0
    7.1.0-4-GCCcore-11.2.0
    7.0.10-1-GCCcore-9.3.0



Infernal
-------- 



**Description** 


Infernal ("INFERence of RNA ALignment") is for searching DNA sequence databases for RNA structure and sequence similarities. 


**More Information** 


http://eddylab.org/infernal/ 


**Available Versions**:: 


    1.1.2-foss-2018b



Ipopt
----- 



**Description** 


Ipopt (Interior Point OPTimizer, pronounced eye-pea-Opt) is a software package for large-scale nonlinear optimization. 


**More Information** 


https://coin-or.github.io/Ipopt 


**Available Versions**:: 


    3.12.13-intel-2020b



JAGS
---- 



**Description** 


JAGS is Just Another Gibbs Sampler.  It is a program for analysis  of Bayesian hierarchical models using Markov Chain Monte Carlo (MCMC) simulation   


**More Information** 


http://mcmc-jags.sourceforge.net/ 


**Available Versions**:: 


    4.3.0-foss-2020a
    4.3.0-foss-2021b
    4.3.0-foss-2019b
    4.3.0-foss-2021a



JasPer
------ 



**Description** 


The JasPer Project is an open-source initiative to provide a free software-based reference implementation of the codec specified in the JPEG-2000 Part-1 standard. 


**More Information** 


http://www.ece.uvic.ca/~frodo/jasper/ 


**Available Versions**:: 


    2.0.14-GCCcore-7.3.0
    2.0.24-GCCcore-10.2.0
    1.900.1-intel-2020b
    2.0.28-GCCcore-10.3.0
    2.0.14-GCCcore-10.2.0
    2.0.14-GCCcore-9.3.0
    2.0.33-GCCcore-11.2.0
    2.0.14-GCCcore-8.2.0
    2.0.14-GCCcore-8.3.0



Java
---- 



**Description** 


Java Platform, Standard Edition (Java SE) lets you develop and deploy Java applications on desktops and servers. 


**More Information** 


https://java.com/ 


**Available Versions**:: 


    1.7.0_60
    1.8.0_241
    1.8.0_131
    11.0.2



Jellyfish
--------- 



**Description** 


Jellyfish is a tool for fast, memory-efficient counting of k-mers in DNA. 


**More Information** 


http://www.genome.umd.edu/jellyfish.html 


**Available Versions**:: 


    2.3.0-GCC-8.3.0



JsonCpp
------- 



**Description** 


JsonCpp is a C++ library that allows manipulating JSON values, including serialization and deserialization to and from strings. It can also preserve existing comment in unserialization/serialization steps, making it a convenient format to store user input files.  


**More Information** 


https://open-source-parsers.github.io/jsoncpp-docs/doxygen/index.html 


**Available Versions**:: 


    1.9.4-GCCcore-11.2.0
    1.9.4-GCCcore-9.3.0
    1.9.4-GCCcore-10.3.0
    1.9.4-GCCcore-10.2.0
    1.9.3-GCCcore-8.3.0



Judy
---- 



**Description** 


A C library that implements a dynamic array. 


**More Information** 


http://judy.sourceforge.net/ 


**Available Versions**:: 


    1.0.5-GCCcore-10.3.0
    1.0.5-GCCcore-10.2.0
    1.0.5-GCCcore-8.3.0



Julia
----- 



**Description** 


Julia is a high-level, high-performance dynamic programming language for numerical computing 


**More Information** 


https://julialang.org 


**Available Versions**:: 


    1.6.2-linux-x86_64
    1.5.1-linux-x86_64
    1.5.3-linux-x86_64



JupyterHub
---------- 



**Description** 


JupyterHub is a multiuser version of the Jupyter (IPython) notebook designed for centralized deployments in companies, university classrooms and research labs. 


**More Information** 


https://jupyter.org 


**Available Versions**:: 


    1.1.0-GCCcore-10.2.0



JupyterLab
---------- 



**Description** 


JupyterLab is the next-generation user interface for Project Jupyter offering all the familiar building blocks of the classic Jupyter Notebook (notebook, terminal, text editor, file browser, rich outputs, etc.) in a flexible and powerful user interface. JupyterLab will eventually replace the classic Jupyter Notebook. 


**More Information** 


https://jupyter.org/ 


**Available Versions**:: 


    2.2.8-GCCcore-10.2.0



KMC
--- 



**Description** 


KMC is a disk-based programm for counting k-mers from (possibly gzipped) FASTQ/FASTA files. 


**More Information** 


http://sun.aei.polsl.pl/kmc 


**Available Versions**:: 


    3.1.0-foss-2018b



Kalign
------ 



**Description** 


Kalign is a fast multiple sequence alignment program for biological sequences. 


**More Information** 


https://github.com/TimoLassmann/kalign 


**Available Versions**:: 


    3.3.1-GCCcore-10.2.0



Kent_tools
---------- 



**Description** 


Kent utilities: collection of tools used by the UCSC genome browser. 


**More Information** 


https://genome.cse.ucsc.edu/ 


**Available Versions**:: 


    411-GCC-10.2.0
    401-gompi-2019b
    418-GCC-10.3.0



Keras
----- 



**Description** 


Keras is a minimalist, highly modular neural networks library, written in Python andcapable of running on top of either TensorFlow or Theano. 


**More Information** 


https://keras.io/ 


**Available Versions**:: 


    2.3.1-foss-2019b-Python-3.7.4
    2.4.3-fosscuda-2020b



Kraken2
------- 



**Description** 


Kraken is a system for assigning taxonomic labels to short DNA sequences, usually obtained through metagenomic studies. Previous attempts by other bioinformatics software to accomplish this task have often used sequence alignment or machine learning techniques that were quite slow, leading to the development of less sensitive but much faster abundance estimation programs. Kraken aims to achieve high sensitivity and high speed by utilizing exact alignments of k-mers and a novel classification algorithm. 


**More Information** 


https://github.com/DerrickWood/kraken2/wiki 


**Available Versions**:: 


    2.1.1-gompi-2020b



LAME
---- 



**Description** 


LAME is a high quality MPEG Audio Layer III (MP3) encoder licensed under the LGPL. 


**More Information** 


http://lame.sourceforge.net/ 


**Available Versions**:: 


    3.100-GCCcore-8.2.0
    3.100-GCCcore-8.3.0
    3.100-GCCcore-10.2.0
    3.100-GCCcore-10.3.0
    3.100-GCCcore-9.3.0



LAMMPS
------ 



**Description** 


LAMMPS is a classical molecular dynamics code, and an acronymfor Large-scale Atomic/Molecular Massively Parallel Simulator. LAMMPS haspotentials for solid-state materials (metals, semiconductors) and soft matter(biomolecules, polymers) and coarse-grained or mesoscopic systems. It can beused to model atoms or, more generically, as a parallel particle simulator atthe atomic, meso, or continuum scale. LAMMPS runs on single processors or inparallel using message-passing techniques and a spatial-decomposition of thesimulation domain. The code is designed to be easy to modify or extend with newfunctionality. 


**More Information** 


https://lammps.sandia.gov/ 


**Available Versions**:: 


    3Mar2020-foss-2020a-Python-3.8.2-kokkos
    3Mar2020-foss-2020a-Python-3.8.2-kokkos-QUIP



LAPACK
------ 



**Description** 


LAPACK is written in Fortran90 and provides routines for solving systems of simultaneous linear equations, least-squares solutions of linear systems of equations, eigenvalue problems, and singular value problems. 


**More Information** 


https://www.netlib.org/lapack/ 


**Available Versions**:: 


    3.9.1-GCC-10.2.0
    3.9.1-GCC-9.3.0
    3.9.1-GCC-11.2.0
    3.9.1-GCC-10.3.0



LDC
--- 



**Description** 


The LLVM-based D Compiler 


**More Information** 


https://wiki.dlang.org/LDC 


**Available Versions**:: 


    0.17.6-x86_64
    1.26.0-GCCcore-10.3.0
    1.25.1-GCCcore-10.2.0
    1.28.1-GCCcore-8.3.0



LLVM
---- 



**Description** 


The LLVM Core libraries provide a modern source- and target-independent optimizer, along with code generation support for many popular CPUs (as well as some less common ones!) These libraries are built around a well specified code representation known as the LLVM intermediate representation ("LLVM IR"). The LLVM Core libraries are well documented, and it is particularly easy to invent your own language (or port an existing compiler) to use LLVM as an optimizer and code generator. 


**More Information** 


https://llvm.org/ 


**Available Versions**:: 


    7.0.1-GCCcore-8.2.0
    11.1.0-GCCcore-10.3.0
    6.0.0-GCCcore-7.3.0
    8.0.1-GCCcore-8.3.0
    11.0.0-GCCcore-10.2.0
    9.0.1-GCCcore-9.3.0
    9.0.0-GCCcore-8.3.0
    12.0.1-GCCcore-11.2.0



LMDB
---- 



**Description** 


LMDB is a fast, memory-efficient database. With memory-mapped files, it has the read performance of a pure in-memory database while retaining the persistence of standard disk-based databases. 


**More Information** 


https://symas.com/lmdb 


**Available Versions**:: 


    0.9.28-GCCcore-10.3.0
    0.9.24-GCCcore-9.3.0
    0.9.22-GCCcore-7.3.0
    0.9.24-GCCcore-8.3.0
    0.9.24-GCCcore-10.2.0
    0.9.29-GCCcore-11.2.0



LTR_retriever
------------- 



**Description** 


LTR_retriever is a highly accurate and sensitive program for identification of LTR retrotransposons; The LTR Assembly Index (LAI) is also included in this package. 


**More Information** 


https://github.com/oushujun/LTR_retriever 


**Available Versions**:: 


    2.9.0-foss-2020b
    2.9.0-foss-2021a



LZO
--- 



**Description** 


Portable lossless data compression library 


**More Information** 


https://www.oberhumer.com/opensource/lzo/ 


**Available Versions**:: 


    2.10-GCCcore-10.3.0
    2.10-GCCcore-8.3.0
    2.10-GCCcore-10.2.0



Leptonica
--------- 



**Description** 


Leptonica is a collection of pedagogically-oriented open source software that is broadly useful for image processing and image analysis applications. 


**More Information** 


http://www.leptonica.org 


**Available Versions**:: 


    1.78.0-GCCcore-8.2.0



LibTIFF
------- 



**Description** 


tiff: Library and tools for reading and writing TIFF data files 


**More Information** 


https://libtiff.maptools.org/ 


**Available Versions**:: 


    4.1.0-GCCcore-9.3.0
    4.0.10-GCCcore-8.2.0
    4.2.0-GCCcore-10.3.0
    4.3.0-GCCcore-11.2.0
    4.0.9-GCCcore-7.3.0
    4.0.10-GCCcore-8.3.0
    4.1.0-GCCcore-10.2.0



Libint
------ 



**Description** 


Libint library is used to evaluate the traditional (electron repulsion) and certain novel two-body matrix elements (integrals) over Cartesian Gaussian functions used in modern atomic and molecular theory. 


**More Information** 


https://sourceforge.net/p/libint/ 


**Available Versions**:: 


    1.1.6-foss-2019b
    1.1.6-GCC-8.2.0-2.31.1
    2.6.0-gompi-2020a-lmax-6-cp2k
    2.6.0-iccifort-2020.4.304-lmax-6-cp2k
    2.6.0-GCC-10.2.0-lmax-6-cp2k
    1.1.6-foss-2020a
    2.6.0-iimpi-2020a-lmax-6-cp2k
    1.1.6-intel-2020a



Lighter
------- 



**Description** 


Fast and memory-efficient sequencing error corrector 


**More Information** 


https://github.com/mourisl/Lighter 


**Available Versions**:: 


    1.1.2-foss-2018b



LinkTest
-------- 



**Description** 


The mpilinktest program is a parallel ping-pong test between all connections of a machine. Output of this program is a fullcommunication matrix which shows the bandwidth between each processorpair and a report including the minimum bandwidth. The linktest runsfor n processors in n steps where in each step n/2 pairs of processorswill perform the MPI pingpong test (3 iterations, 128 kBmessages). The selection of the pairs is random but after running allsteps all possible pairs are covered. 


**More Information** 


http://www.fz-juelich.de/ias/jsc/EN/Expertise/Support/Software/LinkTest/linktest-download_node.html 


**Available Versions**:: 


    1.2p1-foss-2019b
    1.2p1-foss-2020a



LittleCMS
--------- 



**Description** 


Little CMS intends to be an OPEN SOURCE small-footprint color management engine, with special focus on accuracy and performance.  


**More Information** 


http://www.littlecms.com/ 


**Available Versions**:: 


    2.9-GCCcore-9.3.0
    2.11-GCCcore-10.2.0
    2.12-GCCcore-10.3.0
    2.9-GCCcore-8.3.0
    2.12-GCCcore-11.2.0



Lua
--- 



**Description** 


Lua is a powerful, fast, lightweight, embeddable scripting language. Lua combines simple procedural syntax with powerful data description constructs based on associative arrays and extensible semantics. Lua is dynamically typed, runs by interpreting bytecode for a register-based virtual machine, and has automatic memory management with incremental garbage collection, making it ideal for configuration, scripting, and rapid prototyping. 


**More Information** 


https://www.lua.org/ 


**Available Versions**:: 


    5.1.5-GCCcore-8.3.0
    5.3.5-GCCcore-9.3.0
    5.4.2-GCCcore-10.2.0
    5.4.3-GCCcore-10.3.0



M4
-- 



**Description** 


GNU M4 is an implementation of the traditional Unix macro processor. It is mostly SVR4 compatible  although it has some extensions (for example, handling more than 9 positional parameters to macros). GNU M4 also has built-in functions for including files, running shell commands, doing arithmetic, etc. 


**More Information** 


https://www.gnu.org/software/m4/m4.html 


**Available Versions**:: 


    1.4.18-GCCcore-10.2.0
    1.4.18
    1.4.19-GCCcore-11.3.0
    1.4.17
    1.4.18-GCCcore-9.3.0
    1.4.19-GCCcore-11.2.0
    1.4.19
    1.4.18-GCCcore-8.1.0
    1.4.18-GCCcore-10.3.0
    1.4.18-GCCcore-7.3.0
    1.4.18-GCCcore-8.3.0
    1.4.18-GCCcore-8.2.0



MACS2
----- 



**Description** 


Model Based Analysis for ChIP-Seq data 


**More Information** 


https://github.com/taoliu/MACS/ 


**Available Versions**:: 


    2.2.5-foss-2018b-Python-3.6.6



MAFFT
----- 



**Description** 


MAFFT is a multiple sequence alignment program for unix-like operating systems.It offers a range of multiple alignment methods, L-INS-i (accurate; for alignmentof <~200 sequences), FFT-NS-2 (fast; for alignment of <~30,000 sequences), etc. 


**More Information** 


https://mafft.cbrc.jp/alignment/software/ 


**Available Versions**:: 


    7.487-gompi-2021a-with-extensions
    7.470-gompi-2020a-with-extensions
    7.475-gompi-2020b-with-extensions
    7.453-GCC-9.3.0-with-extensions



MALT
---- 



**Available Versions**:: 


    0.5.3



MATIO
----- 



**Description** 


matio is an C library for reading and writing Matlab MAT files. 


**More Information** 


https://sourceforge.net/projects/matio/ 


**Available Versions**:: 


    1.5.17-GCCcore-8.3.0



MATLAB
------ 



**Available Versions**:: 


    R2019b
    R2021b
    R2020a
    R2022a
    R2020b



MCL
--- 



**Description** 


The MCL algorithm is short for the Markov Cluster Algorithm, a fastand scalable unsupervised cluster algorithm for graphs (also known as networks) basedon simulation of (stochastic) flow in graphs.  


**More Information** 


https://micans.org/mcl/ 


**Available Versions**:: 


    14.137-GCCcore-8.3.0
    14.137-GCCcore-9.3.0



MDAnalysis
---------- 



**Description** 


MDAnalysis is an object-oriented Python library to analyze trajectories from molecular dynamics (MD)simulations in many popular formats. 


**More Information** 


https://www.mdanalysis.org/ 


**Available Versions**:: 


    0.20.1-foss-2019b-Python-3.7.4



MEGAHIT
------- 



**Description** 


An ultra-fast single-node solution for large and complex metagenomics assembly via succinct de Bruijn graph 


**More Information** 


https://github.com/voutcn/megahit 


**Available Versions**:: 


    1.2.9-GCCcore-9.3.0
    1.1.4-foss-2018b-Python-2.7.15



MEME
---- 



**Description** 


The MEME Suite allows you to: * discover motifs using MEME, DREME (DNA only) or GLAM2 on groups of related DNA or protein sequences, * search sequence databases with motifs using MAST, FIMO, MCAST or GLAM2SCAN, * compare a motif to all motifs in a database of motifs, * associate motifs with Gene Ontology terms via their putative target genes, and * analyse motif enrichment using SpaMo or CentriMo. 


**More Information** 


https://meme-suite.org/meme/index.html 


**Available Versions**:: 


    5.4.1-GCC-10.3.0



METIS
----- 



**Description** 


METIS is a set of serial programs for partitioning graphs, partitioning finite element meshes, and producing fill reducing orderings for sparse matrices. The algorithms implemented in METIS are based on the multilevel recursive-bisection, multilevel k-way, and multi-constraint partitioning schemes. 


**More Information** 


http://glaros.dtc.umn.edu/gkhome/metis/metis/overview 


**Available Versions**:: 


    5.1.0-GCCcore-10.3.0
    5.1.0-foss-2018b
    5.1.0-GCCcore-8.2.0
    5.1.0-GCCcore-8.3.0
    5.1.0-GCCcore-9.3.0
    5.1.0-GCCcore-10.2.0



MGLTools
-------- 



**Available Versions**:: 


    1.5.7-old
    1.5.7



MIRA
---- 



**Description** 


MIRA is a whole genome shotgun and EST sequence assembler for Sanger, 454, Solexa (Illumina), IonTorrent data and PacBio (the latter at the moment only CCS and error-corrected CLR reads). 


**More Information** 


https://sourceforge.net/p/mira-assembler/wiki/Home/ 


**Available Versions**:: 


    4.0.2-gompi-2019b



MMseqs2
------- 



**Description** 


MMseqs2: ultra fast and sensitive search and clustering suite 


**More Information** 


https://mmseqs.com 


**Available Versions**:: 


    13-45111-gompi-2020b
    10-6d92c-gompi-2019b



MPC
--- 



**Description** 


Gnu Mpc is a C library for the arithmetic of complex numbers with arbitrarily high precision and correct rounding of the result. It extends the principles of the IEEE-754 standard for fixed precision real floating point numbers to complex numbers, providing well-defined semantics for every operation. At the same time, speed of operation at high precision is a major design goal. 


**More Information** 


http://www.multiprecision.org/ 


**Available Versions**:: 


    1.1.0-GCC-9.3.0
    1.1.0-GCC-8.3.0
    1.2.1-GCCcore-10.3.0



MPFR
---- 



**Description** 


The MPFR library is a C library for multiple-precision floating-point computations with correct rounding. 


**More Information** 


https://www.mpfr.org 


**Available Versions**:: 


    4.1.0-GCCcore-11.2.0
    4.1.0-GCCcore-10.2.0
    4.0.2-GCCcore-9.3.0
    4.0.2-GCCcore-8.2.0
    4.0.1-GCCcore-7.3.0
    4.1.0-GCCcore-10.3.0
    4.0.2-GCCcore-8.3.0



MUMPS
----- 



**Description** 


A parallel sparse direct solver 


**More Information** 


https://graal.ens-lyon.fr/MUMPS/ 


**Available Versions**:: 


    5.4.0-foss-2021a-metis
    5.3.5-foss-2020b-metis
    5.2.1-foss-2020a-metis
    5.2.1-intel-2020a-metis
    5.2.1-foss-2019b-metis



MUMmer
------ 



**Description** 


MUMmer is a system for rapidly aligning entire genomes, whether in complete or draft form. AMOS makes use of it. 


**More Information** 


http://mummer.sourceforge.net/ 


**Available Versions**:: 


    4.0.0beta2-GCCcore-10.2.0
    4.0.0beta2-foss-2018b



MUSCLE
------ 



**Description** 


MUSCLE is one of the best-performing multiple alignment programs  according to published benchmark tests, with accuracy and speed that are consistently  better than CLUSTALW. MUSCLE can align hundreds of sequences in seconds. Most users  learn everything they need to know about MUSCLE in a few minutes-only a handful of command-line options are needed to perform common alignment tasks. 


**More Information** 


https://drive5.com/muscle/ 


**Available Versions**:: 


    3.8.31-foss-2018b



MaSuRCA
------- 



**Description** 


MaSuRCA is whole genome assembly software. It combines the efficiency of the de Bruijn graph and Overlap-Layout-Consensus (OLC) approaches. MaSuRCA can assemble data sets containing only short reads from Illumina sequencing or a mixture of short reads and long reads (Sanger, 454, Pacbio and Nanopore). 


**More Information** 


https://www.genome.umd.edu/masurca.html 


**Available Versions**:: 


    4.0.9-foss-2021a-Perl-5.32.1



Mako
---- 



**Description** 


A super-fast templating language that borrows the best ideas from the existing templating languages 


**More Information** 


https://www.makotemplates.org 


**Available Versions**:: 


    1.1.4-GCCcore-11.2.0
    1.0.7-foss-2018b-Python-2.7.15
    1.1.2-GCCcore-9.3.0
    1.0.8-GCCcore-8.2.0
    1.1.4-GCCcore-10.3.0
    1.1.0-GCCcore-8.3.0
    1.1.3-GCCcore-10.2.0



MariaDB
------- 



**Description** 


MariaDB is an enhanced, drop-in replacement for MySQL.Included engines: myISAM, Aria, InnoDB, RocksDB, TokuDB, OQGraph, Mroonga. 


**More Information** 


https://mariadb.org/ 


**Available Versions**:: 


    10.4.13-gompi-2019b
    10.5.8-GCC-10.2.0
    10.6.4-GCC-10.3.0



MariaDB-connector-c
------------------- 



**Description** 


MariaDB Connector/C is used to connect applications developed in C/C++ to MariaDB and MySQL databases. 


**More Information** 


https://downloads.mariadb.org/connector-c/ 


**Available Versions**:: 


    2.3.7-GCCcore-8.3.0



Mash
---- 



**Description** 


Fast genome and metagenome distance estimation using MinHash 


**More Information** 


http://mash.readthedocs.org 


**Available Versions**:: 


    2.1-foss-2018b



Mathematica
----------- 



**Available Versions**:: 


    11.3.0
    13.0.0
    12.2.0



Mellanox
-------- 



**Available Versions**:: 


    ib_mgmt-5.8.1-1.el8



Mercurial
--------- 



**Description** 


Mercurial is a free, distributed source control management tool. It efficiently handles projectsof any size and offers an easy and intuitive interface. 


**More Information** 


https://www.mercurial-scm.org 


**Available Versions**:: 


    5.7.1-GCCcore-10.2.0



Mesa
---- 



**Description** 


Mesa is an open-source implementation of the OpenGL specification - a system for rendering interactive 3D graphics. 


**More Information** 


https://www.mesa3d.org/ 


**Available Versions**:: 


    21.1.1-GCCcore-10.3.0
    19.1.7-GCCcore-8.3.0
    19.0.1-GCCcore-8.2.0
    21.1.7-GCCcore-11.2.0
    20.0.2-GCCcore-9.3.0
    18.1.1-foss-2018b
    20.2.1-GCCcore-10.2.0



Meson
----- 



**Description** 


Meson is a cross-platform build system designed to be both as fast and as user friendly as possible. 


**More Information** 


https://mesonbuild.com 


**Available Versions**:: 


    0.51.2-GCCcore-8.3.0-Python-3.7.4
    0.55.1-GCCcore-9.3.0-Python-3.8.2
    0.58.0-GCCcore-10.3.0
    0.58.2-GCCcore-11.2.0
    0.50.0-GCCcore-8.2.0-Python-3.7.2
    0.55.3-GCCcore-10.2.0



MetaPhlAn2
---------- 



**Description** 


MetaPhlAn is a computational tool for profiling the composition of microbial communities (Bacteria, Archaea, Eukaryotes and Viruses) from metagenomic shotgun sequencing data (i.e. not 16S) with species-level. With the newly added StrainPhlAn module, it is now possible to perform accurate strain-level microbial profiling. 


**More Information** 


https://bitbucket.org/biobakery/metaphlan2/ 


**Available Versions**:: 


    2.7.8-foss-2018b-Python-3.6.6



MiniSat
------- 



**Description** 


MiniSat is a minimalistic, open-source SAT solver, developed to helpresearchers and developers alike to get started on SAT. 


**More Information** 


http://minisat.se/ 


**Available Versions**:: 


    2.2.0-GCC-9.3.0



Miniconda3
---------- 



**Description** 


Miniconda is a free minimal installer for conda. It is a small, bootstrap version of Anaconda that includes only conda, Python, the packages they depend on, and a small number of other useful packages. 


**More Information** 


https://docs.conda.io/en/latest/miniconda.html 


**Available Versions**:: 


    4.7.10
    4.9.2



Molden
------ 



**Description** 


Molden is a package for displaying Molecular Density from the Ab Initio packages GAMESS-UK, GAMESS-US and GAUSSIAN and the Semi-Empirical packages Mopac/Ampac 


**More Information** 


http://www.cmbi.ru.nl/molden/ 


**Available Versions**:: 


    5.7-foss-2018b



Mono
---- 



**Description** 


An open source, cross-platform, implementation of C# and the CLR that is binary compatible with Microsoft.NET. 


**More Information** 


https://www.mono-project.com/ 


**Available Versions**:: 


    6.8.0.105-GCCcore-8.3.0



Mothur
------ 



**Description** 


Mothur is a single piece of open-source, expandable software to fill the bioinformatics needs of the microbial ecology community. 


**More Information** 


https://www.mothur.org/ 


**Available Versions**:: 


    1.43.0-foss-2019a-Python-3.7.2



MrBayes
------- 



**Description** 


MrBayes is a program for Bayesian inference and model choice across a wide range of phylogenetic and evolutionary models. 


**More Information** 


https://nbisweden.github.io/MrBayes/ 


**Available Versions**:: 


    3.2.7-gompi-2020b



MultiQC
------- 



**Description** 


Aggregate results from bioinformatics analyses across many samples into a single  report. MultiQC searches a given directory for analysis logs and compiles a HTML report. It's a general use tool, perfect for summarising the output from numerous bioinformatics tools. 


**More Information** 


https://multiqc.info 


**Available Versions**:: 


    1.9-foss-2020a-Python-3.8.2



NAMD
---- 



**Description** 


NAMD is a parallel molecular dynamics code designed for high-performance simulation of large biomolecular systems. 


**More Information** 


https://www.ks.uiuc.edu/Research/namd/ 


**Available Versions**:: 


    2.14-intel-2020a-mpi
    2.14-fosscuda-2019b



NASM
---- 



**Description** 


NASM: General-purpose x86 assembler 


**More Information** 


https://www.nasm.us/ 


**Available Versions**:: 


    2.14.02-GCCcore-8.3.0
    2.14.02-GCCcore-9.3.0
    2.13.03-GCCcore-7.3.0
    2.14.02-GCCcore-8.2.0
    2.15.05-GCCcore-11.2.0
    2.15.05-GCCcore-10.2.0
    2.15.05-GCCcore-10.3.0



NCCL
---- 



**Description** 


The NVIDIA Collective Communications Library (NCCL) implements multi-GPU and multi-node collectivecommunication primitives that are performance optimized for NVIDIA GPUs. 


**More Information** 


https://developer.nvidia.com/nccl 


**Available Versions**:: 


    2.10.3-GCCcore-10.3.0-CUDA-11.3.1
    2.8.3-GCCcore-10.2.0-CUDA-11.1.1
    2.10.3-GCCcore-11.2.0-CUDA-11.4.1
    2.4.8-gcccuda-2019b
    2.8.3-CUDA-11.1.1
    2.8.3-CUDA-11.0.2



NCO
--- 



**Description** 


manipulates and analyzes data stored in netCDF-accessible formats, including DAP, HDF4, and HDF5 


**More Information** 


https://nco.sourceforge.net 


**Available Versions**:: 


    4.9.7-foss-2020b
    5.0.1-foss-2021a



NEURON
------ 



**Description** 


Empirically-based simulations of neurons and networks of neurons. 


**More Information** 


http://www.neuron.yale.edu/neuron 


**Available Versions**:: 


    7.6.5-foss-2019a-Python-2.7.15
    7.8.2-foss-2021b



NGS
--- 



**Description** 


NGS is a new, domain-specific API for accessing reads, alignments and pileupsproduced from Next Generation Sequencing. 


**More Information** 


https://github.com/ncbi/ngs 


**Available Versions**:: 


    2.10.9-GCCcore-10.2.0
    2.9.3-foss-2018b-Java-1.8



NLopt
----- 



**Description** 


NLopt is a free/open-source library for nonlinear optimization, providing a common interface for a number of different free optimization routines available online as well as original implementations of various other algorithms.  


**More Information** 


http://ab-initio.mit.edu/wiki/index.php/NLopt 


**Available Versions**:: 


    2.7.0-GCCcore-11.2.0
    2.6.1-GCCcore-8.3.0
    2.4.2-GCCcore-7.3.0
    2.7.0-GCCcore-10.3.0
    2.6.1-GCCcore-9.3.0
    2.6.2-GCCcore-10.2.0



NSPR
---- 



**Description** 


Netscape Portable Runtime (NSPR) provides a platform-neutral API for system level and libc-like functions. 


**More Information** 


https://developer.mozilla.org/en-US/docs/Mozilla/Projects/NSPR 


**Available Versions**:: 


    4.30-GCCcore-10.3.0
    4.29-GCCcore-10.2.0
    4.25-GCCcore-9.3.0
    4.21-GCCcore-8.3.0
    4.20-GCCcore-7.3.0



NSS
--- 



**Description** 


Network Security Services (NSS) is a set of libraries designed to support cross-platform development of security-enabled client and server applications. 


**More Information** 


https://developer.mozilla.org/en-US/docs/Mozilla/Projects/NSS 


**Available Versions**:: 


    3.39-GCCcore-7.3.0
    3.45-GCCcore-8.3.0
    3.51-GCCcore-9.3.0
    3.65-GCCcore-10.3.0
    3.57-GCCcore-10.2.0



NVHPC
----- 



**Description** 


C, C++ and Fortran compilers included with the NVIDIA HPC SDK (previously: PGI) 


**More Information** 


https://developer.nvidia.com/hpc-sdk/ 


**Available Versions**:: 


    21.9
    20.11
    21.11
    21.7



NWChem
------ 



**Description** 


NWChem aims to provide its users with computational chemistry tools that are scalable both in their ability to treat large scientific computational chemistry problems efficiently, and in their use of available parallel computing resources from high-performance parallel supercomputers to conventional workstation clusters. NWChem software can handle: biomolecules, nanostructures, and solid-state; from quantum to classical, and all combinations; Gaussian basis functions or plane-waves; scaling from one to thousands of processors; properties and relativity. 


**More Information** 


http://www.nwchem-sw.org 


**Available Versions**:: 


    7.0.0-foss-2019b-Python-3.7.4
    7.0.2-intel-2021a



NetLogo
------- 



**Description** 


NetLogo is a multi-agent programmable modeling environment. Itis used by tens of thousands of students, teachers and researchers worldwide.It also powers HubNet participatory simulations. It is authored by Uri Wilenskyand developed at the CCL. 


**More Information** 


http://ccl.northwestern.edu/netlogo/ 


**Available Versions**:: 


    6.0.4-64



NgsRelate
--------- 



**Available Versions**:: 


    2022



Ninja
----- 



**Description** 


Ninja is a small build system with a focus on speed. 


**More Information** 


https://ninja-build.org/ 


**Available Versions**:: 


    1.10.1-GCCcore-10.2.0
    1.10.2-GCCcore-11.2.0
    1.9.0-GCCcore-8.2.0
    1.10.2-GCCcore-10.3.0
    1.10.0-GCCcore-9.3.0
    1.9.0-GCCcore-8.3.0



OPARI2
------ 



**Description** 


OPARI2, the successor of Forschungszentrum Juelich's OPARI, is a source-to-source instrumentation tool for OpenMP and hybrid codes. It surrounds OpenMP directives and runtime library calls with calls to the POMP2 measurement interface. 


**More Information** 


https://www.score-p.org 


**Available Versions**:: 


    2.0.5-GCCcore-9.3.0



ORCA
---- 



**Description** 


ORCA is a flexible, efficient and easy-to-use general purpose tool for quantum chemistry with specific emphasis on spectroscopic properties of open-shell molecules. It features a wide variety of standard quantum chemical methods ranging from semiempirical methods to DFT to single- and multireference correlated ab initio methods. It can also treat environmental and relativistic effects. 


**More Information** 


https://orcaforum.kofo.mpg.de 


**Available Versions**:: 


    5.0.0-gompi-2019b
    5.0.1-gompi-2019b-dba
    4.2.1-gompi-2019b
    5.0.3-gompi-2021b
    5.0.2-gompi-2019b-dba



OSU-Micro-Benchmarks
-------------------- 



**Description** 


OSU Micro-Benchmarks 


**More Information** 


https://mvapich.cse.ohio-state.edu/benchmarks/ 


**Available Versions**:: 


    5.8-fosscuda-2020b



OTF2
---- 



**Description** 


The Open Trace Format 2 is a highly scalable, memory efficient event trace data format plus support library. It is the new standard trace format for Scalasca, Vampir, and TAU and is open for other tools. 


**More Information** 


https://www.score-p.org 


**Available Versions**:: 


    2.2-GCCcore-9.3.0



Octave
------ 



**Description** 


GNU Octave is a high-level interpreted language, primarily intended for numerical computations. 


**More Information** 


https://www.gnu.org/software/octave/ 


**Available Versions**:: 


    5.1.0-foss-2019b



OpenBLAS
-------- 



**Description** 


OpenBLAS is an optimized BLAS library based on GotoBLAS2 1.13 BSD version. 


**More Information** 


http://xianyi.github.com/OpenBLAS/ 


**Available Versions**:: 


    0.3.1-GCC-7.3.0-2.30
    0.3.5-GCC-8.2.0-2.31.1
    0.3.20-GCC-11.3.0
    0.3.18-GCC-11.2.0
    0.3.9-GCC-9.3.0
    0.3.7-GCC-8.3.0
    0.3.15-GCC-10.3.0
    0.3.12-GCC-10.2.0



OpenBabel
--------- 



**Description** 


Open Babel is a chemical toolbox designed to speak the many languages of chemical data. It's an open, collaborative project allowing anyone to search, convert, analyze, or store data from molecular modeling, chemistry, solid-state materials, biochemistry, or related areas. 


**More Information** 


https://openbabel.org 


**Available Versions**:: 


    3.1.1-gompi-2019b-Python-3.7.4



OpenCV
------ 



**Description** 


OpenCV (Open Source Computer Vision Library) is an open source computer vision and machine learning software library. OpenCV was built to provide a common infrastructure for computer vision applications and to accelerate the use of machine perception in the commercial products. 


**More Information** 


https://opencv.org/ 


**Available Versions**:: 


    3.4.7-foss-2019a-Python-3.7.2
    4.2.0-foss-2020a-Python-3.8.2



OpenColorIO
----------- 



**Description** 


OpenColorIO (OCIO) is a complete color management solution geared towards motion picture production with an emphasis on visual effects and computer animation. 


**More Information** 


http://opencolorio.org/ 


**Available Versions**:: 


    1.1.0-foss-2018b



OpenEXR
------- 



**Description** 


OpenEXR is a high dynamic-range (HDR) image file format developed by Industrial Light & Magic for use in computer imaging applications 


**More Information** 


https://www.openexr.com/ 


**Available Versions**:: 


    2.5.5-GCCcore-10.2.0
    2.4.1-GCCcore-9.3.0
    2.4.0-GCCcore-8.3.0



OpenFOAM
-------- 



**Description** 


OpenFOAM is a free, open source CFD software package. OpenFOAM has an extensive range of features to solve anything from complex fluid flows involving chemical reactions, turbulence and heat transfer, to solid dynamics and electromagnetics. 


**More Information** 


https://www.openfoam.org/ 


**Available Versions**:: 


    6-intel-2020a
    8-foss-2020b
    v2106-foss-2021a
    v2006-intel-2020a
    9-foss-2021a
    8-foss-2020a
    v2006-foss-2020a
    6-foss-2018b
    v1912-foss-2019b
    v2006-foss-2019b
    v2012-foss-2020a
    6-foss-2019b
    5.0-20180108-foss-2018b



OpenFOAM-ESI
------------ 



**Available Versions**:: 


    v2006-foss-2020a



OpenImageIO
----------- 



**Description** 


OpenImageIO is a library for reading and writing images, and a bunch of related classes, utilities, and applications. 


**More Information** 


https://openimageio.org/ 


**Available Versions**:: 


    2.1.12.0-gompi-2020a
    2.0.12-gompi-2019b



OpenMM
------ 



**Description** 


OpenMM is a toolkit for molecular simulation. 


**More Information** 


https://openmm.org 


**Available Versions**:: 


    7.5.1-fosscuda-2020b
    7.4.1-foss-2019b-Python-3.7.4
    7.4.2-intel-2020a-Python-3.8.2



OpenMPI
------- 



**Description** 


The Open MPI Project is an open source MPI-3 implementation. 


**More Information** 


https://www.open-mpi.org/ 


**Available Versions**:: 


    3.1.4-GCC-8.3.0
    3.1.3-GCC-8.2.0-2.31.1
    3.1.4-GCC-8.3.0-dba
    4.0.3-gcccuda-2020a
    4.0.5-gcccuda-2020b
    4.1.1-GCC-10.3.0
    3.1.4-gcccuda-2019b
    4.1.1-GCC-10.3.0-CUDA-11.1.1
    3.1.1-GCC-7.3.0-2.30
    4.0.5-iccifort-2020.4.304
    4.0.5-GCC-10.2.0
    4.1.1-GCC-11.2.0
    3.1.4-PGI-19.10-GCC-8.3.0-2.32
    3.1.1-gcccuda-2018b
    4.1.4-GCC-11.3.0
    4.0.3-GCC-9.3.0



OpenMolcas
---------- 



**Description** 


OpenMolcas is a quantum chemistry software package 


**More Information** 


https://gitlab.com/Molcas/OpenMolcas 


**Available Versions**:: 


    21.06-intel-2021a
    20.10-intel-2020a-Python-3.8.2
    18.09-intel-2020b-Python-3.6.6



OpenPGM
------- 



**Description** 


OpenPGM is an open source implementation of the Pragmatic General Multicast (PGM) specification in RFC 3208 available at www.ietf.org. PGM is a reliable and scalable multicast protocol that enables receivers to detect loss, request retransmission of lost data, or notify an application of unrecoverable loss. PGM is a receiver-reliable protocol, which means the receiver is responsible for ensuring all data is received, absolving the sender of reception responsibility. 


**More Information** 


https://code.google.com/p/openpgm/ 


**Available Versions**:: 


    5.2.122-GCCcore-11.2.0
    5.2.122-GCCcore-10.3.0
    5.2.122-GCCcore-8.3.0
    5.2.122-GCCcore-10.2.0
    5.2.122-GCCcore-9.3.0



OpenSSL
------- 



**Description** 


The OpenSSL Project is a collaborative effort to develop a robust, commercial-grade, full-featured, and Open Source toolchain implementing the Secure Sockets Layer (SSL v2/v3) and Transport Layer Security (TLS v1)  protocols as well as a full-strength general purpose cryptography library.  


**More Information** 


https://www.openssl.org/ 


**Available Versions**:: 


    1.1
    1.1.1h-GCCcore-10.2.0



OpenSees
-------- 



**Description** 


Open System for Earthquake Engineering Simulation 


**More Information** 


https://opensees.berkeley.edu/index.php 


**Available Versions**:: 


    3.2.0-intel-2020a-Python-3.8.2



OptaDOS
------- 



**Available Versions**:: 


    v2018-foss-2018b



OrthoFinder
----------- 



**Description** 


OrthoFinder is a fast, accurate and comprehensive platform for comparative genomics 


**More Information** 


https://github.com/davidemms/OrthoFinder 


**Available Versions**:: 


    2.3.8-foss-2019b-Python-2.7.16
    2.3.8-foss-2019b-Python-3.7.4



PAPI
---- 



**Description** 


PAPI provides the tool designer and application engineer with a consistent interface and methodology for use of the performance counter hardware found in most major microprocessors. PAPI enables software engineers to see, in near real time, the relation between software performance and processor events. In addition Component PAPI provides access to a collection of components that expose performance measurement opportunites across the hardware and software stack. 


**More Information** 


https://icl.cs.utk.edu/projects/papi/ 


**Available Versions**:: 


    6.0.0-GCCcore-9.3.0
    6.0.0-GCCcore-10.2.0



PAUP
---- 



**Description** 


PAUP* (Phylogenetic Analysis Using Parsimony *and other methods) is acomputational phylogenetics program for inferring evolutionary trees. 


**More Information** 


https://paup.phylosolutions.com/ 


**Available Versions**:: 


    4.0a168-centos64



PCMSolver
--------- 



**Description** 


An API for the Polarizable Continuum Model. 


**More Information** 


https://pcmsolver.readthedocs.org 


**Available Versions**:: 


    1.2.3-gompi-2019a-Python-3.7.2



PCRE
---- 



**Description** 


The PCRE library is a set of functions that implement regular expression pattern matching using the same syntax and semantics as Perl 5. 


**More Information** 


https://www.pcre.org/ 


**Available Versions**:: 


    8.44-GCCcore-10.3.0
    8.43-GCCcore-8.2.0
    8.44-GCCcore-10.2.0
    8.44-GCCcore-9.3.0
    8.41-GCCcore-7.3.0
    8.43-GCCcore-8.3.0
    8.45-GCCcore-11.2.0



PCRE2
----- 



**Description** 


The PCRE library is a set of functions that implement regular expression pattern matching using the same syntax and semantics as Perl 5. 


**More Information** 


https://www.pcre.org/ 


**Available Versions**:: 


    10.33-GCCcore-8.3.0
    10.37-GCCcore-11.2.0
    10.36-GCCcore-10.3.0
    10.34-GCCcore-9.3.0
    10.35-GCCcore-10.2.0



PDT
--- 



**Description** 


Program Database Toolkit (PDT) is a framework for analyzing source code written in several programming languages and for making rich program knowledge accessible to developers of static and dynamic analysis tools. PDT implements a standard program representation, the program database (PDB), that can be accessed in a uniform way through a class library supporting common PDB operations. 


**More Information** 


https://www.cs.uoregon.edu/research/pdt/ 


**Available Versions**:: 


    3.25.1-GCCcore-9.3.0



PETSc
----- 



**Description** 


PETSc, pronounced PET-see (the S is silent), is a suite of data structures and routines for the scalable (parallel) solution of scientific applications modeled by partial differential equations. 


**More Information** 


https://www.mcs.anl.gov/petsc 


**Available Versions**:: 


    3.12.4-foss-2020a-Python-3.8.2
    3.15.1-foss-2021a
    3.12.4-foss-2019b-Python-3.7.4
    3.14.4-foss-2020b
    3.15.0-foss-2020b
    3.12.4-intel-2020a-Python-3.8.2



PGI
--- 



**Description** 


C, C++ and Fortran compilers from The Portland Group - PGI 


**More Information** 


https://www.pgroup.com/ 


**Available Versions**:: 


    18.10-GCC-8.1.0-2.30
    19.10-GCC-8.3.0-2.32
    16.5-GCC-7.3.0-2.30
    18.10-GCC-7.3.0-2.30



PLINK
----- 



**Description** 


Whole-genome association analysis toolset 


**More Information** 


https://www.cog-genomics.org/plink/2.0/ 


**Available Versions**:: 


    2.00a2.3_x86_64



PLINKSEQ
-------- 



**Description** 


PLINK/SEQ is an open-source C/C++ library for working with human  genetic variation data. The specific focus is to provide a platform for analytic tool  development for variation data from large-scale resequencing and genotyping projects,  particularly whole-exome and whole-genome studies. It is independent of (but designed  to be complementary to) the existing PLINK package.  


**More Information** 


https://atgu.mgh.harvard.edu/plinkseq/ 


**Available Versions**:: 


    0.10-GCCcore-7.3.0



PLUMED
------ 



**Description** 


PLUMED is an open source library for free energy calculations in molecular systems which works together with some of the most popular molecular dynamics engines. Free energy calculations can be performed as a function of many order parameters with a particular  focus on biological problems, using state of the art methods such as metadynamics, umbrella sampling and Jarzynski-equation based steered MD. The software, written in C++, can be easily interfaced with both fortran and C/C++ codes. 


**More Information** 


https://www.plumed.org 


**Available Versions**:: 


    2.6.0-intel-2020a-Python-3.8.2
    2.6.2-foss-2020b
    2.5.1-foss-2019b
    2.7.0-foss-2020a
    2.7.0-foss-2020b
    2.6.0-foss-2020a-Python-3.8.2
    2.6.2-intel-2020b
    2.7.2-foss-2021a
    2.7.0-intel-2020b
    2.5.1-foss-2019a
    2.5.1-foss-2020a
    2.6.2-foss-2020a-Python-3.8.2



PLY
--- 



**Description** 


PLY is yet another implementation of lex and yacc for Python. 


**More Information** 


https://www.dabeaz.com/ply/ 


**Available Versions**:: 


    3.11-GCCcore-8.3.0-Python-3.7.4



PMIx
---- 



**Description** 


Process Management for Exascale EnvironmentsPMI Exascale (PMIx) represents an attempt toprovide an extended version of the PMI standard specifically designedto support clusters up to and including exascale sizes. The overallobjective of the project is not to branch the existing pseudo-standarddefinitions - in fact, PMIx fully supports both of the existing PMI-1and PMI-2 APIs - but rather to (a) augment and extend those APIs toeliminate some current restrictions that impact scalability, and (b)provide a reference implementation of the PMI-server that demonstratesthe desired level of scalability. 


**More Information** 


https://pmix.org/ 


**Available Versions**:: 


    4.1.0-GCCcore-11.2.0
    3.1.5-GCCcore-9.3.0
    4.1.2-GCCcore-11.3.0
    3.1.5-GCCcore-10.2.0
    3.2.3-GCCcore-10.3.0



POV-Ray
------- 



**Description** 


The Persistence of Vision Raytracer, or POV-Ray, is a ray tracing program which generates images from a text-based scene description, and is available for a variety of computer platforms. POV-Ray is a high-quality, Free Software tool for creating stunning three-dimensional graphics. The source code is available for those wanting to do their own ports. 


**More Information** 


https://www.povray.org/ 


**Available Versions**:: 


    3.7.0.8-GCC-10.2.0



PRANK
----- 



**Description** 


PRANK is a probabilistic multiple alignment program for DNA,  codon and amino-acid sequences. PRANK is based on a novel algorithm that treats  insertions correctly and avoids over-estimation of the number of deletion events. 


**More Information** 


http://wasabiapp.org/software/prank/ 


**Available Versions**:: 


    170427-GCC-9.3.0



PROJ
---- 



**Description** 


Program proj is a standard Unix filter function which convertsgeographic longitude and latitude coordinates into cartesian coordinates 


**More Information** 


https://proj.org 


**Available Versions**:: 


    7.2.1-GCCcore-10.2.0
    7.0.0-GCCcore-9.3.0
    8.1.0-GCCcore-11.2.0
    6.2.1-GCCcore-8.3.0
    8.0.1-GCCcore-10.3.0
    6.0.0-GCCcore-8.2.0
    5.0.0-foss-2018b



PSI4
---- 



**Description** 


PSI4 is an open-source suite of ab initio quantum chemistry programs designed for efficient, high-accuracy simulations of a variety of molecular properties. We can routinely perform computations with more than 2500 basis functions running serially or in parallel. 


**More Information** 


http://www.psicode.org/ 


**Available Versions**:: 


    1.3.1-foss-2019a-Python-3.7.2



Pango
----- 



**Description** 


Pango is a library for laying out and rendering of text, with an emphasis on internationalization.Pango can be used anywhere that text layout is needed, though most of the work on Pango so far has been done in thecontext of the GTK+ widget toolkit. Pango forms the core of text and font handling for GTK+-2.x. 


**More Information** 


https://www.pango.org/ 


**Available Versions**:: 


    1.43.0-GCCcore-8.2.0
    1.47.0-GCCcore-10.2.0
    1.42.4-foss-2018b
    1.44.7-GCCcore-8.3.0
    1.48.5-GCCcore-10.3.0
    1.44.7-GCCcore-9.3.0



ParMETIS
-------- 



**Description** 


ParMETIS is an MPI-based parallel library that implements a variety of algorithms for partitioning unstructured graphs, meshes, and for computing fill-reducing orderings of sparse matrices. ParMETIS extends the functionality provided by METIS and includes routines that are especially suited for parallel AMR computations and large scale numerical simulations. The algorithms implemented in ParMETIS are based on the parallel multilevel k-way graph-partitioning, adaptive repartitioning, and parallel multi-constrained partitioning schemes. 


**More Information** 


http://glaros.dtc.umn.edu/gkhome/metis/parmetis/overview 


**Available Versions**:: 


    4.0.3-gompi-2019b
    4.0.3-gompi-2019a
    4.0.3-iimpi-2020a
    4.0.3-gompi-2020a
    4.0.3-gompi-2020b



ParaView
-------- 



**Description** 


ParaView is a scientific parallel visualizer. 


**More Information** 


https://www.paraview.org 


**Available Versions**:: 


    5.9.1-foss-2021a-mpi
    5.8.0-intel-2020a-Python-3.8.2-mpi
    5.6.2-foss-2019b-Python-3.7.4-mpi
    5.8.0-foss-2020a-Python-3.8.2-mpi
    5.8.1-foss-2020b-mpi



PennCNV
------- 



**Description** 


A free software tool for Copy Number Variation (CNV) detection from SNP genotyping arrays. Currently it can handle signal intensity data from Illumina and Affymetrix arrays. With appropriate preparation of file format, it can also handle other types of SNP arrays and oligonucleotide arrays. 


**More Information** 


https://penncnv.openbioinformatics.org/ 


**Available Versions**:: 


    1.0.5-GCCcore-8.3.0



Perl
---- 



**Description** 


Larry Wall's Practical Extraction and Report LanguageThis is a minimal build without any modules. Should only be used for build dependencies. 


**More Information** 


https://www.perl.org/ 


**Available Versions**:: 


    5.32.0-GCCcore-10.2.0-minimal
    5.32.1-GCCcore-10.3.0
    5.34.0-GCCcore-11.2.0
    5.34.0-GCCcore-11.2.0-ARC
    5.28.1-GCCcore-8.2.0
    5.28.0-GCCcore-10.2.0
    5.32.1-GCCcore-10.3.0-minimal
    5.30.2-GCCcore-9.3.0
    5.28.0-GCCcore-7.3.0
    5.28.0-GCCcore-8.1.0
    5.34.1-GCCcore-11.3.0
    5.30.0-GCCcore-8.3.0
    5.28.1-GCCcore-8.3.0
    5.30.0-GCCcore-7.3.0
    5.32.0-GCCcore-10.2.0



PhyloBayes-MPI
-------------- 



**Description** 


A Bayesian software for phylogenetic reconstruction using mixture models 


**More Information** 


https://github.com/bayesiancook/pbmpi 


**Available Versions**:: 


    20161021-intel-2020b



Pillow
------ 



**Description** 


Pillow is the 'friendly PIL fork' by Alex Clark and Contributors. PIL is the Python Imaging Library by Fredrik Lundh and Contributors. 


**More Information** 


https://pillow.readthedocs.org/ 


**Available Versions**:: 


    8.0.1-GCCcore-10.2.0
    6.0.0-GCCcore-8.2.0
    6.2.1-GCCcore-8.3.0
    8.2.0-GCCcore-10.3.0
    7.0.0-GCCcore-9.3.0-Python-3.8.2



Pillow-SIMD
----------- 



**Description** 


Pillow is the 'friendly PIL fork' by Alex Clark and Contributors. PIL is the Python Imaging Library by Fredrik Lundh and Contributors. 


**More Information** 


https://github.com/uploadcare/pillow-simd 


**Available Versions**:: 


    6.0.x.post0-GCCcore-8.2.0



Pilon
----- 



**Description** 


Pilon is an automated genome assembly improvement and variant detection tool 


**More Information** 


https://github.com/broadinstitute/pilon 


**Available Versions**:: 


    1.23-Java-1.8



Platypus
-------- 



**Description** 


Platypus is a tool designed for efficient and accurate variant-detection# in high-throughput sequencing data. 


**More Information** 


https://vcru.wisc.edu/simonlab/bioinformatics/programs/install/platypus.htm 


**Available Versions**:: 


    latest-GCC-11.2.0-Python-2.7.18



Pmw
--- 



**Description** 


Pmw is a toolkit for building high-level compound widgets in Python using the Tkinter module.  


**More Information** 


http://pmw.sourceforge.net 


**Available Versions**:: 


    2.0.1-foss-2020a-Python-3.8.2
    2.0.1-foss-2019b-Python-2.7.16



PnetCDF
------- 



**Description** 


Parallel netCDF: A Parallel I/O Library for NetCDF File Access 


**More Information** 


https://trac.mcs.anl.gov/projects/parallel-netcdf 


**Available Versions**:: 


    1.12.2-gompic-2020b
    1.12.2-gompi-2020b
    1.12.1-gompi-2020a



PostgreSQL
---------- 



**Description** 


PostgreSQL is a powerful, open source object-relational database system. It is fully ACID compliant, has full support for foreign keys, joins, views, triggers, and stored procedures (in multiple languages). It includes most SQL:2008 data types, including INTEGER, NUMERIC, BOOLEAN, CHAR, VARCHAR, DATE, INTERVAL, and TIMESTAMP. It also supports storage of binary large objects, including pictures, sounds, or video. It has native programming interfaces for C/C++, Java, .Net, Perl, Python, Ruby, Tcl, ODBC, among others, and exceptional documentation. 


**More Information** 


https://www.postgresql.org/ 


**Available Versions**:: 


    13.3-GCCcore-10.3.0
    13.4-GCCcore-11.2.0



PostgresQL
---------- 



**Description** 


PostgreSQL is a powerful, open source object-relational database system. It is fully ACID compliant, has full support for foreign keys, joins, views, triggers, and stored procedures (in multiple languages). It includes most SQL:2008 data types, including INTEGER, NUMERIC, BOOLEAN, CHAR, VARCHAR, DATE, INTERVAL, and TIMESTAMP. It also supports storage of binary large objects, including pictures, sounds, or video. It has native programming interfaces for C/C++, Java, .Net, Perl, Python, Ruby, Tcl, ODBC, among others, and exceptional documentation. 


**More Information** 


https://www.postgresql.org/ 


**Available Versions**:: 


    13.4-GCCcore-11.2.0



PyCairo
------- 



**Description** 


Python bindings for the cairo library 


**More Information** 


http://cairographics.org/pycairo/ 


**Available Versions**:: 


    1.18.0-foss-2018b-Python-2.7.15



PyCifRW
------- 



**Description** 


PyCIFRW provides support for reading and writing CIF(Crystallographic Information Format) files using Python. 


**More Information** 


https://bitbucket.org/jamesrhester/pycifrw/src/development 


**Available Versions**:: 


    4.4.2-GCCcore-9.3.0



PyGObject
--------- 



**Description** 


Python Bindings for GLib/GObject/GIO/GTK+ 


**More Information** 


http://www.pygtk.org/ 


**Available Versions**:: 


    2.28.7-foss-2018b-Python-2.7.15



PyGTK
----- 



**Description** 


PyGTK lets you to easily create programs with a graphical user interface using the Python programming language. 


**More Information** 


http://www.pygtk.org/ 


**Available Versions**:: 


    2.24.0-foss-2018b-Python-2.7.15



PyMOL
----- 



**Description** 


PyMOL is an OpenGL based molecular visualization system 


**More Information** 


http://sourceforge.net/projects/pymol 


**Available Versions**:: 


    2.3.0-foss-2019b-Python-2.7.16
    2.5.0-foss-2020a-Python-3.8.2



PyQt5
----- 



**Description** 


PyQt5 is a set of Python bindings for v5 of the Qt application framework from The Qt Company.This bundle includes PyQtWebEngine, a set of Python bindings for The Qt Company’s Qt WebEngine framework. 


**More Information** 


https://www.riverbankcomputing.com/software/pyqt 


**Available Versions**:: 


    5.15.1-GCCcore-9.3.0-Python-3.8.2



PySCF
----- 



**Description** 


PySCF is an open-source collection of electronic structure modules powered by Python. 


**More Information** 


http://www.pyscf.org 


**Available Versions**:: 


    1.7.6-foss-2021a
    1.7.0-foss-2020a-Python-3.8.2
    1.6.3-foss-2019a-Python-3.7.2



PyTorch
------- 



**Description** 


Tensors and Dynamic neural networks in Python with strong GPU acceleration.PyTorch is a deep learning framework that puts Python first. 


**More Information** 


https://pytorch.org/ 


**Available Versions**:: 


    1.7.1-fosscuda-2020b
    1.6.0-fosscuda-2019b-Python-3.7.4
    1.6.0-foss-2019b-Python-3.7.4



PyYAML
------ 



**Description** 


PyYAML is a YAML parser and emitter for the Python programming language. 


**More Information** 


https://github.com/yaml/pyyaml 


**Available Versions**:: 


    5.3.1-GCCcore-10.2.0
    5.3-GCCcore-9.3.0
    5.1-GCCcore-8.2.0
    5.1.2-GCCcore-8.3.0



Pysam
----- 



**Description** 


Pysam is a python module for reading and manipulating Samfiles. It's a lightweight wrapper of the samtools C-API. Pysam also includes an interface for tabix. 


**More Information** 


https://github.com/pysam-developers/pysam 


**Available Versions**:: 


    0.15.3-GCC-8.3.0
    0.16.0.1-GCC-9.3.0



Python
------ 



**Description** 


Python is a programming language that lets you work more quickly and integrate your systems more effectively. 


**More Information** 


http://python.org/ 


**Available Versions**:: 


    3.6.6-intel-2020b
    3.9.6-GCCcore-11.2.0
    3.7.4-GCCcore-8.3.0
    2.7.16-GCCcore-8.3.0
    3.6.6-foss-2018b
    3.7.2-GCCcore-8.2.0
    2.7.18-GCCcore-11.2.0
    3.9.5-GCCcore-10.3.0
    2.7.18-GCCcore-10.2.0
    2.7.15-foss-2018b
    2.7.15-GCCcore-8.2.0
    2.7.18-GCCcore-10.3.0-bare
    2.7.18-GCCcore-9.3.0
    2.7.15-fosscuda-2018b
    3.10.4-GCCcore-11.3.0-bare
    2.7.15-GCCcore-7.3.0-bare
    3.9.5-GCCcore-10.3.0-bare
    3.8.6-GCCcore-10.2.0
    3.9.6-GCCcore-11.2.0-bare
    3.8.2-GCCcore-9.3.0



QIIME2
------ 



**Description** 


QIIME is an open-source bioinformatics pipeline for performing microbiome analysis from raw DNA sequencing data. 


**More Information** 


http://qiime2.org/ 


**Available Versions**:: 


    2019.7



QUAST
----- 



**Description** 


QUAST evaluates genome assemblies by computing various metrics. It works both with and without reference genomes. The tool accepts multiple assemblies, thus is suitable for comparison. 


**More Information** 


https://github.com/ablab/quast 


**Available Versions**:: 


    5.0.2-foss-2020a-Python-3.8.2



Qhull
----- 



**Description** 


Qhull computes the convex hull, Delaunay triangulation, Voronoi diagram, halfspace intersection about a point, furthest-site Delaunay triangulation, and furthest-site Voronoi diagram. The source code runs in 2-d, 3-d, 4-d, and higher dimensions. Qhull implements the Quickhull algorithm for computing the convex hull. 


**More Information** 


http://www.qhull.org 


**Available Versions**:: 


    2020.2-GCCcore-10.3.0
    2019.1-GCCcore-8.3.0



Qiskit
------ 



**Description** 


Qiskit is an open-source framework for working with noisy quantum computers at the level of pulses, circuits, and algorithms. 


**More Information** 


https://qiskit.org 


**Available Versions**:: 


    0.23.1-foss-2020a-Python-3.8.2



Qt5
--- 



**Description** 


Qt is a comprehensive cross-platform C++ application framework. 


**More Information** 


https://qt.io/ 


**Available Versions**:: 


    5.14.1-GCCcore-9.3.0
    5.15.2-GCCcore-10.3.0
    5.14.2-GCCcore-10.2.0
    5.13.1-GCCcore-8.3.0



Quandl
------ 



**Description** 


A Python library for Quandl’s RESTful API. 


**More Information** 


https://pypi.python.org/pypi/Quandl 


**Available Versions**:: 


    3.4.8-foss-2019a
    3.5.3-foss-2020a-Python-3.8.2
    3.6.1-foss-2021a



QuantumESPRESSO
--------------- 



**Description** 


Quantum ESPRESSO  is an integrated suite of computer codes for electronic-structure calculations and materials modeling at the nanoscale. It is based on density-functional theory, plane waves, and pseudopotentials  (both norm-conserving and ultrasoft). 


**More Information** 


https://www.quantum-espresso.org 


**Available Versions**:: 


    6.6-foss-2020b
    6.7-intel-2020a
    6.5-intel-2020a
    6.6-intel-2020a
    6.7-foss-2020b



Qwt
--- 



**Description** 


The Qwt library contains GUI Components and utility classes which are primarily useful for programs with a technical background. 


**More Information** 


https://qwt.sourceforge.net/ 


**Available Versions**:: 


    6.1.4-GCCcore-8.3.0



R
- 



**Description** 


R is a free software environment for statistical computing and graphics. 


**More Information** 


https://www.r-project.org/ 


**Available Versions**:: 


    3.6.2-foss-2019b
    4.0.2-foss-2020a-ARC
    4.1.2-foss-2021b-ARC
    4.0.2-foss-2020a
    4.1.2-foss-2021b
    4.1.0-foss-2021a
    4.1.0-foss-2021a-ARC



R-bundle-Bioconductor
--------------------- 



**Description** 


Bioconductor provides tools for the analysis and coprehension of high-throughput genomic data. 


**More Information** 


https://bioconductor.org 


**Available Versions**:: 


    3.10-foss-2019b
    3.14-foss-2021b-R-4.1.2
    3.13-foss-2021a-R-4.1.0



RAxML
----- 



**Description** 


RAxML search algorithm for maximum likelihood based inference of phylogenetic trees. 


**More Information** 


https://github.com/stamatak/standard-RAxML 


**Available Versions**:: 


    8.2.12-gompi-2020a-hybrid-avx2



RAxML-NG
-------- 



**Description** 


RAxML-NG is a phylogenetic tree inference tool whichuses maximum-likelihood (ML) optimality criterion. Its search heuristic is based on iteratively performing a series of Subtree Pruning and Regrafting (SPR) moves, which allows to quickly navigate to the best-known ML tree. 


**More Information** 


https://github.com/amkozlov/raxml-ng 


**Available Versions**:: 


    1.0.2-gompi-2020b



RDKit
----- 



**Description** 


RDKit is a collection of cheminformatics and machine-learning software written in C++ and Python. 


**More Information** 


https://www.rdkit.org 


**Available Versions**:: 


    2020.03.3-foss-2020a-Python-3.8.2



RE2
--- 



**Description** 


RE2 is a fast, safe, thread-friendly alternative to backtracking regularexpression engines like those used in PCRE, Perl, and Python. It is a C++library.  


**More Information** 


https://github.com/google/re2 


**Available Versions**:: 


    2020-07-01-GCCcore-8.3.0



RECON
----- 



**Description** 


Patched version of RECON to be used with RepeatModeler. 


**More Information** 


https://www.repeatmasker.org/RepeatModeler/ 


**Available Versions**:: 


    1.08-GCC-10.3.0
    1.08-GCC-10.2.0



RELION
------ 



**Description** 


RELION (for REgularised LIkelihood OptimisatioN, pronounce rely-on) is a stand-alone computer program that employs an empirical Bayesian approach to refinement of (multiple) 3D reconstructions or 2D class averages in electron cryo-microscopy (cryo-EM). 


**More Information** 


http://www2.mrc-lmb.cam.ac.uk/relion/index.php/Main_Page 


**Available Versions**:: 


    3.0.4-foss-2019b



RMBlast
------- 



**Description** 


RMBlast is a RepeatMasker compatible version of the standard NCBI BLAST suite. The primary difference between this distribution and the NCBI distribution is the addition of a new program 'rmblastn' for use with RepeatMasker and RepeatModeler. 


**More Information** 


https://www.repeatmasker.org/RMBlast.html 


**Available Versions**:: 


    2.11.0-gompi-2021a
    2.11.0-gompi-2020b



ROOT
---- 



**Description** 


The ROOT system provides a set of OO frameworks with all the functionality    needed to handle and analyze large amounts of data in a very efficient way. 


**More Information** 


https://root.cern.ch/drupal/ 


**Available Versions**:: 


    6.24.06-foss-2021b



RStudio
------- 



**Available Versions**:: 


    2022.02-R-4.2.1-ARC



RapidJSON
--------- 



**Description** 


A fast JSON parser/generator for C++ with both SAX/DOM style API 


**More Information** 


https://rapidjson.org 


**Available Versions**:: 


    1.1.0-GCCcore-8.3.0



Ray-assembler
------------- 



**Description** 


Parallel genome assemblies for parallel DNA sequencing 


**More Information** 


http://denovoassembler.sourceforge.net/ 


**Available Versions**:: 


    2.3.1-iimpi-2020a



RepeatMasker
------------ 



**Description** 


RepeatMasker is a program that screens DNA sequences for interspersed repeats and low complexity DNA sequences. 


**More Information** 


https://www.repeatmasker.org/ 


**Available Versions**:: 


    4.1.2-p1-foss-2021a
    4.1.2-p1-foss-2020b



RepeatModeler
------------- 



**Description** 


RepeatModeler is a de novo transposable element (TE) family identification and modeling package.  


**More Information** 


https://www.repeatmasker.org/ 


**Available Versions**:: 


    2.0.2a-foss-2020b
    2.0.2a-foss-2021a



RepeatScout
----------- 



**Description** 


De Novo Repeat Finder, Price A.L., Jones N.C. and Pevzner P.A. Developed and tested with our multiple sequence version of RepeatScout ( 1.0.6 ) 


**More Information** 


https://www.repeatmasker.org/ 


**Available Versions**:: 


    1.0.6-GCC-10.3.0
    1.0.6-GCC-10.2.0



RevBayes
-------- 



**Description** 


RevBayes provides an interactive environment for statistical computation in phylogenetics. It is primarily intended for modeling, simulation, and Bayesian inference in evolutionary biology, particularly phylogenetics. 


**More Information** 


https://revbayes.github.io/ 


**Available Versions**:: 


    1.1.1-GCC-10.2.0



Ruby
---- 



**Description** 


Ruby is a dynamic, open source programming language with a focus on simplicity and productivity. It has an elegant syntax that is natural to read and easy to write. 


**More Information** 


https://www.ruby-lang.org 


**Available Versions**:: 


    3.0.1-GCCcore-10.3.0
    2.6.1-GCCcore-7.3.0



Rust
---- 



**Description** 


Rust is a systems programming language that runs blazingly fast, prevents segfaults, and guarantees thread safety. 


**More Information** 


https://www.rust-lang.org 


**Available Versions**:: 


    1.42.0-GCCcore-8.3.0
    1.54.0-GCCcore-11.2.0
    1.52.1-GCCcore-10.3.0
    1.42.0-GCCcore-9.3.0



SAMtools
-------- 



**Description** 


SAM Tools provide various utilities for manipulating alignments in the SAM format,  including sorting, merging, indexing and generating alignments in a per-position format. 


**More Information** 


https://www.htslib.org/ 


**Available Versions**:: 


    0.1.20-GCC-8.3.0
    1.10-GCC-8.3.0
    1.11-GCC-10.2.0
    0.1.20-foss-2020a
    1.9-foss-2018b
    1.10-GCC-9.3.0
    1.14-GCC-11.2.0
    1.12-GCC-10.2.0



SCOTCH
------ 



**Description** 


Software package and libraries for sequential and parallel graph partitioning,static mapping, and sparse matrix block ordering, and sequential mesh and hypergraph partitioning. 


**More Information** 


http://gforge.inria.fr/projects/scotch/ 


**Available Versions**:: 


    6.0.6-foss-2018b
    6.0.6-gompi-2019a
    6.1.0-gompi-2021a
    6.1.0-gompi-2020b
    6.0.9-iimpi-2020a
    6.0.9-gompi-2019b
    6.0.9-gompi-2020a



SCons
----- 



**Description** 


SCons is a software construction tool. 


**More Information** 


http://www.scons.org/ 


**Available Versions**:: 


    3.0.1-fosscuda-2018b-Python-2.7.15
    4.0.1-GCCcore-10.2.0
    3.0.5-GCCcore-8.2.0
    3.1.1-GCCcore-8.3.0
    3.0.5-GCCcore-8.2.0-Python-2.7.15
    3.0.1-foss-2018b-Python-2.7.15



SDL2
---- 



**Description** 


SDL: Simple DirectMedia Layer, a cross-platform multimedia library 


**More Information** 


https://www.libsdl.org/ 


**Available Versions**:: 


    2.0.14-GCCcore-10.2.0



SIONlib
------- 



**Description** 


SIONlib is a scalable I/O library for the parallel access totask-local files. The library not only supports writing and readingbinary data to or from from several thousands of processors into asingle or a small number of physical files but also provides forglobal open and close functions to access SIONlib file inparallel. SIONlib provides different interfaces: parallel access usingMPI, OpenMp, or their combination and sequential access forpost-processing utilities.  


**More Information** 


http://www.fz-juelich.de/ias/jsc/EN/Expertise/Support/Software/SIONlib/_node.html 


**Available Versions**:: 


    1.7.6-foss-2020a
    1.7.6-foss-2019b
    1.7.6-GCCcore-9.3.0-tools



SKESA
----- 



**Description** 


SKESA is a de-novo sequence read assembler for cultured single isolate genomes based on DeBruijn graphs. 


**More Information** 


https://github.com/ncbi/SKESA 


**Available Versions**:: 


    2.3.0-foss-2018b



SLEPc
----- 



**Description** 


SLEPc (Scalable Library for Eigenvalue Problem Computations) is a software library for the solution of large scale sparse eigenvalue problems on parallel computers. It is an extension of PETSc and can be used for either standard or generalized eigenproblems, with real or complex arithmetic. It can also be used for computing a partial SVD of a large, sparse, rectangular matrix, and to solve quadratic eigenvalue problems. 


**More Information** 


https://slepc.upv.es/ 


**Available Versions**:: 


    3.12.2-foss-2019b-Python-3.7.4
    3.12.2-intel-2020a-Python-3.8.2



SNeP1
----- 



**Available Versions**:: 


    1



SOAPdenovo2
----------- 



**Description** 


SOAPdenovo is a novel short-read assembly method that can build a  de novo draft assembly for human-sized genomes. The program is specially designed to  assemble Illumina short reads. It creates new opportunities for building reference  sequences and carrying out accurate analyses of unexplored genomes in a cost effective way.  SOAPdenovo2 is the successor of SOAPdenovo. 


**More Information** 


http://soap.genomics.org.cn/index.html 


**Available Versions**:: 


    r241-foss-2020a



SOCI
---- 



**Description** 


SOCI is a database access library for C++ that makes the illusion of embedding SQL queries in the regular C++ code, staying entirely within the Standard C++. 


**More Information** 


http://soci.sourceforge.net/ 


**Available Versions**:: 


    4.0.2-GCC-10.3.0



SPAdes
------ 



**Description** 


Genome assembler for single-cell and isolates data sets 


**More Information** 


https://cab.spbu.ru/software/spades/ 


**Available Versions**:: 


    3.15.2-GCC-10.2.0
    3.13.0-foss-2018b



SQLite
------ 



**Description** 


SQLite: SQL Database Engine in a C Library 


**More Information** 


https://www.sqlite.org/ 


**Available Versions**:: 


    3.27.2-GCCcore-8.2.0
    3.35.4-GCCcore-10.3.0
    3.31.1-GCCcore-9.3.0
    3.38.3-GCCcore-11.3.0
    3.24.0-GCCcore-7.3.0
    3.36-GCCcore-11.2.0
    3.33.0-GCCcore-10.2.0
    3.29.0-GCCcore-8.3.0
    3.24.0-GCCcore-10.2.0
    3.27.2-GCCcore-8.3.0



STAR
---- 



**Description** 


STAR aligns RNA-seq reads to a reference genome using uncompressed suffix arrays. 


**More Information** 


https://github.com/alexdobin/STAR 


**Available Versions**:: 


    2.7.9a-GCC-11.2.0
    2.7.1a-GCC-8.2.0-2.31.1
    2.7.3a-GCC-9.3.0



STARCCM+
-------- 



**Available Versions**:: 


    2022.1
    2021.2.1
    2020.2.1



STEAK
----- 



**Description** 


Detects integrations of any sort in high-throughput sequencing (HTS) data. STEAK was built for validating and discovering transposable element (TE) and retroviral integrations in a variety of HTS data. The software performs on both single-end (SE) and paired-end (PE) libraries and on a variety of HTS sequencing strategies. It can be applied to a broad range of research interests and clinical uses such as population genetic studies and detecting polymorphic integrations. 


**More Information** 


https://omictools.com/steak-tool 


**Available Versions**:: 


    2019.09.12-foss-2019b-Python-2.7.16



SUNDIALS
-------- 



**Description** 


SUNDIALS: SUite of Nonlinear and DIfferential/ALgebraic Equation Solvers 


**More Information** 


https://computation.llnl.gov/projects/sundials 


**Available Versions**:: 


    5.1.0-foss-2019b



SWIG
---- 



**Description** 


SWIG is a software development tool that connects programs written in C and C++ with a variety of high-level programming languages. 


**More Information** 


http://www.swig.org/ 


**Available Versions**:: 


    3.0.12-GCCcore-8.2.0-Python-2.7.15
    4.0.1-GCCcore-8.3.0
    3.0.12-foss-2018b-Python-2.7.15
    4.0.2-GCCcore-10.2.0
    3.0.12-GCCcore-8.3.0
    4.0.2-GCCcore-10.3.0
    4.0.1-GCCcore-9.3.0



Salmon
------ 



**Description** 


Salmon is a wicked-fast program to produce a highly-accurate, transcript-level quantification estimates from RNA-seq data. 


**More Information** 


https://github.com/COMBINE-lab/salmon 


**Available Versions**:: 


    1.4.0-gompi-2020b



ScaFaCoS
-------- 



**Description** 


ScaFaCoS is a library of scalable fast coulomb solvers. 


**More Information** 


http://www.scafacos.de/ 


**Available Versions**:: 


    1.0.1-intel-2020a
    1.0.1-foss-2020a



ScaLAPACK
--------- 



**Description** 


The ScaLAPACK (or Scalable LAPACK) library includes a subset of LAPACK routines redesigned for distributed memory MIMD parallel computers. 


**More Information** 


https://www.netlib.org/scalapack/ 


**Available Versions**:: 


    2.1.0-gompi-2021b-fb
    2.0.2-gompic-2019b
    2.0.2-gompi-2018b-OpenBLAS-0.3.1
    2.0.2-gompi-2019a-OpenBLAS-0.3.5
    2.1.0-gompi-2020b
    2.1.0-gompi-2020a
    2.0.2-gompi-2019b
    2.1.0-gompic-2020b
    2.0.2-gompic-2018b-OpenBLAS-0.3.1
    2.1.0-gompic-2020a
    2.1.0-gompi-2021a-fb
    2.2.0-gompi-2022a-fb



Scalasca
-------- 



**Description** 


Scalasca is a software tool that supports the performance optimization of parallel programs by measuring and analyzing their runtime behavior. The analysis identifies potential performance bottlenecks -- in particular those concerning communication and synchronization -- and offers guidance in exploring their causes. 


**More Information** 


https://www.scalasca.org/ 


**Available Versions**:: 


    2.5-gompi-2020a



SciPy-bundle
------------ 



**Description** 


Bundle of Python packages for scientific software 


**More Information** 


http://python.org/ 


**Available Versions**:: 


    2019.03-foss-2019a
    2020.11-foss-2020b
    2019.10-fosscuda-2019b-Python-3.7.4
    2021.05-foss-2021a
    2019.10-intel-2019b-Python-3.7.4
    2020.03-intel-2020a-Python-3.8.2
    2019.10-foss-2019b-Python-2.7.16
    2020.11-intel-2020b
    2020.03-foss-2020a-Python-3.8.2
    2020.11-fosscuda-2020b
    2019.10-foss-2019b-Python-3.7.4
    2019.10-fosscuda-2019b-Python-2.7.16
    2021.10-foss-2021b
    2020.03-fosscuda-2020a-Python-3.8.2



Score-P
------- 



**Description** 


The Score-P measurement infrastructure is a highly scalable and easy-to-use tool suite for profiling, event tracing, and online analysis of HPC applications. 


**More Information** 


https://www.score-p.org 


**Available Versions**:: 


    6.0-gompi-2020a



Seaborn
------- 



**Description** 


Seaborn is a Python visualization library based on matplotlib. It provides a high-level interface for drawing attractive statistical graphics.  


**More Information** 


https://seaborn.pydata.org/ 


**Available Versions**:: 


    0.10.0-foss-2019b-Python-3.7.4



SentencePiece
------------- 



**Description** 


Unsupervised text tokenizer for Neural Network-based text generation. 


**More Information** 


https://github.com/google/sentencepiece 


**Available Versions**:: 


    0.1.85-GCC-8.3.0-Python-3.7.4



SeqKit
------ 



**Description** 


SeqKit - a cross-platform and ultrafast toolkit for FASTA/Q file manipulation 


**More Information** 


https://bioinf.shenwei.me/seqkit/ 


**Available Versions**:: 


    0.13.2



Serf
---- 



**Description** 


The serf library is a high performance C-based HTTP client library built upon the Apache Portable Runtime (APR) library 


**More Information** 


https://serf.apache.org/ 


**Available Versions**:: 


    1.3.9-GCCcore-10.2.0



Siesta
------ 



**Description** 


SIESTA is both a method and its computer program implementation, to perform efficient electronic structure calculations and ab initio molecular dynamics simulations of molecules and solids. 


**More Information** 


http://departments.icmab.es/leem/siesta 


**Available Versions**:: 


    4.1-b4-intel-2020a
    4.1.5-foss-2020a



Stacks
------ 



**Description** 


Stacks is a software pipeline for building loci from short-read sequences, such as those generated on the Illumina platform. Stacks was developed to work with restriction enzyme-based data, such as RAD-seq, for the purpose of building genetic maps and conducting population genomics and phylogeography. 


**More Information** 


http://catchenlab.life.illinois.edu/stacks 


**Available Versions**:: 


    2.41-GCC-8.3.0
    2.41-iccifort-2019.5.281
    2.53-foss-2019b



Stata
----- 



**Available Versions**:: 


    14



Subversion
---------- 



**Description** 


Subversion is an open source version control system. 


**More Information** 


https://subversion.apache.org/ 


**Available Versions**:: 


    1.14.0-GCCcore-10.2.0



SuiteSparse
----------- 



**Description** 


SuiteSparse is a collection of libraries manipulate sparse matrices. 


**More Information** 


https://faculty.cse.tamu.edu/davis/suitesparse.html 


**Available Versions**:: 


    5.7.1-intel-2020a-METIS-5.1.0
    5.6.0-foss-2019b-METIS-5.1.0
    5.10.1-foss-2021a-METIS-5.1.0
    5.4.0-foss-2019a-METIS-5.1.0
    5.8.1-foss-2020b-METIS-5.1.0
    5.7.1-foss-2020a-METIS-5.1.0



SuperLU
------- 



**Description** 


SuperLU is a general purpose library for the direct solution of large, sparse, nonsymmetric systems of linear equations on high performance machines. 


**More Information** 


https://crd-legacy.lbl.gov/~xiaoye/SuperLU/ 


**Available Versions**:: 


    5.2.2-intel-2020a



SymEngine
--------- 



**Description** 


SymEngine is a standalone fast C++ symbolic manipulation library 


**More Information** 


https://github.com/symengine/symengine 


**Available Versions**:: 


    0.7.0-GCC-10.3.0



SymEngine-python
---------------- 



**Description** 


Python wrappers to the C++ library SymEngine, a fast C++ symbolic manipulation library. 


**More Information** 


https://github.com/symengine/symengine.py 


**Available Versions**:: 


    0.7.2-GCC-10.3.0



Szip
---- 



**Description** 


Szip compression software, providing lossless compression of scientific data 


**More Information** 


https://www.hdfgroup.org/doc_resource/SZIP/ 


**Available Versions**:: 


    2.1.1-GCCcore-9.3.0
    2.1.1-intel-2020a
    2.1.1-GCCcore-7.3.0
    2.1.1-GCCcore-8.3.0
    2.1.1-GCCcore-10.2.0
    2.1.1-GCCcore-11.2.0
    2.1.1-GCCcore-10.3.0
    2.1.1-GCCcore-8.2.0
    2.1.1-GCCcore-11.3.0



TRF
--- 



**Description** 


Tandem Repeats Finder: a program to analyze DNA sequences. 


**More Information** 


https://tandem.bu.edu/trf/trf.html 


**Available Versions**:: 


    4.09.1-GCCcore-10.2.0
    4.09.1-GCCcore-10.3.0



TWL-NINJA
--------- 



**Description** 


Nearly Infinite Neighbor Joining Application. 


**More Information** 


https://github.com/TravisWheelerLab/NINJA 


**Available Versions**:: 


    0.97-cluster_only-GCC-10.3.0
    0.97-cluster_only-GCC-10.2.0



Tcl
--- 



**Description** 


Tcl (Tool Command Language) is a very powerful but easy to learn dynamic programming language, suitable for a very wide range of uses, including web and desktop applications, networking, administration, testing and many more. 


**More Information** 


https://www.tcl.tk/ 


**Available Versions**:: 


    8.6.12-GCCcore-11.3.0
    8.6.8-GCCcore-7.3.0
    8.6.8-GCCcore-10.2.0
    8.6.10-GCCcore-9.3.0
    8.6.11-GCCcore-11.2.0
    8.6.9-GCCcore-8.2.0
    8.6.10-GCCcore-10.2.0
    8.6.9-GCCcore-8.3.0
    8.6.11-GCCcore-10.3.0



TensorFlow
---------- 



**Description** 


An open-source software library for Machine Intelligence 


**More Information** 


https://www.tensorflow.org/ 


**Available Versions**:: 


    2.5.0-fosscuda-2020b
    2.3.1-foss-2020a-Python-3.8.2
    2.4.1-fosscuda-2020b
    2.6.0-foss-2021a-CUDA-11.3.1
    1.15.2-fosscuda-2019b-Python-3.7.4
    2.1.0-foss-2019b-Python-3.7.4
    2.7.1-foss-2021b-CUDA-11.4.1
    2.5.3-foss-2021a-CUDA-11.3.1



Theano
------ 



**Description** 


Theano is a Python library that allows you to define, optimize,and evaluate mathematical expressions involving multi-dimensional arrays efficiently. 


**More Information** 


https://deeplearning.net/software/theano 


**Available Versions**:: 


    1.0.4-foss-2019b-Python-3.7.4
    1.0.4-fosscuda-2019b-Python-3.7.4
    1.1.2-fosscuda-2020b-PyMC



Tk
-- 



**Description** 


Tk is an open source, cross-platform widget toolchain that provides a library of basic elements for building a graphical user interface (GUI) in many different programming languages. 


**More Information** 


https://www.tcl.tk/ 


**Available Versions**:: 


    8.6.11-GCCcore-11.2.0
    8.6.10-GCCcore-9.3.0
    8.6.9-GCCcore-8.3.0
    8.6.8-GCCcore-7.3.0
    8.6.9-GCCcore-8.2.0
    8.6.11-GCCcore-10.3.0
    8.6.10-GCCcore-10.2.0



Tkinter
------- 



**Description** 


Tkinter module, built with the Python buildsystem 


**More Information** 


https://python.org/ 


**Available Versions**:: 


    3.7.4-GCCcore-8.3.0
    2.7.15-foss-2018b-Python-2.7.15
    2.7.15-fosscuda-2018b-Python-2.7.15
    3.9.5-GCCcore-10.3.0
    3.8.6-GCCcore-10.2.0
    3.8.2-GCCcore-9.3.0
    2.7.16-GCCcore-8.3.0



Togl
---- 



**Description** 


A Tcl/Tk widget for OpenGL rendering. 


**More Information** 


https://sourceforge.net/projects/togl/ 


**Available Versions**:: 


    2.0-GCCcore-8.3.0



TopHat
------ 



**Description** 


TopHat is a fast splice junction mapper for RNA-Seq reads. 


**More Information** 


http://ccb.jhu.edu/software/tophat/ 


**Available Versions**:: 


    2.1.2-gompi-2019b



TreeMix
------- 



**Description** 


TreeMix is a method for inferring the patterns of population splits and mixtures in the history of a set of populations. 


**More Information** 


http://bitbucket.org/nygcresearch/treemix 


**Available Versions**:: 


    1.13-GCC-10.3.0
    1.13-intel-2020a



Trilinos
-------- 



**Description** 


The Trilinos Project is an effort to develop algorithms and enabling technologies within an object-oriented software framework for the solution of large-scale, complex multi-physics engineering and scientific problems. A unique design feature of Trilinos is its focus on packages. 


**More Information** 


https://trilinos.org 


**Available Versions**:: 


    12.12.1-foss-2019b-Python-3.7.4



Trim_Galore
----------- 



**Description** 


A wrapper tool around Cutadapt and FastQC to consistently apply quality and adapter trimming to FastQ files, with some extra functionality for MspI-digested RRBS-type (Reduced Representation Bisufite-Seq) libraries. 


**More Information** 


http://www.bioinformatics.babraham.ac.uk/projects/trim_galore/ 


**Available Versions**:: 


    0.5.0-foss-2018b
    0.6.5-GCCcore-8.3.0-Java-11-Python-3.7.4



Trimmomatic
----------- 



**Description** 


Trimmomatic performs a variety of useful trimming tasks for illumina  paired-end and single ended data.The selection of trimming steps and their associated  parameters are supplied on the command line.  


**More Information** 


http://www.usadellab.org/cms/?page=trimmomatic 


**Available Versions**:: 


    0.39-Java-1.8
    0.38-Java-1.8



TwoBit
------ 



**Available Versions**:: 


    2022



UCC
--- 



**Description** 


UCC (Unified Collective Communication) is a collectivecommunication operations API and library that is flexible, complete, and feature-rich for current and emerging programming models and runtimes. 


**More Information** 


https://www.openucx.org/ 


**Available Versions**:: 


    1.0.0-GCCcore-11.3.0



UCX
--- 



**Description** 


Unified Communication XAn open-source production grade communication framework for data centricand high-performance applications 


**More Information** 


http://www.openucx.org/ 


**Available Versions**:: 


    1.9.0-GCCcore-10.3.0-CUDA-11.1.1
    1.10.0-GCCcore-10.3.0
    1.8.0-GCCcore-9.3.0-CUDA-11.0.2
    1.9.0-GCCcore-10.2.0-CUDA-11.1.1
    1.12.1-GCCcore-11.3.0
    1.9.0-GCCcore-10.2.0
    1.11.2-GCCcore-11.2.0
    1.8.0-GCCcore-9.3.0



UCX-CUDA
-------- 



**Description** 


Unified Communication XAn open-source production grade communication framework for data centricand high-performance applicationsThis module adds the UCX CUDA support. 


**More Information** 


http://www.openucx.org/ 


**Available Versions**:: 


    1.11.2-GCCcore-11.2.0-CUDA-11.4.1
    1.10.0-GCCcore-10.3.0-CUDA-11.3.1



UDUNITS
------- 



**Description** 


UDUNITS supports conversion of unit specifications between formatted and binary forms, arithmetic manipulation of units, and conversion of values between compatible scales of measurement. 


**More Information** 


https://www.unidata.ucar.edu/software/udunits/ 


**Available Versions**:: 


    2.2.26-foss-2020a
    2.2.26-GCCcore-8.3.0
    2.2.28-GCCcore-11.2.0
    2.2.28-GCCcore-10.3.0
    2.2.26-GCCcore-10.2.0



UFL
--- 



**Description** 


The Unified Form Language (UFL) is a domain specific language  for declaration of finite element discretizations of variational forms.  More precisely, it defines a flexible interface for choosing finite element  spaces and defining expressions for weak forms in a notation close to  mathematical notation. 


**More Information** 


https://bitbucket.org/fenics-project/ufl 


**Available Versions**:: 


    2019.1.0-foss-2019b-Python-3.7.4



UMI-tools
--------- 



**Description** 


Tools for handling Unique Molecular Identifiers in NGS data sets 


**More Information** 


https://umi-tools.readthedocs.io 


**Available Versions**:: 


    1.0.1-foss-2020a-Python-3.8.2



UnZip
----- 



**Description** 


UnZip is an extraction utility for archives compressedin .zip format (also called "zipfiles"). Although highly compatible bothwith PKWARE's PKZIP and PKUNZIP utilities for MS-DOS and with Info-ZIP'sown Zip program, our primary objectives have been portability andnon-MSDOS functionality. 


**More Information** 


http://www.info-zip.org/UnZip.html 


**Available Versions**:: 


    6.0-GCCcore-11.3.0
    6.0-GCCcore-10.2.0
    6.0-GCCcore-9.3.0
    6.0-GCCcore-10.3.0
    6.0-GCCcore-11.2.0



V8
-- 



**Description** 


R interface to Google's open source JavaScript engine 


**More Information** 


https://cran.r-project.org/web/packages/V8/ 


**Available Versions**:: 


    2.2-foss-2020a-R-4.0.2
    3.6.0-foss-2021b-R-4.1.2
    3.6.0-foss-2021a-R-4.1.0



VASP
---- 



**Available Versions**:: 


    1.0-info



VCF-kit
------- 



**Description** 


VCF-kit is a command-line based collection of utilities for performing analysis on Variant Call Format (VCF) files. 


**More Information** 


https://github.com/AndersenLab/VCF-kit 


**Available Versions**:: 


    0.1.6-foss-2018b-Python-2.7.15



VCFtools
-------- 



**Description** 


The aim of VCFtools is to provide  easily accessible methods for working with complex  genetic variation data in the form of VCF files. 


**More Information** 


https://vcftools.github.io 


**Available Versions**:: 


    0.1.16-GCC-11.2.0
    0.1.16-GCC-9.3.0



VTK
--- 



**Description** 


The Visualization Toolkit (VTK) is an open-source, freely available software system for 3D computer graphics, image processing and visualization. VTK consists of a C++ class library and several interpreted interface layers including Tcl/Tk, Java, and Python. VTK supports a wide variety of visualization algorithms including: scalar, vector, tensor, texture, and volumetric methods; and advanced modeling techniques such as: implicit modeling, polygon reduction, mesh smoothing, cutting, contouring, and Delaunay triangulation. 


**More Information** 


https://www.vtk.org 


**Available Versions**:: 


    8.2.0-intel-2020a-Python-3.8.2
    8.2.0-foss-2020a-Python-3.8.2
    8.2.0-foss-2019b-Python-3.7.4
    9.1.0-foss-2021b



VTune
----- 



**Description** 


Intel VTune Amplifier XE is the premier performance profiler for C, C++, C#, Fortran, Assembly and Java. 


**More Information** 


https://software.intel.com/en-us/vtune 


**Available Versions**:: 


    2021.9.0
    2019_update5



Valgrind
-------- 



**Description** 


Valgrind: Debugging and profiling tools 


**More Information** 


https://valgrind.org 


**Available Versions**:: 


    3.16.1-gompi-2020a
    3.14.0-foss-2018b
    3.16.1-iimpi-2020a
    3.18.1-gompi-2021b
    3.17.0-gompi-2021a



Velvet
------ 



**Description** 


Sequence assembler for very short reads 


**More Information** 


http://www.ebi.ac.uk/~zerbino/velvet/ 


**Available Versions**:: 


    1.2.10-foss-2018b-mt-kmer_191



Vim
--- 



**Description** 


Vim is an advanced text editor that seeks to provide the power  of the de-facto Unix editor 'Vi', with a more complete feature set.  


**More Information** 


http://www.vim.org 


**Available Versions**:: 


    8.1.1209-GCCcore-8.2.0-Python-3.7.2



Voro++
------ 



**Description** 


Voro++ is a software library for carrying out three-dimensional computations of the Voronoitessellation. A distinguishing feature of the Voro++ library is that it carries out cell-based calculations,computing the Voronoi cell for each particle individually. It is particularly well-suited for applications thatrely on cell-based statistics, where features of Voronoi cells (eg. volume, centroid, number of faces) can be usedto analyze a system of particles. 


**More Information** 


http://math.lbl.gov/voro++/ 


**Available Versions**:: 


    0.4.6-GCCcore-9.3.0



WIEN2k
------ 



**Available Versions**:: 


    1.0-info
    21.1-intel-2021a
    21.1-JP-intel-2021a



Wannier90
--------- 



**Description** 


A tool for obtaining maximally-localised Wannier functions 


**More Information** 


http://www.wannier.org 


**Available Versions**:: 


    3.1.0-foss-2020b



WannierTools
------------ 



**Description** 


WannierTools is an open source software that studies the physical properties of given tight-binding model. 


**More Information** 


http://www.wanniertools.com 


**Available Versions**:: 


    2.5.1-intel-2020b



X11
--- 



**Description** 


The X Window System (X11) is a windowing system for bitmap displays 


**More Information** 


https://www.x.org 


**Available Versions**:: 


    20200222-GCCcore-9.3.0
    20180604-GCCcore-7.3.0
    20190311-GCCcore-8.2.0
    20190717-GCCcore-8.3.0
    20210518-GCCcore-10.3.0
    20210802-GCCcore-11.2.0
    20201008-GCCcore-10.2.0



XCFun
----- 



**Description** 


Arbitrary order exchange-correlation functional library 


**More Information** 


https://xcfun.readthedocs.io 


**Available Versions**:: 


    2.1.1-GCCcore-10.3.0
    20190127-foss-2020a-Python-3.8.2
    20190127-foss-2019a-Python-3.7.2



XCfun
----- 



**Description** 


XCFun is a library of DFT exchange-correlation (XC) functionals. It is based on automatic differentiation and can therefore generate arbitrary order derivatives of these functionals.  


**More Information** 


http://dftlibs.org/xcfun/ 


**Available Versions**:: 


    20190127-foss-2019a-Python-3.7.2



XCrySDen
-------- 



**Description** 


XCrySDen is a crystalline and molecular structure visualisation program aiming at display of isosurfaces and contours, which can be superimposed on crystalline structures and interactively rotated and manipulated. 


**More Information** 


http://www.xcrysden.org/ 


**Available Versions**:: 


    1.6.2-foss-2019b
    1.6.2-intel-2019b



XMDS2
----- 



**Description** 


The purpose of XMDS2 is to simplify the process of creating simulations that solve systems of initial-value first-order partial and ordinary differential equations. 


**More Information** 


http://www.xmds.org/ 


**Available Versions**:: 


    3.1.0-foss-2020a-Python-3.8.2
    2.2.3-foss-2018b-Python-2.7.15



XML-Parser
---------- 



**Description** 


This is a Perl extension interface to James Clark's XML parser, expat. 


**More Information** 


http://search.cpan.org/~toddr/XML-Parser-2.41/ 


**Available Versions**:: 


    2.44_01-GCCcore-7.3.0-Perl-5.28.0



XZ
-- 



**Description** 


xz: XZ utilities 


**More Information** 


https://tukaani.org/xz/ 


**Available Versions**:: 


    5.2.4-GCCcore-8.3.0
    5.2.5-GCCcore-10.3.0
    5.2.5-GCCcore-9.3.0
    5.2.4-GCCcore-8.2.0
    5.2.5-GCCcore-10.2.0
    5.2.5-GCCcore-11.2.0
    5.2.5-GCCcore-11.3.0
    5.2.4-GCCcore-10.2.0
    5.2.4-GCCcore-7.3.0



Xvfb
---- 



**Description** 


Xvfb is an X server that can run on machines with no display hardware and no physical input devices. It emulates a dumb framebuffer using virtual memory. 


**More Information** 


https://www.x.org/releases/X11R7.6/doc/man/man1/Xvfb.1.xhtml 


**Available Versions**:: 


    1.20.11-GCCcore-10.3.0
    1.20.9-GCCcore-10.2.0
    1.20.13-GCCcore-11.2.0



YAXT
---- 



**Description** 


Yet Another eXchange Tool 


**More Information** 


https://www.dkrz.de/redmine/projects/yaxt 


**Available Versions**:: 


    0.9.0-gompi-2020b



Yasm
---- 



**Description** 


Yasm: Complete rewrite of the NASM assembler with BSD license 


**More Information** 


https://www.tortall.net/projects/yasm/ 


**Available Versions**:: 


    1.3.0-GCCcore-10.2.0
    1.3.0-GCCcore-8.2.0
    1.3.0-GCCcore-8.3.0
    1.3.0-GCCcore-10.3.0
    1.3.0-GCCcore-9.3.0



Z3
-- 



**Description** 


Z3 is a theorem prover from Microsoft Research. 


**More Information** 


https://github.com/Z3Prover/z3 


**Available Versions**:: 


    4.8.10-GCCcore-10.2.0



ZeroMQ
------ 



**Description** 


ZeroMQ looks like an embeddable networking library but acts like a concurrency framework. It gives you sockets that carry atomic messages across various transports like in-process, inter-process, TCP, and multicast. You can connect sockets N-to-N with patterns like fanout, pub-sub, task distribution, and request-reply. It's fast enough to be the fabric for clustered products. Its asynchronous I/O model gives you scalable multicore applications, built as asynchronous message-processing tasks. It has a score of language APIs and runs on most operating systems. 


**More Information** 


https://www.zeromq.org/ 


**Available Versions**:: 


    4.3.3-GCCcore-10.2.0
    4.3.4-GCCcore-11.2.0
    4.3.2-GCCcore-8.3.0
    4.3.2-GCCcore-9.3.0
    4.3.4-GCCcore-10.3.0



Zip
--- 



**Description** 


Zip is a compression and file packaging/archive utility.Although highly compatible both with PKWARE's PKZIP and PKUNZIPutilities for MS-DOS and with Info-ZIP's own UnZip, our primary objectiveshave been portability and other-than-MSDOS functionality 


**More Information** 


http://www.info-zip.org/Zip.html 


**Available Versions**:: 


    3.0-GCCcore-11.2.0
    3.0-GCCcore-10.2.0
    3.0-GCCcore-10.3.0
    3.0-GCCcore-8.3.0
    3.0-GCCcore-9.3.0



angsd
----- 



**Description** 


Program for analysing NGS data. 


**More Information** 


http://www.popgen.dk/angsd 


**Available Versions**:: 


    0.933-GCC-8.3.0
    0.935-GCC-10.2.0
    0.925-foss-2018b



annovar
------- 



**Description** 


ANNOVAR is an efficient software tool to utilize update-to-date information to functionally annotate genetic variants detected from diverse genomes (including human genome hg18, hg19, hg38, as well as mouse, worm, fly, yeast and many others). 


**More Information** 


http://annovar.openbioinformatics.org/en/latest/ 


**Available Versions**:: 


    20191024-GCCcore-8.3.0-Perl-5.28.1



ant
--- 



**Description** 


Apache Ant is a Java library and command-line tool whose mission is to drive processes described in build files as targets and extension points dependent upon each other. The main known usage of Ant is the build of Java applications. 


**More Information** 


https://ant.apache.org/ 


**Available Versions**:: 


    1.10.9-Java-11
    1.10.8-Java-11
    1.10.7-Java-11
    1.10.11-Java-11



archspec
-------- 



**Description** 


A library for detecting, labeling, and reasoning about microarchitectures 


**More Information** 


https://github.com/archspec/archspec 


**Available Versions**:: 


    0.1.0-GCCcore-9.3.0-Python-3.8.2



aria2
----- 



**Description** 


aria2 is a lightweight multi-protocol & multi-source command-line download utility. 


**More Information** 


https://aria2.github.io 


**Available Versions**:: 


    1.35.0-GCCcore-10.3.0



arpack-ng
--------- 



**Description** 


ARPACK is a collection of Fortran77 subroutines designed to solve large scale eigenvalue problems. 


**More Information** 


https://github.com/opencollab/arpack-ng 


**Available Versions**:: 


    3.7.0-foss-2020a
    3.7.0-foss-2019b
    3.7.0-intel-2020a



arrow-R
------- 



**Description** 


R interface to the Apache Arrow C++ library 


**More Information** 


https://cran.r-project.org/web/packages/arrow 


**Available Versions**:: 


    6.0.0.2-foss-2021b-R-4.1.2



at-spi2-atk
----------- 



**Description** 


AT-SPI 2 toolkit bridge 


**More Information** 


https://wiki.gnome.org/Accessibility 


**Available Versions**:: 


    2.34.2-GCCcore-9.3.0
    2.32.0-GCCcore-8.2.0
    2.38.0-GCCcore-10.2.0
    2.34.1-GCCcore-8.3.0



at-spi2-core
------------ 



**Description** 


Assistive Technology Service Provider Interface. 


**More Information** 


https://wiki.gnome.org/Accessibility 


**Available Versions**:: 


    2.34.0-GCCcore-8.3.0
    2.38.0-GCCcore-10.2.0
    2.36.0-GCCcore-9.3.0
    2.32.0-GCCcore-8.2.0



bcl2fastq2
---------- 



**Description** 


bcl2fastq Conversion Software both demultiplexes data and converts BCL files generated by Illumina sequencing systems to standard FASTQ file formats for downstream analysis. 


**More Information** 


https://support.illumina.com/sequencing/sequencing_software/bcl2fastq-conversion-software.html 


**Available Versions**:: 


    2.20.0-foss-2018b
    2.20.0-GCC-10.2.0



beagle-lib
---------- 



**Description** 


beagle-lib is a high-performance library that can perform the core calculations at the heart of most Bayesian and Maximum Likelihood phylogenetics packages. 


**More Information** 


https://github.com/beagle-dev/beagle-lib 


**Available Versions**:: 


    3.1.2-GCC-7.3.0-2.30
    3.1.2-GCC-10.2.0
    3.1.2-GCC-8.2.0-2.31.1



binutils
-------- 



**Description** 


binutils: GNU binary utilities 


**More Information** 


http://directory.fsf.org/project/binutils/ 


**Available Versions**:: 


    2.26
    2.34
    2.35-GCCcore-10.2.0
    2.37-GCCcore-11.2.0
    2.30-GCCcore-7.3.0
    2.38-GCCcore-11.3.0
    2.38
    2.36.1-GCCcore-10.3.0
    2.31.1-GCCcore-8.3.0
    2.25
    2.32-GCCcore-9.3.0
    2.28
    2.30
    2.27
    2.30-GCCcore-8.1.0
    2.35
    2.32
    2.31.1
    2.32-GCCcore-8.3.0
    2.31.1-GCCcore-8.2.0
    2.37
    2.34-GCCcore-9.3.0
    2.36.1



bokeh
----- 



**Description** 


Statistical and novel interactive HTML plots for Python 


**More Information** 


https://github.com/bokeh/bokeh 


**Available Versions**:: 


    1.4.0-foss-2019b-Python-3.7.4
    2.2.3-foss-2020b
    2.2.3-fosscuda-2020b



bwidget
------- 



**Description** 


The BWidget Toolkit is a high-level Widget Set for Tcl/Tk built using native Tcl/Tk 8.x namespaces. 


**More Information** 


https://core.tcl-lang.org/bwidget/home 


**Available Versions**:: 


    1.9.14-GCCcore-9.3.0
    1.9.13-GCCcore-8.2.0
    1.9.14-GCCcore-8.3.0



bzip2
----- 



**Description** 


bzip2 is a freely available, patent free, high-quality data compressor. It typically compresses files to within 10% to 15% of the best available techniques (the PPM family of statistical compressors), whilst being around twice as fast at compression and six times faster at decompression. 


**More Information** 


https://sourceware.org/bzip2 


**Available Versions**:: 


    1.0.8-GCCcore-7.3.0
    1.0.8-GCCcore-10.3.0
    1.0.6-GCCcore-8.3.0
    1.0.8-GCCcore-11.2.0
    1.0.8-GCCcore-11.3.0
    1.0.6-GCCcore-10.2.0
    1.0.8-GCCcore-10.2.0
    1.0.6-GCCcore-7.3.0
    1.0.8-GCCcore-8.3.0
    1.0.6-GCCcore-8.2.0
    1.0.8-GCCcore-9.3.0



c-ares
------ 



**Description** 


c-ares is a C library for asynchronous DNS requests (including name resolves) 


**More Information** 


https://c-ares.haxx.se 


**Available Versions**:: 


    1.17.2-GCCcore-10.3.0



c2x
--- 



**Available Versions**:: 


    2.40



cURL
---- 



**Description** 


libcurl is a free and easy-to-use client-side URL transfer library, supporting DICT, FILE, FTP, FTPS, Gopher, HTTP, HTTPS, IMAP, IMAPS, LDAP, LDAPS, POP3, POP3S, RTMP, RTSP, SCP, SFTP, SMTP, SMTPS, Telnet and TFTP. libcurl supports SSL certificates, HTTP POST, HTTP PUT, FTP uploading, HTTP form based upload, proxies, cookies, user+password authentication (Basic, Digest, NTLM, Negotiate, Kerberos), file transfer resume, http proxy tunneling and more. 


**More Information** 


https://curl.haxx.se 


**Available Versions**:: 


    7.66.0-GCCcore-7.3.0
    7.63.0-GCCcore-8.2.0
    7.69.1-GCCcore-9.3.0
    7.76.0-GCCcore-10.3.0
    7.60.0-GCCcore-10.2.0
    7.83.0-GCCcore-11.3.0
    7.78.0-GCCcore-11.2.0
    7.72.0-GCCcore-10.2.0
    7.60.0-GCCcore-7.3.0
    7.66.0-GCCcore-8.3.0



cairo
----- 



**Description** 


Cairo is a 2D graphics library with support for multiple output devices. Currently supported output targets include the X Window System (via both Xlib and XCB), Quartz, Win32, image buffers, PostScript, PDF, and SVG file output. Experimental backends include OpenGL, BeOS, OS/2, and DirectFB 


**More Information** 


http://cairographics.org 


**Available Versions**:: 


    1.16.0-GCCcore-8.2.0
    1.16.0-GCCcore-9.3.0
    1.16.0-GCCcore-11.2.0
    1.16.0-GCCcore-8.3.0
    1.16.0-GCCcore-10.3.0
    1.14.12-GCCcore-7.3.0
    1.16.0-GCCcore-10.2.0



cfx
--- 



**Available Versions**:: 


    2021R2



code-server
----------- 



**Description** 


Run VS Code on any machine anywhere and access it in the browser. 


**More Information** 


https://github.com/cdr/code-server 


**Available Versions**:: 


    3.7.3



configurable-http-proxy
----------------------- 



**Description** 


HTTP proxy for node.js including a REST API for updating the routing table. Developed as a part of the Jupyter Hub multi-user server. 


**More Information** 


https://github.com/jupyterhub/configurable-http-proxy 


**Available Versions**:: 


    4.2.1-GCCcore-10.2.0



constraint
---------- 



**Description** 


The Python constraint module offers solvers for Constraint Satisfaction Problems (CSPs) over finite domains in simple and pure Python. CSP is class of problems which may be represented in terms of variables (a, b, ...), domains (a in [1, 2, 3], ...), and constraints (a < b, ...).   


**More Information** 


https://github.com/python-constraint/python-constraint 


**Available Versions**:: 


    1.4.0-foss-2020a-Python-3.8.2



crest
----- 



**Description** 


CREST is an utility/driver program for the xtb program. Originally it was designed asconformer sampling program, hence the abbreviation Conformer-Rotamer Ensemble Sampling Tool, but now offers also some utility functions for calculations with the GFNn-xTB methods. Generally the program functions as an IO based OMP scheduler (i.e., calculations are performed by the xtb program) and tool for the creation and analysation of structure ensembles. 


**More Information** 


https://xtb-docs.readthedocs.io/en/latest/crest.html 


**Available Versions**:: 


    2.10.2



cuDNN
----- 



**Description** 


The NVIDIA CUDA Deep Neural Network library (cuDNN) isa GPU-accelerated library of primitives for deep neural networks. 


**More Information** 


https://developer.nvidia.com/cudnn 


**Available Versions**:: 


    8.2.2.26-CUDA-11.4.1
    7.6.4.38-gcccuda-2019b
    8.2.1.32-CUDA-11.3.1
    8.0.4.30-CUDA-11.0.2
    8.0.4.30-CUDA-11.1.1
    8.1.1.33-CUDA-11.2.2



cutadapt
-------- 



**Description** 


Cutadapt finds and removes adapter sequences, primers, poly-A tails and other types of unwanted sequence from your high-throughput sequencing reads. 


**More Information** 


https://opensource.scilifelab.se/projects/cutadapt/ 


**Available Versions**:: 


    1.18-foss-2018b-Python-3.6.6
    2.7-GCCcore-8.3.0-Python-3.7.4
    3.5-GCCcore-11.2.0



dask
---- 



**Description** 


Dask natively scales Python. Dask provides advanced parallelism for analytics, enabling performance at scale for the tools you love. 


**More Information** 


https://dask.org/ 


**Available Versions**:: 


    2021.2.0-fosscuda-2020b
    2021.2.0-foss-2020b
    2.8.0-foss-2019b-Python-3.7.4



datamash
-------- 



**Description** 


GNU datamash performs basic numeric, textual and statistical operations on input data files 


**More Information** 


https://www.gnu.org/software/datamash/ 


**Available Versions**:: 


    1.5-GCCcore-10.2.0



dbus-glib
--------- 



**Description** 


D-Bus is a message bus system, a simple way for applications to talk to one another. 


**More Information** 


https://dbus.freedesktop.org/doc/dbus-glib 


**Available Versions**:: 


    0.110-GCCcore-8.3.0



deepdiff
-------- 



**Description** 


DeepDiff: Deep Difference of dictionaries, iterables and almost any other object recursively. 


**More Information** 


https://deepdiff.readthedocs.io/en/latest/ 


**Available Versions**:: 


    4.0.6-GCCcore-8.2.0-Python-3.7.2



dgd
--- 



**Available Versions**:: 


    2021-openmpi



dijitso
------- 



**Description** 


dijitso is a Python module for distributed just-in-time shared library building. 


**More Information** 


https://bitbucket.org/fenics-project/dijitso 


**Available Versions**:: 


    2019.1.0-foss-2019b-Python-3.7.4



dill
---- 



**Description** 


dill extends pythons pickle module for serializing and de-serializing python objects to the majority of the built-in python types. Serialization is the process of converting an object to a byte stream, and the inverse of which is converting a byte stream back to a python object hierarchy.   


**More Information** 


https://pypi.org/project/dill/ 


**Available Versions**:: 


    0.3.3-foss-2020a-Python-3.8.2
    0.3.4-GCCcore-10.3.0



dmd
--- 



**Available Versions**:: 


    2



dot
--- 



**Available Versions**:: 


    



double-conversion
----------------- 



**Description** 


Efficient binary-decimal and decimal-binary conversion routines for IEEE doubles. 


**More Information** 


https://github.com/google/double-conversion 


**Available Versions**:: 


    3.1.5-GCCcore-10.3.0
    3.1.5-GCCcore-9.3.0
    3.1.4-GCCcore-8.3.0
    3.1.5-GCCcore-10.2.0
    3.1.5-GCCcore-11.2.0



ecCodes
------- 



**Description** 


ecCodes is a package developed by ECMWF which provides an application programming interface and a set of tools for decoding and encoding messages in the following formats: WMO FM-92 GRIB edition 1 and edition 2, WMO FM-94 BUFR edition 3 and edition 4, WMO GTS abbreviated header (only decoding). 


**More Information** 


https://software.ecmwf.int/wiki/display/ECC/ecCodes+Home 


**Available Versions**:: 


    2.20.0-gompi-2020b



eilmer
------ 



**Available Versions**:: 


    4.0-9c87a327



elfutils
-------- 



**Description** 


The elfutils project provides libraries and tools for ELF files and DWARF data. 


**More Information** 


https://elfutils.org/ 


**Available Versions**:: 


    0.183-GCCcore-10.2.0
    0.185-GCCcore-8.3.0



enaBrowserTools
--------------- 



**Description** 


enaBrowserTools is a set of scripts that interface with the ENA web services to download data from ENA easily, without any knowledge of scripting required. 


**More Information** 


https://github.com/enasequence/enaBrowserTools/ 


**Available Versions**:: 


    1.6-GCCcore-8.2.0-Python-3.7.2



expat
----- 



**Description** 


Expat is an XML parser library written in C. It is a stream-oriented parser in which an application registers handlers for things the parser might find in the XML document (like start tags) 


**More Information** 


https://libexpat.github.io 


**Available Versions**:: 


    2.2.5-GCCcore-8.1.0
    2.4.1-GCCcore-11.2.0
    2.2.7-GCCcore-7.3.0
    2.2.9-GCCcore-10.3.0
    2.2.5-GCCcore-7.3.0
    2.2.9-GCCcore-9.3.0
    2.2.6-GCCcore-8.3.0
    2.2.7-GCCcore-8.3.0
    2.4.8-GCCcore-11.3.0
    2.2.5-GCCcore-10.2.0
    2.2.6-GCCcore-8.2.0
    2.2.9-GCCcore-10.2.0



fast-gbs
-------- 



**Available Versions**:: 


    v2



file
---- 



**Description** 


The file command is 'a file type guesser', that is, a command-line tool that tells you in words what kind of data a file contains. 


**More Information** 


http://www.darwinsys.com/file/ 


**Available Versions**:: 


    5.35-GCCcore-7.3.0
    5.39-GCCcore-10.2.0



fineRADstructure
---------------- 



**Description** 


A package for population structure inference from RAD-seq data 


**More Information** 


http://cichlid.gurdon.cam.ac.uk/fineRADstructure.html 


**Available Versions**:: 


    20180709-intel-2020a



fineSTRUCTURE
------------- 



**Description** 


fineSTRUCTURE is a fast and powerful algorithm for identifying population structure using dense sequencing data. 


**More Information** 


https://people.maths.bris.ac.uk/~madjl/finestructure/finestructure_info.html 


**Available Versions**:: 


    2.1.3-intel-2020a



flatbuffers
----------- 



**Description** 


FlatBuffers: Memory Efficient Serialization Library 


**More Information** 


https://github.com/google/flatbuffers/ 


**Available Versions**:: 


    1.12.0-GCCcore-9.3.0
    2.0.0-GCCcore-11.2.0
    2.0.0-GCCcore-10.3.0
    1.12.0-GCCcore-10.2.0
    1.12.0-GCCcore-8.3.0



flatbuffers-python
------------------ 



**Description** 


Python Flatbuffers runtime library. 


**More Information** 


https://github.com/google/flatbuffers/ 


**Available Versions**:: 


    1.12-GCCcore-10.2.0
    2.0-GCCcore-10.3.0
    2.0-GCCcore-11.2.0



flex
---- 



**Description** 


Flex (Fast Lexical Analyzer) is a tool for generating scanners. A scanner,  sometimes called a tokenizer, is a program which recognizes lexical patterns in text. 


**More Information** 


http://flex.sourceforge.net/ 


**Available Versions**:: 


    2.6.0
    2.6.4-GCCcore-10.2.0
    2.5.39-GCCcore-8.3.0
    2.6.4-GCCcore-8.2.0
    2.6.4-GCCcore-11.2.0
    2.6.3
    2.5.39
    2.6.4-GCCcore-8.3.0
    2.6.4-GCCcore-8.1.0
    2.6.4-GCCcore-7.3.0
    2.6.4-GCCcore-10.3.0
    2.6.4-GCCcore-11.3.0
    2.6.4-GCCcore-9.3.0
    2.6.4



fluent
------ 



**Available Versions**:: 


    2021R2



fontconfig
---------- 



**Description** 


Fontconfig is a library designed to provide system-wide font configuration, customization and application access. 


**More Information** 


https://www.freedesktop.org/wiki/Software/fontconfig/ 


**Available Versions**:: 


    2.13.93-GCCcore-10.3.0
    2.13.92-GCCcore-10.2.0
    2.13.92-GCCcore-9.3.0
    2.13.94-GCCcore-11.2.0
    2.13.0-GCCcore-7.3.0
    2.13.1-GCCcore-8.2.0
    2.13.1-GCCcore-8.3.0



foss
---- 



**Description** 


GNU Compiler Collection (GCC) based compiler toolchain, including OpenMPI for MPI support, OpenBLAS (BLAS and LAPACK support), FFTW and ScaLAPACK. 


**More Information** 


https://easybuild.readthedocs.io/en/master/Common-toolchains.html#foss-toolchain 


**Available Versions**:: 


    2021a
    2020b
    2018b
    2021b
    2019b
    2019a
    2020a
    2022a



fosscuda
-------- 



**Description** 


GCC based compiler toolchain __with CUDA support__, and including OpenMPI for MPI support, OpenBLAS (BLAS and LAPACK support), FFTW and ScaLAPACK. 


**More Information** 


(none) 


**Available Versions**:: 


    2020b
    2018b
    2019b
    2020a



freeglut
-------- 



**Description** 


freeglut is a completely OpenSourced alternative to the OpenGL Utility Toolkit (GLUT) library. 


**More Information** 


http://freeglut.sourceforge.net/ 


**Available Versions**:: 


    3.2.1-GCCcore-9.3.0
    3.2.1-GCCcore-8.3.0
    3.2.1-GCCcore-11.2.0



freetype
-------- 



**Description** 


FreeType 2 is a software font engine that is designed to be small, efficient, highly customizable, and portable while capable of producing high-quality output (glyph images). It can be used in graphics libraries, display servers, font conversion tools, text image generation tools, and many other products as well. 


**More Information** 


https://www.freetype.org 


**Available Versions**:: 


    2.11.0-GCCcore-11.2.0
    2.10.3-GCCcore-10.2.0
    2.9.1-GCCcore-8.2.0
    2.10.4-GCCcore-10.3.0
    2.9.1-GCCcore-7.3.0
    2.10.1-GCCcore-8.3.0
    2.10.1-GCCcore-9.3.0



garlic
------ 



**Available Versions**:: 


    22



gc
-- 



**Description** 


The Boehm-Demers-Weiser conservative garbage collector can be used as a garbage collecting replacement for C malloc or C++ new. 


**More Information** 


https://hboehm.info/gc/ 


**Available Versions**:: 


    7.6.12-GCCcore-8.3.0
    7.6.10-GCCcore-8.2.0
    7.6.12-GCCcore-9.3.0



gcccuda
------- 



**Description** 


GNU Compiler Collection (GCC) based compiler toolchain, along with CUDA toolkit. 


**More Information** 


(none) 


**Available Versions**:: 


    2020a
    2018b
    2020b
    2019b



gdc-client
---------- 



**Description** 


The gdc-client provides several convenience functions over the GDC API which provides general download/upload via HTTPS. 


**More Information** 


https://gdc.nci.nih.gov/access-data/gdc-data-transfer-tool 


**Available Versions**:: 


    1.6.0-GCCcore-10.2.0



gettext
------- 



**Description** 


GNU 'gettext' is an important step for the GNU Translation Project, as it is an asset on which we maybuild many other steps. This package offers to programmers, translators, and even users, a well integrated set of toolsand documentation 


**More Information** 


https://www.gnu.org/software/gettext/ 


**Available Versions**:: 


    0.20.1-GCCcore-8.3.0
    0.19.8.1
    0.19.8.1-GCCcore-8.2.0
    0.21-GCCcore-10.3.0
    0.21
    0.20.1-GCCcore-9.3.0
    0.21-GCCcore-10.2.0
    0.21-GCCcore-11.2.0
    0.19.8.1-GCCcore-7.3.0
    0.20.1



gflags
------ 



**Description** 


The gflags package contains a C++ library that implements commandline flagsprocessing.  It includes built-in support for standard types such as stringand the ability to define flags in the source file in which they are used. 


**More Information** 


https://github.com/gflags/gflags 


**Available Versions**:: 


    2.2.2-GCCcore-8.2.0



giflib
------ 



**Description** 


giflib is a library for reading and writing gif images.It is API and ABI compatible with libungif which was in wide use whilethe LZW compression algorithm was patented. 


**More Information** 


http://giflib.sourceforge.net/ 


**Available Versions**:: 


    5.2.1-GCCcore-8.3.0
    5.2.1-GCCcore-10.3.0
    5.2.1-GCCcore-11.2.0
    5.1.4-GCCcore-8.2.0
    5.2.1-GCCcore-9.3.0
    5.2.1-GCCcore-10.2.0



git
--- 



**Description** 


Git is a free and open source distributed version control system designedto handle everything from small to very large projects with speed and efficiency. 


**More Information** 


https://git-scm.com/ 


**Available Versions**:: 


    2.23.0-GCCcore-8.3.0
    2.28.0-GCCcore-10.2.0-nodocs
    2.23.0-GCCcore-9.3.0-nodocs
    2.23.0-GCCcore-8.3.0-nodocs
    2.32.0-GCCcore-10.3.0-nodocs
    2.33.1-GCCcore-11.2.0-nodocs



git-lfs
------- 



**Description** 


Git Large File Storage (LFS) replaces large files such as audio samples, videos, datasets, and graphics with text pointers inside Git, while  storing the file contents on a remote server like GitHub.com 


**More Information** 


https://git-lfs.github.com 


**Available Versions**:: 


    2.11.0



glew
---- 



**Description** 


The OpenGL Extension Wrangler Library (GLEW) is a cross-platform open-sourceC/C++ extension loading library. GLEW provides efficient run-time mechanismsfor determining which OpenGL extensions are supported on the target platform. 


**More Information** 


http://glew.sourceforge.net/ 


**Available Versions**:: 


    2.1.0-GCCcore-9.3.0
    2.1.0-GCCcore-8.3.0



glog
---- 



**Description** 


A C++ implementation of the Google logging module. 


**More Information** 


https://github.com/google/glog 


**Available Versions**:: 


    0.4.0-GCCcore-8.2.0



gmpy2
----- 



**Description** 


GMP/MPIR, MPFR, and MPC interface to Python 2.6+ and 3.x 


**More Information** 


https://github.com/aleaxit/gmpy 


**Available Versions**:: 


    2.1.0b5-GCC-10.3.0
    2.1.0b5-GCC-9.3.0
    2.1.0b4-GCC-8.3.0



gmsh
---- 



**Description** 


Gmsh is a 3D finite element grid generator with a build-in CAD engine and post-processor. 


**More Information** 


https://gmsh.info/ 


**Available Versions**:: 


    4.7.1-intel-2020a-Python-3.8.2



gnuplot
------- 



**Description** 


Portable interactive, function plotting utility 


**More Information** 


https://gnuplot.sourceforge.net/ 


**Available Versions**:: 


    5.2.8-GCCcore-9.3.0
    5.4.2-GCCcore-10.3.0
    5.2.8-GCCcore-8.3.0
    5.4.1-GCCcore-10.2.0



gompi
----- 



**Description** 


GNU Compiler Collection (GCC) based compiler toolchain, including OpenMPI for MPI support. 


**More Information** 


(none) 


**Available Versions**:: 


    2018b
    2019a
    2019b-dba
    2020b
    2019b
    2020a
    2021a
    2021b
    2022a



gompic
------ 



**Description** 


GNU Compiler Collection (GCC) based compiler toolchain along with CUDA toolkit, including OpenMPI for MPI support with CUDA features enabled. 


**More Information** 


(none) 


**Available Versions**:: 


    2019b
    2020a
    2020b
    2018b



googletest
---------- 



**Description** 


Google's framework for writing C++ tests on a variety of platforms 


**More Information** 


https://github.com/google/googletest 


**Available Versions**:: 


    1.10.0-GCCcore-10.2.0



gperf
----- 



**Description** 


GNU gperf is a perfect hash function generator. For a given list of strings, it produces a hash function and hash table, in form of C or C++ code, for looking up a value depending on the input string. The hash function is perfect, which means that the hash table has no collisions, and the hash table lookup needs a single string comparison only. 


**More Information** 


https://www.gnu.org/software/gperf/ 


**Available Versions**:: 


    3.1-GCCcore-8.3.0
    3.1-GCCcore-10.2.0
    3.1-GCCcore-7.3.0
    3.1-GCCcore-8.2.0
    3.1-GCCcore-10.3.0
    3.1-GCCcore-9.3.0
    3.1-GCCcore-11.2.0



gperftools
---------- 



**Description** 


gperftools are for use by developers so that they can create more robust applications. Especially of use to those developing multi-threaded applications in C++ with templates. Includes TCMalloc, heap-checker, heap-profiler and cpu-profiler. 


**More Information** 


https://github.com/gperftools/gperftools 


**Available Versions**:: 


    2.7.90-GCCcore-8.3.0
    2.9.1-GCCcore-10.3.0



grib_api
-------- 



**Description** 


The ECMWF GRIB API is an application program interface accessible from C, FORTRAN and Python programs developed for encoding and decoding WMO FM-92 GRIB edition 1 and edition 2 messages. A useful set of command line tools is also provided to give quick access to GRIB messages. 


**More Information** 


https://software.ecmwf.int/wiki/display/GRIB/Home 


**Available Versions**:: 


    1.24.0-intel-2020b



groff
----- 



**Description** 


Groff (GNU troff) is a typesetting system that reads plain text mixed with formatting commands and produces formatted output. 


**More Information** 


https://www.gnu.org/software/groff 


**Available Versions**:: 


    1.22.4-GCCcore-11.3.0
    1.22.4-GCCcore-9.3.0
    1.22.4-GCCcore-11.2.0
    1.22.4-GCCcore-10.3.0



gubbins
------- 



**Description** 


Gubbins (Genealogies Unbiased By recomBinations In Nucleotide Sequences)is an algorithm that iteratively identifies loci containing elevated densities of basesubstitutions while concurrently constructing a phylogeny based on the putative point mutationsoutside of these regions. Simulations demonstrate the algorithm generates highly accuratereconstructions under realistic models of short-term bacterial evolution, and can be runin only a few hours on alignments of hundreds of bacterial genome sequences. 


**More Information** 


https://sanger-pathogens.github.io/gubbins 


**Available Versions**:: 


    2.4.0



gzip
---- 



**Description** 


gzip (GNU zip) is a popular data compression program as a replacement for compress 


**More Information** 


https://www.gnu.org/software/gzip/ 


**Available Versions**:: 


    1.10-GCCcore-10.3.0
    1.10-GCCcore-10.2.0
    1.10-GCCcore-11.2.0
    1.10-GCCcore-9.3.0
    1.10-GCCcore-8.3.0



h5py
---- 



**Description** 


HDF5 for Python (h5py) is a general-purpose Python interface to the Hierarchical Data Format library, version 5. HDF5 is a versatile, mature scientific software library designed for the fast, flexible storage of enormous amounts of data. 


**More Information** 


https://www.h5py.org/ 


**Available Versions**:: 


    2.10.0-fosscuda-2020a-Python-3.8.2
    2.9.0-foss-2019a
    2.10.0-foss-2019b-Python-3.7.4
    3.6.0-foss-2021b
    3.2.1-foss-2021a
    2.8.0-foss-2018b-Python-2.7.15
    2.10.0-foss-2020a-Python-3.8.2
    3.1.0-foss-2020b
    3.1.0-fosscuda-2020b
    2.10.0-fosscuda-2019b-Python-3.7.4
    2.10.0-intel-2020a-Python-3.8.2



help2man
-------- 



**Description** 


help2man produces simple manual pages from the '--help' and '--version' output of other commands. 


**More Information** 


https://www.gnu.org/software/help2man/ 


**Available Versions**:: 


    1.48.3-GCCcore-10.3.0
    1.47.16-GCCcore-10.2.0
    1.47.8-GCCcore-8.3.0
    1.48.3-GCCcore-11.2.0
    1.47.7-GCCcore-8.2.0
    1.47.4-GCCcore-7.3.0
    1.47.6-GCCcore-8.1.0
    1.47.4
    1.49.2-GCCcore-11.3.0
    1.47.12-GCCcore-9.3.0



htop
---- 



**Description** 


An interactive process viewer for Unix 


**More Information** 


http://hisham.hm/htop/ 


**Available Versions**:: 


    2.0.1



hwloc
----- 



**Description** 


The Portable Hardware Locality (hwloc) software package provides a portable abstraction (across OS, versions, architectures, ...) of the hierarchical topology of modern architectures, including NUMA memory nodes, sockets, shared caches, cores and simultaneous multithreading. It also gathers various system attributes such as cache and memory information as well as the locality of I/O devices such as network interfaces, InfiniBand HCAs or GPUs. It primarily aims at helping applications with gathering information about modern computing hardware so as to exploit it accordingly and efficiently. 


**More Information** 


https://www.open-mpi.org/projects/hwloc/ 


**Available Versions**:: 


    2.2.0-GCCcore-10.2.0
    2.2.0-GCCcore-9.3.0
    1.11.12-GCCcore-9.3.0
    1.11.10-GCCcore-7.3.0
    2.5.0-GCCcore-11.2.0
    2.4.1-GCCcore-10.3.0
    2.7.1-GCCcore-11.3.0
    1.11.11-GCCcore-8.2.0
    1.11.12-GCCcore-8.3.0



hypothesis
---------- 



**Description** 


Hypothesis is an advanced testing library for Python. It lets you write tests which are parametrized by a source of examples, and then generates simple and comprehensible examples that make your tests fail. This lets you find more bugs in your code with less work. 


**More Information** 


https://github.com/HypothesisWorks/hypothesis 


**Available Versions**:: 


    6.14.6-GCCcore-11.2.0
    5.41.2-GCCcore-10.2.0
    4.23.4-GCCcore-8.2.0
    5.41.5-GCCcore-10.2.0
    4.44.2-GCCcore-8.3.0-Python-3.7.4
    6.13.1-GCCcore-10.3.0



icc
--- 



**Description** 


Intel C and C++ compilers 


**More Information** 


https://software.intel.com/en-us/intel-compilers/ 


**Available Versions**:: 


    2019.1.144-GCC-8.2.0-2.31.1
    2018.3.222-GCC-7.3.0-2.30



iccifort
-------- 



**Description** 


Intel C, C++ & Fortran compilers 


**More Information** 


http://software.intel.com/en-us/intel-cluster-toolkit-compiler/ 


**Available Versions**:: 


    2019.1.144-GCC-8.2.0-2.31.1
    2018.3.222-GCC-7.3.0-2.30
    2020.4.304
    2019.5.281
    2020.1.217



iccifortcuda
------------ 



**Description** 


Intel C, C++ & Fortran compilers with CUDA toolkit 


**More Information** 


(none) 


**Available Versions**:: 


    2019b
    2020b



ifort
----- 



**Description** 


Intel Fortran compiler 


**More Information** 


https://software.intel.com/en-us/intel-compilers/ 


**Available Versions**:: 


    2019.1.144-GCC-8.2.0-2.31.1
    2018.3.222-GCC-7.3.0-2.30



iimpi
----- 



**Description** 


Intel C/C++ and Fortran compilers, alongside Intel MPI. 


**More Information** 


http://software.intel.com/en-us/intel-cluster-toolkit-compiler/ 


**Available Versions**:: 


    2018b
    2022a
    2020a
    2021a
    2020b
    2021b
    2019b
    2019a



iimpic
------ 



**Description** 


Intel C/C++ and Fortran compilers, alongside Intel MPI and CUDA. 


**More Information** 


(none) 


**Available Versions**:: 


    2019b
    2020b



imkl
---- 



**Description** 


Intel oneAPI Math Kernel Library 


**More Information** 


https://software.intel.com/content/www/us/en/develop/tools/oneapi/components/onemkl.html 


**Available Versions**:: 


    2021.2.0-iimpi-2021a
    2020.4.304-iimpi-2020b
    2018.3.222-iimpi-2018b
    2020.4.304-iimpic-2020b
    2019.1.144-iimpi-2019a
    2021.4.0
    2022.1.0
    2019.5.281-iimpic-2019b
    2020.1.217-iimpi-2020a
    2019.5.281-iimpi-2019b



imkl-FFTW
--------- 



**Description** 


FFTW interfaces using Intel oneAPI Math Kernel Library 


**More Information** 


https://software.intel.com/content/www/us/en/develop/tools/oneapi/components/onemkl.html 


**Available Versions**:: 


    2021.4.0-iimpi-2021b
    2022.1.0-iimpi-2022a



impi
---- 



**Description** 


Intel MPI Library, compatible with MPICH ABI 


**More Information** 


https://software.intel.com/en-us/intel-mpi-library/ 


**Available Versions**:: 


    2018.5.288-iccifort-2019.5.281
    2018.3.222-iccifort-2018.3.222-GCC-7.3.0-2.30
    2018.5.288-iccifortcuda-2019b
    2021.4.0-intel-compilers-2021.4.0
    2019.9.304-iccifort-2020.4.304
    2019.7.217-iccifort-2020.1.217
    2021.2.0-intel-compilers-2021.2.0
    2018.4.274-iccifort-2019.1.144-GCC-8.2.0-2.31.1
    2019.9.304-iccifortcuda-2020b
    2021.6.0-intel-compilers-2022.1.0



intel
----- 



**Description** 


Compiler toolchain including Intel compilers, Intel MPI and Intel Math Kernel Library (MKL). 


**More Information** 


https://easybuild.readthedocs.io/en/master/Common-toolchains.html#intel-toolchain 


**Available Versions**:: 


    2019a
    2022a
    2020a
    2021a
    2018b
    2019b
    2020b
    2021b



intel-compilers
--------------- 



**Description** 


Intel C, C++ & Fortran compilers (classic and oneAPI) 


**More Information** 


https://software.intel.com/content/www/us/en/develop/tools/oneapi/hpc-toolkit.html 


**Available Versions**:: 


    2021.4.0
    2021.2.0
    2022.1.0



intelcuda
--------- 



**Description** 


Intel Cluster Toolkit Compiler Edition provides Intel C/C++ and Fortran compilers, Intel MPI & Intel MKL, with CUDA toolkit 


**More Information** 


(none) 


**Available Versions**:: 


    2019b
    2020b



intltool
-------- 



**Description** 


intltool is a set of tools to centralize translation of many different file formats using GNU gettext-compatible PO files. 


**More Information** 


http://freedesktop.org/wiki/Software/intltool/ 


**Available Versions**:: 


    0.51.0-GCCcore-7.3.0-Perl-5.28.0
    0.51.0-GCCcore-11.2.0
    0.51.0-GCCcore-8.2.0
    0.51.0-GCCcore-8.3.0
    0.51.0-GCCcore-10.3.0
    0.51.0-GCCcore-10.2.0
    0.51.0-GCCcore-9.3.0



iompi
----- 



**Description** 


Intel C/C++ and Fortran compilers, alongside Open MPI. 


**More Information** 


https://software.intel.com/en-us/intel-cluster-toolkit-compiler/ 


**Available Versions**:: 


    2020b



jax
--- 



**Description** 


Composable transformations of Python+NumPy programs:differentiate, vectorize, JIT to GPU/TPU, and more 


**More Information** 


https://pypi.python.org/pypi/jax 


**Available Versions**:: 


    0.2.19-fosscuda-2020b



jbigkit
------- 



**Description** 


JBIG-KIT is a software implementation of the JBIG1 data compression standard (ITU-T T.82), which was designed for bi-level image data, such as scanned documents. 


**More Information** 


https://www.cl.cam.ac.uk/~mgk25/jbigkit/ 


**Available Versions**:: 


    2.1-GCCcore-11.2.0



jemalloc
-------- 



**Description** 


jemalloc is a general purpose malloc(3) implementation that emphasizes fragmentation avoidance and scalable concurrency support. 


**More Information** 


http://jemalloc.net 


**Available Versions**:: 


    5.2.1-GCCcore-10.2.0
    5.2.1-GCCcore-10.3.0
    5.2.1-GCCcore-8.3.0



kallisto
-------- 



**Description** 


kallisto is a program for quantifying abundances of transcripts from RNA-Seq data, or more generally of target sequences using high-throughput sequencing reads. 


**More Information** 


https://pachterlab.github.io/kallisto/ 


**Available Versions**:: 


    0.46.2-foss-2020b



kim-api
------- 



**Description** 


Open Knowledgebase of Interatomic Models.KIM is an API and OpenKIM is a collection of interatomic models (potentials) foratomistic simulations.  This is a library that can be used by simulation programsto get access to the models in the OpenKIM database.This EasyBuild only installs the API, the models can be installed with thepackage openkim-models, or the user can install them manually by running    kim-api-collections-management install user MODELNAMEor    kim-api-collections-management install user OpenKIMto install them all.  


**More Information** 


https://openkim.org/ 


**Available Versions**:: 


    2.1.3-intel-2020a
    2.1.3-foss-2020a



libGLU
------ 



**Description** 


The OpenGL Utility Library (GLU) is a computer graphics library for OpenGL.  


**More Information** 


https://mesa.freedesktop.org/archive/glu/ 


**Available Versions**:: 


    9.0.1-GCCcore-8.3.0
    9.0.2-GCCcore-11.2.0
    9.0.0-foss-2018b
    9.0.1-GCCcore-10.2.0
    9.0.1-GCCcore-10.3.0
    9.0.1-GCCcore-9.3.0



libaio
------ 



**Description** 


Asynchronous input/output library that uses the kernels native interface. 


**More Information** 


https://pagure.io/libaio 


**Available Versions**:: 


    0.3.111-GCCcore-8.3.0
    0.3.112-GCCcore-10.3.0
    0.3.112-GCCcore-10.2.0



libarchive
---------- 



**Description** 


Multi-format archive and compression library 


**More Information** 


https://www.libarchive.org/ 


**Available Versions**:: 


    3.4.0-GCCcore-8.2.0
    3.5.1-GCCcore-8.3.0
    3.5.1-GCCcore-10.3.0
    3.4.3-GCCcore-10.2.0
    3.6.1-GCCcore-11.3.0
    3.5.1-GCCcore-11.2.0



libcerf
------- 



**Description** 


libcerf is a self-contained numeric library that provides an efficient and accurate implementation of complex error functions, along with Dawson, Faddeeva, and Voigt functions. 


**More Information** 


https://jugit.fz-juelich.de/mlz/libcerf 


**Available Versions**:: 


    1.17-GCCcore-10.3.0
    1.7-GCCcore-7.3.0
    1.14-GCCcore-10.2.0
    1.13-GCCcore-9.3.0
    1.13-GCCcore-8.3.0



libcint
------- 



**Description** 


libcint is an open source library for analytical Gaussian integrals. 


**More Information** 


https://github.com/sunqm/libcint 


**Available Versions**:: 


    4.4.0-foss-2021a



libctl
------ 



**Description** 


libctl is a free Guile-based library implementing flexible control files for scientific simulations. 


**More Information** 


https://libctl.readthedocs.io/en/latest/ 


**Available Versions**:: 


    4.1.3-GCCcore-9.3.0



libdap
------ 



**Description** 


A C++ SDK which contains an implementation of DAP 2.0 and DAP4.0. This includes both Client- and Server-side support classes. 


**More Information** 


https://www.opendap.org/software/libdap 


**Available Versions**:: 


    3.20.7-GCCcore-10.3.0
    3.20.7-GCCcore-10.2.0



libdeflate
---------- 



**Description** 


Heavily optimized library for DEFLATE/zlib/gzip compression and decompression. 


**More Information** 


https://github.com/ebiggers/libdeflate 


**Available Versions**:: 


    1.8-GCCcore-11.2.0



libdrm
------ 



**Description** 


Direct Rendering Manager runtime library. 


**More Information** 


https://dri.freedesktop.org 


**Available Versions**:: 


    2.4.92-GCCcore-7.3.0
    2.4.97-GCCcore-8.2.0
    2.4.106-GCCcore-10.3.0
    2.4.102-GCCcore-10.2.0
    2.4.100-GCCcore-9.3.0
    2.4.99-GCCcore-8.3.0
    2.4.107-GCCcore-11.2.0



libdwarf
-------- 



**Description** 


The DWARF Debugging Information Format is of interest to programmers working on compilersand debuggers (and anyone interested in reading or writing DWARF information)) 


**More Information** 


https://www.prevanders.net/dwarf.html 


**Available Versions**:: 


    20201201-GCCcore-10.2.0



libedit
------- 



**Description** 


This BSD-style licensed command line editor library provides generic line editing,history, and tokenization functions, similar to those found in GNU Readline. 


**More Information** 


https://thrysoee.dk/editline/ 


**Available Versions**:: 


    20191231-GCCcore-9.3.0



libelf
------ 



**Description** 


libelf is a free ELF object file access library 


**More Information** 


https://sourceware.org/elfutils 


**Available Versions**:: 


    0.8.13-GCCcore-10.2.0



libepoxy
-------- 



**Description** 


Epoxy is a library for handling OpenGL function pointer management for you 


**More Information** 


https://github.com/anholt/libepoxy 


**Available Versions**:: 


    1.5.3-GCCcore-8.2.0
    1.5.4-GCCcore-9.3.0
    1.5.4-GCCcore-10.2.0
    1.5.4-GCCcore-8.3.0



libevent
-------- 



**Description** 


The libevent API provides a mechanism to execute a callback function when a specific event occurs on a file descriptor or after a timeout has been reached.  Furthermore, libevent also support callbacks due to signals or regular timeouts. 


**More Information** 


https://libevent.org/ 


**Available Versions**:: 


    2.1.11-GCCcore-8.3.0
    2.1.12-GCCcore-10.2.0
    2.1.12-GCCcore-11.2.0
    2.1.12-GCCcore-10.3.0
    2.1.11-GCCcore-9.3.0
    2.1.12-GCCcore-11.3.0
    2.1.8



libfabric
--------- 



**Description** 


Libfabric is a core component of OFI. It is the library that defines and exportsthe user-space API of OFI, and is typically the only software that applicationsdeal with directly. It works in conjunction with provider libraries, which areoften integrated directly into libfabric. 


**More Information** 


https://ofiwg.github.io/libfabric/ 


**Available Versions**:: 


    1.15.1-GCCcore-11.3.0
    1.12.1-GCCcore-10.3.0
    1.13.2-GCCcore-11.2.0
    1.11.0-GCCcore-10.2.0
    1.11.0-GCCcore-9.3.0



libffi
------ 



**Description** 


The libffi library provides a portable, high level programming interface to various calling conventions. This allows a programmer to call any function specified by a call interface description at run-time. 


**More Information** 


https://sourceware.org/libffi/ 


**Available Versions**:: 


    3.4.2-GCCcore-11.3.0
    3.3-GCCcore-10.3.0
    3.2.1-GCCcore-8.2.0
    3.4.2-GCCcore-11.2.0
    3.3-GCCcore-9.3.0
    3.3-GCCcore-10.2.0
    3.2.1-GCCcore-8.3.0
    3.2.1-GCCcore-10.2.0
    3.2.1-GCCcore-7.3.0



libgd
----- 



**Description** 


GD is an open source code library for the dynamic creation of images by programmers. 


**More Information** 


https://libgd.github.io/ 


**Available Versions**:: 


    2.3.0-GCCcore-10.2.0
    2.3.0-GCCcore-9.3.0
    2.2.5-GCCcore-7.3.0
    2.3.1-GCCcore-10.3.0
    2.2.5-GCCcore-8.3.0



libgeotiff
---------- 



**Description** 


Library for reading and writing coordinate system information from/to GeoTIFF files 


**More Information** 


https://directory.fsf.org/wiki/Libgeotiff 


**Available Versions**:: 


    1.7.0-GCCcore-11.2.0
    1.5.1-GCCcore-8.3.0
    1.6.0-GCCcore-10.2.0
    1.6.0-GCCcore-10.3.0
    1.5.1-GCCcore-8.2.0
    1.4.2-foss-2018b
    1.5.1-GCCcore-9.3.0



libgit2
------- 



**Description** 


libgit2 is a portable, pure C implementation of the Git core methods provided as a re-entrantlinkable library with a solid API, allowing you to write native speed custom Git applications in any languagewhich supports C bindings. 


**More Information** 


https://libgit2.org/ 


**Available Versions**:: 


    1.1.0-GCCcore-10.3.0
    1.1.1-GCCcore-11.2.0



libglade
-------- 



**Description** 


Libglade is a library for constructing user interfaces dynamically from XML descriptions. 


**More Information** 


https://developer.gnome.org/libglade/ 


**Available Versions**:: 


    2.6.4-foss-2018b



libglvnd
-------- 



**Description** 


libglvnd is a vendor-neutral dispatch layer for arbitrating OpenGL API calls between multiple vendors. 


**More Information** 


https://gitlab.freedesktop.org/glvnd/libglvnd 


**Available Versions**:: 


    1.3.3-GCCcore-10.3.0
    1.3.3-GCCcore-11.2.0
    1.2.0-GCCcore-9.3.0
    1.3.2-GCCcore-10.2.0



libgpuarray
----------- 



**Description** 


Library to manipulate tensors on the GPU. 


**More Information** 


http://deeplearning.net/software/libgpuarray/ 


**Available Versions**:: 


    0.7.6-fosscuda-2019b-Python-3.7.4
    0.7.6-fosscuda-2020b



libiconv
-------- 



**Description** 


Libiconv converts from one character encoding to another through Unicode conversion 


**More Information** 


https://www.gnu.org/software/libiconv 


**Available Versions**:: 


    1.16-GCCcore-8.2.0
    1.15-GCCcore-7.3.0
    1.16-GCCcore-11.2.0
    1.16-GCCcore-8.3.0
    1.16-GCCcore-10.2.0
    1.16-GCCcore-9.3.0
    1.16-GCCcore-10.3.0



libidn2
------- 



**Description** 


GNU Libidn is a fully documented implementation of the Stringprep, Punycode and IDNA specifications.Libidn's purpose is to encode and decode internationalized domain names. 


**More Information** 


https://www.gnu.org/software/libidn 


**Available Versions**:: 


    2.3.0-GCCcore-10.2.0



libjpeg-turbo
------------- 



**Description** 


libjpeg-turbo is a fork of the original IJG libjpeg which uses SIMD to accelerate baseline JPEG compression and decompression. libjpeg is a library that implements JPEG image encoding, decoding and transcoding. 


**More Information** 


https://sourceforge.net/projects/libjpeg-turbo/ 


**Available Versions**:: 


    2.0.3-GCCcore-8.3.0
    2.0.4-GCCcore-9.3.0
    2.0.0-GCCcore-7.3.0
    2.0.6-GCCcore-11.2.0
    2.0.2-GCCcore-8.2.0
    2.0.5-GCCcore-10.2.0
    2.0.6-GCCcore-10.3.0



libmatheval
----------- 



**Description** 


GNU libmatheval is a library (callable from C and Fortran) to parse and evaluate symbolic expressions input as text. 


**More Information** 


http://www.gnu.org/software/libmatheval/ 


**Available Versions**:: 


    1.1.11-GCCcore-8.2.0
    1.1.11-GCCcore-8.3.0
    1.1.11-GCCcore-9.3.0



libogg
------ 



**Description** 


Ogg is a multimedia container format, and the native file and stream format for the Xiph.orgmultimedia codecs. 


**More Information** 


https://xiph.org/ogg/ 


**Available Versions**:: 


    1.3.4-GCCcore-10.3.0
    1.3.4-GCCcore-10.2.0
    1.3.5-GCCcore-11.2.0



libpciaccess
------------ 



**Description** 


Generic PCI access library. 


**More Information** 


https://cgit.freedesktop.org/xorg/lib/libpciaccess/ 


**Available Versions**:: 


    0.16-GCCcore-10.2.0
    0.14-GCCcore-7.3.0
    0.16-GCCcore-9.3.0
    0.16-GCCcore-10.3.0
    0.14-GCCcore-8.3.0
    0.14-GCCcore-8.2.0
    0.16-GCCcore-11.2.0
    0.16-GCCcore-11.3.0



libpng
------ 



**Description** 


libpng is the official PNG reference library 


**More Information** 


http://www.libpng.org/pub/png/libpng.html 


**Available Versions**:: 


    1.6.37-GCCcore-10.3.0
    1.6.34-GCCcore-7.3.0
    1.6.36-GCCcore-8.2.0
    1.6.37-GCCcore-9.3.0
    1.6.37-GCCcore-10.2.0
    1.6.37-GCCcore-8.3.0
    1.6.37-GCCcore-11.2.0



libreadline
----------- 



**Description** 


The GNU Readline library provides a set of functions for use by applications that allow users to edit command lines as they are typed in. Both Emacs and vi editing modes are available. The Readline library includes additional functions to maintain a list of previously-entered command lines, to recall and perhaps reedit those lines, and perform csh-like history expansion on previous commands. 


**More Information** 


http://cnswww.cns.cwru.edu/php/chet/readline/rltop.html 


**Available Versions**:: 


    8.0-GCCcore-8.2.0
    8.0-GCCcore-8.3.0
    7.0-GCCcore-10.2.0
    8.0-GCCcore-7.3.0
    8.0-GCCcore-10.2.0
    8.1-GCCcore-10.3.0
    8.0-GCCcore-9.3.0
    8.1-GCCcore-11.2.0
    7.0-GCCcore-7.3.0
    8.1.2-GCCcore-11.3.0



libsigc++
--------- 



**Description** 


The libsigc++ package implements a typesafe callback system for standard C++. 


**More Information** 


https://libsigcplusplus.github.io/libsigcplusplus/ 


**Available Versions**:: 


    2.10.2-GCCcore-8.3.0



libsndfile
---------- 



**Description** 


Libsndfile is a C library for reading and writing files containing sampled sound (such as MS Windows WAV and the Apple/SGI AIFF format) through one standard library interface. 


**More Information** 


http://www.mega-nerd.com/libsndfile 


**Available Versions**:: 


    1.0.31-GCCcore-10.3.0
    1.0.31-GCCcore-11.2.0
    1.0.28-GCCcore-9.3.0
    1.0.28-GCCcore-7.3.0
    1.0.28-GCCcore-10.2.0
    1.0.28-GCCcore-8.3.0



libsodium
--------- 



**Description** 


Sodium is a modern, easy-to-use software library for encryption, decryption, signatures, password hashing and more. 


**More Information** 


https://doc.libsodium.org/ 


**Available Versions**:: 


    1.0.18-GCCcore-11.2.0
    1.0.18-GCCcore-10.3.0
    1.0.18-GCCcore-10.2.0
    1.0.18-GCCcore-9.3.0
    1.0.18-GCCcore-8.3.0



libspatialite
------------- 



**Description** 


SpatiaLite is an open source library intended to extend the SQLite core to support fully fledged Spatial SQL capabilities. 


**More Information** 


https://www.gaia-gis.it/fossil/libspatialite/home 


**Available Versions**:: 


    4.3.0a-GCC-8.3.0-Python-3.7.4



libtirpc
-------- 



**Description** 


Libtirpc is a port of Suns Transport-Independent RPC library to Linux. 


**More Information** 


https://sourceforge.net/projects/libtirpc/ 


**Available Versions**:: 


    1.3.2-GCCcore-10.3.0
    1.2.6-GCCcore-8.3.0
    1.3.2-GCCcore-11.2.0
    1.3.1-GCCcore-10.2.0



libtool
------- 



**Description** 


GNU libtool is a generic library support script. Libtool hides the complexity of using shared libraries behind a consistent, portable interface. 


**More Information** 


https://www.gnu.org/software/libtool 


**Available Versions**:: 


    2.4.7-GCCcore-11.3.0
    2.4.6-GCCcore-8.1.0
    2.4.6-GCCcore-10.3.0
    2.4.6-GCCcore-8.2.0
    2.4.6-GCCcore-9.3.0
    2.4.6-GCCcore-7.3.0
    2.4.6-GCCcore-11.2.0
    2.4.6-GCCcore-10.2.0
    2.4.6-GCCcore-8.3.0



libunistring
------------ 



**Description** 


This library provides functions for manipulating Unicode strings and for manipulating C strings according to the Unicode standard. 


**More Information** 


http://www.gnu.org/software/libunistring/ 


**Available Versions**:: 


    0.9.10-GCCcore-8.2.0
    0.9.10-GCCcore-8.3.0
    0.9.10-GCCcore-9.3.0



libunwind
--------- 



**Description** 


The primary goal of libunwind is to define a portable and efficient C programming interface (API) to determine the call-chain of a program. The API additionally provides the means to manipulate the preserved (callee-saved) state of each call-frame and to resume execution at any point in the call-chain (non-local goto). The API supports both local (same-process) and remote (across-process) operation. As such, the API is useful in a number of applications 


**More Information** 


https://www.nongnu.org/libunwind/ 


**Available Versions**:: 


    1.5.0-GCCcore-11.2.0
    1.3.1-GCCcore-9.3.0
    1.4.0-GCCcore-10.2.0
    1.3.1-GCCcore-8.2.0
    1.3.1-GCCcore-8.3.0
    1.2.1-GCCcore-7.3.0
    1.4.0-GCCcore-10.3.0



libvdwxc
-------- 



**Description** 


libvdwxc is a general library for evaluating energy and potential forexchange-correlation (XC) functionals from the vdW-DF family that can be used with variousof density functional theory (DFT) codes. 


**More Information** 


https://libvdwxc.org 


**Available Versions**:: 


    0.4.0-foss-2020b
    0.4.0-foss-2021a



libvorbis
--------- 



**Description** 


Ogg Vorbis is a fully open, non-proprietary, patent-and-royalty-free, general-purpose compressedaudio format 


**More Information** 


https://xiph.org/vorbis/ 


**Available Versions**:: 


    1.3.7-GCCcore-10.3.0
    1.3.7-GCCcore-11.2.0
    1.3.7-GCCcore-10.2.0



libwebp
------- 



**Description** 


WebP is a modern image format that provides superiorlossless and lossy compression for images on the web. Using WebP,webmasters and web developers can create smaller, richer images thatmake the web faster. 


**More Information** 


https://developers.google.com/speed/webp/ 


**Available Versions**:: 


    1.1.0-GCCcore-10.2.0
    1.0.2-GCCcore-8.2.0
    1.1.0-GCCcore-9.3.0



libxc
----- 



**Description** 


Libxc is a library of exchange-correlation functionals for density-functional theory. The aim is to provide a portable, well tested and reliable set of exchange and correlation functionals. 


**More Information** 


https://www.tddft.org/programs/libxc 


**Available Versions**:: 


    4.3.4-GCC-7.3.0-2.30
    3.0.1-GCC-8.2.0-2.31.1
    4.3.4-GCC-9.3.0
    5.1.5-intel-compilers-2021.2.0
    5.1.5-GCC-10.3.0
    4.3.4-GCC-10.2.0
    4.3.4-iccifort-2020.1.217
    5.2.3-intel-compilers-2022.1.0
    5.1.2-GCC-10.2.0
    4.3.4-intel-2020a
    4.3.4-iccifort-2020.4.304
    4.3.4-iccifort-2019.1.144-GCC-8.2.0-2.31.1
    4.3.4-iccifort-2019.5.281
    4.3.4-GCC-8.3.0
    3.0.1-foss-2020a
    5.2.3-GCC-11.3.0
    4.3.4-GCC-8.2.0-2.31.1



libxml++
-------- 



**Description** 


libxml++ is a C++ wrapper for the libxml XML parser library. 


**More Information** 


http://libxmlplusplus.sourceforge.net 


**Available Versions**:: 


    2.40.1-GCCcore-8.3.0



libxml2
------- 



**Description** 


Libxml2 is the XML C parser and toolchain developed for the Gnome project (but usable outside of the Gnome platform). 


**More Information** 


http://xmlsoft.org/ 


**Available Versions**:: 


    2.9.13-GCCcore-11.3.0
    2.9.9-GCCcore-9.3.0
    2.9.10-GCCcore-10.2.0
    2.9.8-GCCcore-7.3.0
    2.9.8-GCCcore-8.2.0
    2.9.10-GCCcore-11.2.0
    2.9.10-GCCcore-9.3.0
    2.9.10-GCCcore-10.3.0
    2.9.9-GCCcore-8.3.0



libxslt
------- 



**Description** 


Libxslt is the XSLT C library developed for the GNOME project (but usable outside of the Gnome platform). 


**More Information** 


http://xmlsoft.org/ 


**Available Versions**:: 


    1.1.34-GCCcore-10.3.0
    1.1.34-GCCcore-10.2.0
    1.1.32-GCCcore-7.3.0
    1.1.34-GCCcore-9.3.0



libxsmm
------- 



**Description** 


LIBXSMM is a library for small dense and small sparse matrix-matrix multiplicationstargeting Intel Architecture (x86). 


**More Information** 


https://github.com/hfp/libxsmm 


**Available Versions**:: 


    1.16.1-GCC-9.3.0
    1.10-foss-2019b
    1.10-GCC-8.2.0-2.31.1
    1.16.1-iccifort-2020.1.217
    1.16.1-GCC-10.2.0
    1.16.1-iccifort-2020.4.304



libyaml
------- 



**Description** 


LibYAML is a YAML parser and emitter written in C. 


**More Information** 


https://pyyaml.org/wiki/LibYAML 


**Available Versions**:: 


    0.2.5-GCCcore-10.2.0
    0.2.2-GCCcore-8.2.0
    0.2.2-GCCcore-8.3.0
    0.2.2-GCCcore-9.3.0



likwid
------ 



**Description** 


Likwid stands for Like I knew what I am doing. This project contributes easy to use command line tools for Linux to support programmers in developing high performance multi threaded programs. 


**More Information** 


https://github.com/RRZE-HPC/likwid 


**Available Versions**:: 


    5.0.1-GCCcore-8.3.0



lpsolve
------- 



**Description** 


Mixed Integer Linear Programming (MILP) solver 


**More Information** 


https://sourceforge.net/projects/lpsolve/ 


**Available Versions**:: 


    5.5.2.11-GCC-10.2.0
    5.5.2.5-GCC-8.3.0



lxml
---- 



**Description** 


The lxml XML toolkit is a Pythonic binding for the C libraries libxml2 and libxslt. 


**More Information** 


https://lxml.de/ 


**Available Versions**:: 


    4.6.3-GCCcore-10.3.0
    4.2.5-foss-2018b-Python-2.7.15
    4.5.2-foss-2020a-Python-3.8.2



lz4
--- 



**Description** 


LZ4 is lossless compression algorithm, providing compression speed at 400 MB/s per core.  It features an extremely fast decoder, with speed in multiple GB/s per core. 


**More Information** 


https://lz4.github.io/lz4/ 


**Available Versions**:: 


    1.9.2-GCCcore-10.2.0
    1.9.2-GCCcore-9.3.0
    1.9.3-GCCcore-10.3.0
    1.9.2-GCCcore-8.3.0
    1.9.3-GCCcore-11.2.0



magma
----- 



**Description** 


The MAGMA project aims to develop a dense linear algebra library similar to LAPACK but for heterogeneous/hybrid architectures, starting with current Multicore+GPU systems. 


**More Information** 


https://icl.cs.utk.edu/magma/ 


**Available Versions**:: 


    2.5.4-fosscuda-2019b
    2.5.4-fosscuda-2020b
    2.5.1-fosscuda-2019b



make
---- 



**Description** 


GNU version of make utility 


**More Information** 


https://www.gnu.org/software/make/make.html 


**Available Versions**:: 


    4.3-GCCcore-10.2.0
    4.3-GCCcore-9.3.0



makeinfo
-------- 



**Description** 


makeinfo is part of the Texinfo project, the official documentation format of the GNU project. 


**More Information** 


https://www.gnu.org/software/texinfo/ 


**Available Versions**:: 


    6.7-GCCcore-9.3.0
    6.7-GCCcore-10.3.0
    6.7-GCCcore-10.3.0-minimal



matplotlib
---------- 



**Description** 


matplotlib is a python 2D plotting library which produces publication quality figures in a variety of hardcopy formats and interactive environments across platforms. matplotlib can be used in python scripts, the python and ipython shell, web application servers, and six graphical user interface toolkits. 


**More Information** 


https://matplotlib.org 


**Available Versions**:: 


    2.2.4-fosscuda-2019b-Python-2.7.16
    3.3.3-foss-2020b
    2.2.4-foss-2019b-Python-2.7.16
    3.2.1-intel-2020a-Python-3.8.2
    3.1.1-foss-2019b-Python-3.7.4
    3.4.2-foss-2021a
    3.3.3-fosscuda-2020b
    3.2.1-foss-2020a-Python-3.8.2
    2.2.3-foss-2018b-Python-2.7.15
    2.2.3-fosscuda-2018b-Python-2.7.15



mdtep
----- 



**Available Versions**:: 


    2022



minimap2
-------- 



**Description** 


Minimap2 is a fast sequence mapping and alignmentprogram that can find overlaps between long noisy reads, or map longreads or their assemblies to a reference genome optionally with detailedalignment (i.e. CIGAR). At present, it works efficiently with querysequences from a few kilobases to ~100 megabases in length at an errorrate ~15%. Minimap2 outputs in the PAF or the SAM format. On limitedtest data sets, minimap2 is over 20 times faster than most otherlong-read aligners. It will replace BWA-MEM for long reads and contigalignment. 


**More Information** 


https://github.com/lh3/minimap2 


**Available Versions**:: 


    2.17-GCC-8.3.0



module-git
---------- 



**Available Versions**:: 


    



module-info
----------- 



**Available Versions**:: 


    



modules
------- 



**Available Versions**:: 


    



molmod
------ 



**Description** 


MolMod is a Python library with many compoments that are useful to write molecular modeling programs. 


**More Information** 


https://molmod.github.io/molmod/ 


**Available Versions**:: 


    1.4.5-foss-2020a-Python-3.8.2
    1.4.5-intel-2020a-Python-3.8.2



motif
----- 



**Description** 


Motif refers to both a graphical user interface (GUI) specification and the widget toolkit for building applications that follow that specification under the X Window System on Unix and other POSIX-compliant systems. It was the standard toolkit for the Common Desktop Environment and thus for Unix. 


**More Information** 


https://motif.ics.com/ 


**Available Versions**:: 


    2.3.8-foss-2019a
    2.3.8-GCCcore-8.3.0



mpi
--- 



**Available Versions**:: 


    openmpi-x86_64



mpitest
------- 



**Available Versions**:: 


    1.0



mpmath
------ 



**Description** 


mpmath can be used as an arbitrary-precision substitute for Python's float/complex types and math/cmath modules, but also does much more advanced mathematics. Almost any calculation can be performed just as well at 10-digit or 1000-digit precision, with either real or complex numbers, and in many cases mpmath implements efficient algorithms that scale well for extremely high precision work. 


**More Information** 


https://mpmath.org/ 


**Available Versions**:: 


    1.1.0-GCCcore-9.3.0-Python-3.8.2
    1.0.0-foss-2018b-Python-2.7.15



mule
---- 



**Available Versions**:: 


    4.4.0



muparserx
--------- 



**Description** 


A C++ Library for Parsing Expressions with Strings, Complex Numbers, Vectors, Matrices and more. 


**More Information** 


https://beltoforion.de/en/muparserx/ 


**Available Versions**:: 


    4.0.8-GCCcore-10.3.0



ncbi-vdb
-------- 



**Description** 


The SRA Toolkit and SDK from NCBI is a collection of tools and libraries for using data in the INSDC Sequence Read Archives. 


**More Information** 


https://github.com/ncbi/ncbi-vdb 


**Available Versions**:: 


    2.9.3-foss-2018b
    2.10.9-gompi-2020b



ncdf4
----- 



**Description** 


ncdf4: Interface to Unidata netCDF (version 4 or earlier) format data files 


**More Information** 


https://cran.r-project.org/web/packages/ncdf4 


**Available Versions**:: 


    1.17-foss-2021a-R-4.1.0
    1.17-foss-2019b



ncurses
------- 



**Description** 


The Ncurses (new curses) library is a free software emulation of curses in System V Release 4.0, and more. It uses Terminfo format, supports pads and color and multiple highlights and forms characters and function-key mapping, and has all the other SYSV-curses enhancements over BSD Curses. 


**More Information** 


http://www.gnu.org/software/ncurses/ 


**Available Versions**:: 


    6.0
    6.3-GCCcore-11.3.0
    6.1-GCCcore-7.3.0
    6.2
    6.1-GCCcore-10.2.0
    6.2-GCCcore-10.2.0
    6.1-GCCcore-8.2.0
    6.1
    6.2-GCCcore-9.3.0
    6.2-GCCcore-10.3.0
    6.1-GCCcore-8.3.0
    6.2-GCCcore-11.2.0



ncview
------ 



**Description** 


Ncview is a visual browser for netCDF format files.Typically you would use ncview to get a quick and easy, push-buttonlook at your netCDF files. You can view simple movies of the data,view along various dimensions, take a look at the actual data values,change color maps, invert the data, etc. 


**More Information** 


https://meteora.ucsd.edu/~pierce/ncview_home_page.html 


**Available Versions**:: 


    2.1.7-foss-2019b



netCDF
------ 



**Description** 


NetCDF (network Common Data Form) is a set of software libraries  and machine-independent data formats that support the creation, access, and sharing of array-oriented  scientific data. 


**More Information** 


https://www.unidata.ucar.edu/software/netcdf/ 


**Available Versions**:: 


    4.7.1-gompi-2019b
    4.6.1-foss-2018b
    4.8.0-gompi-2021a
    4.7.4-iimpi-2020a
    4.7.4-gompi-2020a
    4.6.1-fosscuda-2018b
    4.7.4-gompi-2020b
    4.8.1-gompi-2021b
    4.7.4-gompic-2020b
    4.7.4-iimpi-2020b
    4.7.1-gompi-2019b-dba
    4.6.2-gompi-2019a
    4.7.1-gompic-2019b



netCDF-C++4
----------- 



**Description** 


NetCDF (network Common Data Form) is a set of software libraries and machine-independent data formats that support the creation, access, and sharing of array-oriented scientific data. 


**More Information** 


https://www.unidata.ucar.edu/software/netcdf/ 


**Available Versions**:: 


    4.3.1-gompi-2020b
    4.3.1-iimpi-2020a
    4.3.1-gompi-2021a



netCDF-Fortran
-------------- 



**Description** 


NetCDF (network Common Data Form) is a set of software libraries and machine-independent data formats that support the creation, access, and sharing of array-oriented scientific data. 


**More Information** 


https://www.unidata.ucar.edu/software/netcdf/ 


**Available Versions**:: 


    4.5.3-gompic-2020b
    4.5.2-gompi-2019b-dba
    4.5.2-gompi-2019b
    4.5.3-gompi-2021a
    4.5.3-gompi-2020b
    4.4.4-foss-2018b
    4.5.2-iimpi-2020a
    4.5.3-iimpi-2020b
    4.5.2-gompic-2019b
    4.5.2-gompi-2020a
    4.4.4-fosscuda-2018b



netcdf4-python
-------------- 



**Description** 


Python/numpy interface to netCDF. 


**More Information** 


https://unidata.github.io/netcdf4-python/ 


**Available Versions**:: 


    1.5.3-foss-2020a-Python-3.8.2



nettle
------ 



**Description** 


Nettle is a cryptographic library that is designed to fit easily in more or less any context: In crypto toolkits for object-oriented languages (C++, Python, Pike, ...), in applications like LSH or GNUPG, or even in kernel space. 


**More Information** 


http://www.lysator.liu.se/~nisse/nettle/ 


**Available Versions**:: 


    3.4.1-GCCcore-8.2.0
    3.4-foss-2018b
    3.7.3-GCCcore-11.2.0
    3.6-GCCcore-10.2.0
    3.7.2-GCCcore-10.3.0
    3.5.1-GCCcore-8.3.0



networkx
-------- 



**Description** 


NetworkX is a Python package for the creation, manipulation,and study of the structure, dynamics, and functions of complex networks. 


**More Information** 


https://pypi.python.org/pypi/networkx 


**Available Versions**:: 


    2.5.1-foss-2021a
    2.4-foss-2020a-Python-3.8.2
    2.5-foss-2020b
    2.6.3-foss-2021b
    2.4-foss-2019b-Python-3.7.4
    2.2-foss-2018b-Python-2.7.15
    2.3-foss-2019a-Python-3.7.2
    2.4-foss-2019a-Python-3.7.2



ngsF
---- 



**Available Versions**:: 


    2022



ngsLD
----- 



**Available Versions**:: 


    2020



ngspice
------- 



**Description** 


Ngspice is a mixed-level/mixed-signal circuit simulator. Its code is based on three open source software packages: Spice3f5, Cider1b1 and Xspice. 


**More Information** 


https://ngspice.sourceforge.net 


**Available Versions**:: 


    31-foss-2019b



nlohmann_json
------------- 



**Description** 


JSON for Modern C++ 


**More Information** 


https://github.com/nlohmann/json 


**Available Versions**:: 


    3.10.0-GCCcore-10.3.0



nodejs
------ 



**Description** 


Node.js is a platform built on Chrome's JavaScript runtime for easily building fast, scalable network applications. Node.js uses an event-driven, non-blocking I/O model that makes it lightweight and efficient, perfect for data-intensive real-time applications that run across distributed devices. 


**More Information** 


https://nodejs.org 


**Available Versions**:: 


    12.19.0-GCCcore-10.2.0
    14.17.0-GCCcore-10.3.0
    14.17.6-GCCcore-11.2.0
    10.15.1-foss-2020a



nsync
----- 



**Description** 


nsync is a C library that exports various synchronization primitives, such as mutexes 


**More Information** 


https://github.com/google/nsync 


**Available Versions**:: 


    1.24.0-GCCcore-11.2.0
    1.24.0-GCCcore-10.3.0
    1.24.0-GCCcore-10.2.0
    1.24.0-GCCcore-9.3.0
    1.24.0-GCCcore-8.3.0



null
---- 



**Available Versions**:: 


    



numactl
------- 



**Description** 


The numactl program allows you to run your application program on specific cpu's and memory nodes. It does this by supplying a NUMA memory policy to the operating system before running your program. The libnuma library provides convenient ways for you to add NUMA memory policies into your own program. 


**More Information** 


http://oss.sgi.com/projects/libnuma/ 


**Available Versions**:: 


    2.0.12-GCCcore-8.3.0
    2.0.13-GCCcore-10.2.0
    2.0.11-GCCcore-8.1.0
    2.0.14-GCCcore-11.3.0
    2.0.12-GCCcore-8.2.0
    2.0.13-GCCcore-9.3.0
    2.0.14-GCCcore-10.3.0
    2.0.14-GCCcore-11.2.0
    2.0.11-GCCcore-7.3.0



numba
----- 



**Description** 


Numba is an Open Source NumPy-aware optimizing compiler forPython sponsored by Continuum Analytics, Inc. It uses the remarkable LLVMcompiler infrastructure to compile Python syntax to machine code. 


**More Information** 


https://numba.pydata.org/ 


**Available Versions**:: 


    0.46.0-foss-2019a
    0.53.1-foss-2021a



nvtop
----- 



**Description** 


htop-like GPU usage monitor 


**More Information** 


https://github.com/Syllo/nvtop 


**Available Versions**:: 


    1.2.1-GCCcore-10.3.0



orthomcl
-------- 



**Available Versions**:: 


    v2.0.9



parallel
-------- 



**Description** 


parallel: Build and execute shell commands in parallel 


**More Information** 


https://savannah.gnu.org/projects/parallel/ 


**Available Versions**:: 


    20210722-GCCcore-11.2.0
    20190922-GCCcore-8.3.0
    20200522-GCCcore-9.3.0
    20190222-GCCcore-7.3.0
    20210622-GCCcore-10.3.0
    20190622-GCCcore-8.2.0
    20210322-GCCcore-10.2.0



patchelf
-------- 



**Description** 


PatchELF is a small utility to modify the dynamic linker and RPATH of ELF executables. 


**More Information** 


https://github.com/NixOS/patchelf 


**Available Versions**:: 


    0.10-GCCcore-8.3.0
    0.12-GCCcore-10.3.0



petsc4py
-------- 



**Description** 


petsc4py are Python bindings for PETSc, the Portable, Extensible Toolchain for Scientific Computation. 


**More Information** 


https://bitbucket.org/petsc/petsc4py 


**Available Versions**:: 


    3.12.0-foss-2019b-Python-3.7.4



picard
------ 



**Description** 


A set of tools (in Java) for working with next generation sequencing data in the BAM format. 


**More Information** 


https://sourceforge.net/projects/picard 


**Available Versions**:: 


    2.6.0-Java-1.8.0_131



pigz
---- 



**Description** 


pigz, which stands for parallel implementation of gzip, is a fully functional replacement for gzip that exploits multiple processors and multiple cores to the hilt when compressing data. pigz was written by Mark Adler, and uses the zlib and pthread libraries. 


**More Information** 


https://zlib.net/pigz/ 


**Available Versions**:: 


    2.4-GCCcore-7.3.0
    2.4-GCCcore-8.3.0



pixman
------ 



**Description** 


Pixman is a low-level software library for pixel manipulation, providing features such as image compositing and trapezoid rasterization. Important users of pixman are the cairo graphics library and the X server. 


**More Information** 


http://www.pixman.org/ 


**Available Versions**:: 


    0.40.0-GCCcore-11.2.0
    0.38.4-GCCcore-9.3.0
    0.38.0-GCCcore-8.2.0
    0.34.0-GCCcore-7.3.0
    0.40.0-GCCcore-10.2.0
    0.38.4-GCCcore-8.3.0
    0.40.0-GCCcore-10.3.0



pkg-config
---------- 



**Description** 


pkg-config is a helper tool used when compiling applications and libraries. It helps you insert the correct compiler options on the command line so an application can use gcc -o test test.c `pkg-config --libs --cflags glib-2.0` for instance, rather than hard-coding values on where to find glib (or other libraries). 


**More Information** 


https://www.freedesktop.org/wiki/Software/pkg-config/ 


**Available Versions**:: 


    0.29.2-GCCcore-10.2.0
    0.29.2-GCCcore-9.3.0
    0.29.2-GCCcore-7.3.0
    0.29.2
    0.29.2-GCCcore-8.3.0
    0.29.2-GCCcore-11.2.0
    0.29.2-GCCcore-8.2.0
    0.29.2-GCCcore-10.3.0



pkgconf
------- 



**Description** 


pkgconf is a program which helps to configure compiler and linker flags for development libraries. It is similar to pkg-config from freedesktop.org. 


**More Information** 


https://github.com/pkgconf/pkgconf 


**Available Versions**:: 


    1.8.0-GCCcore-11.3.0



pkgconfig
--------- 



**Description** 


pkgconfig is a Python module to interface with the pkg-config command line tool 


**More Information** 


https://github.com/matze/pkgconfig 


**Available Versions**:: 


    1.5.5-GCCcore-11.2.0-python
    1.5.4-GCCcore-10.3.0-python
    1.5.1-GCCcore-8.2.0-python
    1.5.1-GCCcore-9.3.0-Python-3.8.2
    1.5.1-GCCcore-8.3.0-Python-3.7.4
    1.3.1-foss-2018b-Python-2.7.15
    1.5.1-GCCcore-10.2.0-python



pmi
--- 



**Available Versions**:: 


    pmix-x86_64



pocl
---- 



**Description** 


Pocl is a portable open source (MIT-licensed) implementation of the OpenCL standard 


**More Information** 


https://portablecl.org 


**Available Versions**:: 


    1.6-gcccuda-2020b
    1.6-GCC-10.2.0



preseq
------ 



**Description** 


Software for predicting library complexity and genome coverage in high-throughput sequencing. 


**More Information** 


http://smithlabresearch.org/software/preseq 


**Available Versions**:: 


    2.0.3-foss-2018b
    3.1.2-foss-2020b



prodigal
-------- 



**Description** 


Prodigal (Prokaryotic Dynamic Programming Genefinding Algorithm)    is a microbial (bacterial and archaeal) gene finding program developed    at Oak Ridge National Laboratory and the University of Tennessee. 


**More Information** 


http://prodigal.ornl.gov/ 


**Available Versions**:: 


    2.6.3-GCCcore-7.3.0



protobuf
-------- 



**Description** 


Google Protocol Buffers 


**More Information** 


https://github.com/google/protobuf/ 


**Available Versions**:: 


    3.14.0-GCCcore-10.2.0
    3.10.0-GCCcore-8.3.0
    3.17.3-GCCcore-11.2.0
    3.13.0-GCCcore-9.3.0
    3.17.3-GCCcore-10.3.0



protobuf-python
--------------- 



**Description** 


Python Protocol Buffers runtime library. 


**More Information** 


https://github.com/google/protobuf/ 


**Available Versions**:: 


    3.10.0-fosscuda-2019b-Python-3.7.4
    3.13.0-foss-2020a-Python-3.8.2
    3.14.0-GCCcore-10.2.0
    3.13.0-fosscuda-2020a-Python-3.8.2
    3.17.3-GCCcore-11.2.0
    3.17.3-GCCcore-10.3.0
    3.10.0-foss-2019b-Python-3.7.4



psutil
------ 



**Description** 


A cross-platform process and system utilities module for Python 


**More Information** 


https://github.com/giampaolo/psutil 


**Available Versions**:: 


    5.6.3-GCCcore-8.2.0



pullseq
------- 



**Description** 


Utility program for extracting sequences from a fasta/fastq file 


**More Information** 


https://github.com/bcthomas/pullseq 


**Available Versions**:: 


    1.0.2-GCCcore-7.3.0



pyABC
----- 



**Description** 


Massively parallel, distributed and scalable ABC-SMC (Approximate BayesianComputation - Sequential Monte Carlo) for parameter estimation of complexstochastic models.  Implemented in Python with support of the R language. 


**More Information** 


https://github.com/icb-dcm/pyabc 


**Available Versions**:: 


    0.10.4-foss-2019b-Python-3.7.4



pybedtools
---------- 



**Description** 


pybedtools wraps and extends BEDTools and offers feature-level manipulations from within Python. 


**More Information** 


https://daler.github.io/pybedtools 


**Available Versions**:: 


    0.8.1-foss-2019b



pybind11
-------- 



**Description** 


pybind11 is a lightweight header-only library that exposes C++ types in Python and vice versa, mainly to create Python bindings of existing C++ code. 


**More Information** 


https://pybind11.readthedocs.io 


**Available Versions**:: 


    2.7.1-GCCcore-11.2.0
    2.4.3-GCCcore-8.3.0-Python-3.7.4
    2.6.2-GCCcore-10.3.0
    2.4.3-GCCcore-9.3.0-Python-3.8.2
    2.6.0-GCCcore-10.2.0



pytest-xdist
------------ 



**Description** 


xdist: pytest distributed testing pluginThe pytest-xdist plugin extends pytest with some unique test execution modes:    * test run parallelization: if you have multiple CPUs or hosts you      can use those for a combined test run. This allows to speed up      development or to use special resources of remote machines.    * --looponfail: run your tests repeatedly in a subprocess. After        each run pytest waits until a file in your project changes and        then re-runs the previously failing tests. This is repeated        until all tests pass after which again a full run is        performed.    * Multi-Platform coverage: you can specify different Python      interpreters or different platforms and run tests in parallel on      all of them.Before running tests remotely, pytest efficiently “rsyncs” yourprogram source code to the remote place. All test results are reportedback and displayed to your local terminal. You may specify differentPython versions and interpreters. 


**More Information** 


https://github.com/pytest-dev/pytest-xdist 


**Available Versions**:: 


    2.3.0-GCCcore-10.2.0



python-isal
----------- 



**Description** 


Faster zlib and gzip compatible compression and decompression by providing python bindings for the isa-l library. 


**More Information** 


https://github.com/pycompression/python-isal 


**Available Versions**:: 


    0.11.1-GCCcore-11.2.0



qcint
----- 



**Description** 


libcint is an open source library for analytical Gaussian integrals.qcint is an optimized libcint branch for the x86-64 platform. 


**More Information** 


http://wiki.sunqm.net/libcint 


**Available Versions**:: 


    3.0.18-foss-2019a
    4.0.5-foss-2020a
    3.1.1-foss-2020a



qrupdate
-------- 



**Description** 


qrupdate is a Fortran library for fast updates of QR and Cholesky decompositions. 


**More Information** 


https://sourceforge.net/projects/qrupdate/ 


**Available Versions**:: 


    1.1.2-GCCcore-8.3.0



rclone
------ 



**Description** 


Rclone is a command line program to sync files and directories to and from a variety of online storage services 


**More Information** 


https://rclone.org/ 


**Available Versions**:: 


    1.54.1-amd64



re2c
---- 



**Description** 


re2c is a free and open-source lexer generator for C and C++. Its main goal is generatingfast lexers: at least as fast as their reasonably optimized hand-coded counterparts. Instead of usingtraditional table-driven approach, re2c encodes the generated finite state automata directly in the formof conditional jumps and comparisons. 


**More Information** 


https://re2c.org/ 


**Available Versions**:: 


    2.0.3-GCCcore-10.2.0
    1.3-GCCcore-9.3.0
    1.2.1-GCCcore-8.3.0
    2.1.1-GCCcore-10.3.0



retworkx
-------- 



**Description** 


retworkx is a general purpose graph library for python3 written in Rust to take advantage of the performance and safety that Rust provides. It was built as a replacement for qiskit‘s previous (and current) networkx usage (hence the name) but is designed to provide a high performance general purpose graph library for any python application. The project was originally started to build a faster directed graph to use as the underlying data structure for the DAG at the center of qiskit-terras transpiler, but it has since grown to cover all the graph usage in Qiskit and other applications.   


**More Information** 


https://retworkx.readthedocs.io/en/latest/index.html 


**Available Versions**:: 


    0.6.0-foss-2020a-Python-3.8.2
    0.9.0-foss-2021a



rgdal
----- 



**Description** 


Provides bindings to the 'Geospatial' Data Abstraction Library ('GDAL') (>= 1.11.4 and <= 2.5.0) and access to projection/transformation operations from the 'PROJ.4' library. 


**More Information** 


http://rgdal.r-forge.r-project.org/ 


**Available Versions**:: 


    1.4-8-foss-2020a-R-4.0.2
    1.5-23-foss-2021a-R-4.1.0



rose
---- 



**Available Versions**:: 


    2019.01.4



rosetta
------- 



**Available Versions**:: 


    3.10



rpy2
---- 



**Description** 


rpy2 is an interface to R running embedded in a Python process. 


**More Information** 


https://rpy2.bitbucket.io/ 


**Available Versions**:: 


    3.2.6-foss-2019b-Python-3.7.4



samclip
------- 



**Description** 


Filter SAM file for soft and hard clipped alignments 


**More Information** 


https://github.com/tseemann/samclip 


**Available Versions**:: 


    0.2-GCCcore-7.3.0-Perl-5.28.0



scikit-build
------------ 



**Description** 


Scikit-Build, or skbuild, is an improved build system generatorfor CPython C/C++/Fortran/Cython extensions. 


**More Information** 


https://scikit-build.readthedocs.io/en/latest 


**Available Versions**:: 


    0.11.1-GCCcore-10.3.0
    0.11.1-GCCcore-11.2.0
    0.11.1-foss-2020b



scikit-learn
------------ 



**Description** 


Scikit-learn integrates machine learning algorithms in the tightly-knit scientific Python world,building upon numpy, scipy, and matplotlib. As a machine-learning module,it provides versatile tools for data mining and analysis in any field of science and engineering.It strives to be simple and efficient, accessible to everybody, and reusable in various contexts. 


**More Information** 


https://scikit-learn.org/stable/index.html 


**Available Versions**:: 


    0.24.2-foss-2021a
    0.21.3-foss-2019b-Python-3.7.4
    0.23.1-foss-2020a-Python-3.8.2
    0.20.3-foss-2019a



scipy
----- 



**Description** 


SciPy is a collection of mathematical algorithms and convenience functions built on the Numpy extension for Python. 


**More Information** 


https://www.scipy.org 


**Available Versions**:: 


    1.4.1-foss-2019b-Python-3.7.4



semver
------ 



**Description** 


A Python module for semantic versioning. Simplifies comparing versions. 


**More Information** 


http://semver.org/ 


**Available Versions**:: 


    2.13.0-GCCcore-10.2.0



seqtk
----- 



**Description** 


Seqtk is a fast and lightweight tool for processing sequences in the FASTA or FASTQ format.  It seamlessly parses both FASTA and FASTQ files which can also be optionally compressed by gzip. 


**More Information** 


https://github.com/lh3/seqtk/ 


**Available Versions**:: 


    1.3-foss-2018b



shovill
------- 



**Description** 


Faster SPAdes assembly of Illumina reads 


**More Information** 


https://github.com/tseemann/shovill 


**Available Versions**:: 


    1.1.0-foss-2018b-Python-2.7.15



slepc4py
-------- 



**Description** 


Python bindings for SLEPc, the Scalable Library for Eigenvalue Problem Computations. 


**More Information** 


https://bitbucket.org/slepc/slepc4py 


**Available Versions**:: 


    3.12.0-foss-2019b-Python-3.7.4



snakemake
--------- 



**Description** 


The Snakemake workflow management system is a tool to create reproducible and scalable data analyses. 


**More Information** 


https://snakemake.readthedocs.io 


**Available Versions**:: 


    6.1.0-foss-2020b



snappy
------ 



**Description** 


Snappy is a compression/decompression library. It does not aimfor maximum compression, or compatibility with any other compression library;instead, it aims for very high speeds and reasonable compression. 


**More Information** 


https://github.com/google/snappy 


**Available Versions**:: 


    1.1.8-GCCcore-10.3.0
    1.1.8-GCCcore-9.3.0
    1.1.8-GCCcore-10.2.0
    1.1.9-GCCcore-11.2.0
    1.1.7-GCCcore-8.3.0



sparsehash
---------- 



**Description** 


An extremely memory-efficient hash_map implementation. 2 bits/entry overhead! The SparseHash library contains several hash-map implementations, including implementations that optimize for space or speed. 


**More Information** 


https://github.com/sparsehash/sparsehash 


**Available Versions**:: 


    2.0.3-GCCcore-8.3.0



spdlog
------ 



**Description** 


Very fast, header-only/compiled, C++ logging library. 


**More Information** 


https://github.com/gabime/spdlog 


**Available Versions**:: 


    1.9.2-GCCcore-10.3.0



spglib
------ 



**Description** 


Spglib is a library for finding and handling crystal symmetries written in C. 


**More Information** 


https://atztogo.github.io/spglib/ 


**Available Versions**:: 


    1.9.9-intel-2020a



spglib-python
------------- 



**Description** 


Spglib for Python. Spglib is a library for finding and handling crystal symmetries written in C. 


**More Information** 


https://pypi.python.org/pypi/spglib 


**Available Versions**:: 


    1.16.0-foss-2020b
    1.16.1-foss-2021a



spython
------- 



**Description** 


Singularity Python is a Python API to work with the Singularity open source software. 


**More Information** 


https://singularityhub.github.io/singularity-cli/ 


**Available Versions**:: 


    0.1.14-GCCcore-10.2.0



sympy
----- 



**Description** 


SymPy is a Python library for symbolic mathematics. It aims to become a full-featured computer algebra system (CAS) while keeping the code as simple as possible in order to be comprehensible and easily extensible. SymPy is written entirely in Python and does not require any external libraries. 


**More Information** 


http://sympy.org/ 


**Available Versions**:: 


    1.4-foss-2019a
    1.5.1-foss-2019b-Python-3.7.4
    1.6.2-foss-2020a-Python-3.8.2
    1.8-foss-2021a



tbb
--- 



**Description** 


Intel(R) Threading Building Blocks (Intel(R) TBB) lets you easily write parallel C++ programs that take full advantage of multicore performance, that are portable, composable and have future-proof scalability. 


**More Information** 


https://github.com/oneapi-src/oneTBB 


**Available Versions**:: 


    2019_U9-GCCcore-8.3.0
    2020.3-GCCcore-11.2.0
    2020.3-GCCcore-10.2.0
    2020.1-GCCcore-9.3.0
    2018_U5-GCCcore-7.3.0



tesseract
--------- 



**Description** 


Tesseract is an optical character recognition engine 


**More Information** 


https://github.com/tesseract-ocr/tesseract 


**Available Versions**:: 


    4.1.0-GCCcore-8.2.0



testapp
------- 



**Available Versions**:: 


    1.0



texinfo
------- 



**Description** 


Texinfo is the official documentation format of the GNU project. 


**More Information** 


https://www.gnu.org/software/texinfo/ 


**Available Versions**:: 


    6.7-GCCcore-8.3.0



tmux
---- 



**Description** 


tmux is a terminal multiplexer. It lets you switch easily between several programs in one terminal, detach them (they keep running in the background) and reattach them to a different terminal. 


**More Information** 


http://tmux.sourceforge.net/ 


**Available Versions**:: 


    2.3



torchtext
--------- 



**Description** 


Data loaders and abstractions for text and NLP 


**More Information** 


https://github.com/pytorch/text 


**Available Versions**:: 


    0.7.0-foss-2019b-Python-3.7.4-PyTorch-1.6.0



tqdm
---- 



**Description** 


A fast, extensible progress bar for Python and CLI 


**More Information** 


https://github.com/tqdm/tqdm 


**Available Versions**:: 


    4.41.1-GCCcore-8.3.0



typing-extensions
----------------- 



**Description** 


Typing Extensions – Backported and Experimental Type Hints for Python 


**More Information** 


https://github.com/python/typing/blob/master/typing_extensions/README.rst 


**Available Versions**:: 


    3.7.4.3-GCCcore-10.2.0
    3.10.0.0-GCCcore-10.3.0



use.own
------- 



**Available Versions**:: 


    



utf8proc
-------- 



**Description** 


utf8proc is a small, clean C library that provides Unicode normalization, case-folding, and other operations for data in the UTF-8 encoding. 


**More Information** 


https://github.com/JuliaStrings/utf8proc 


**Available Versions**:: 


    2.5.0-GCCcore-10.2.0



util-linux
---------- 



**Description** 


Set of Linux utilities 


**More Information** 


https://www.kernel.org/pub/linux/utils/util-linux 


**Available Versions**:: 


    2.34-GCCcore-8.3.0
    2.35-GCCcore-9.3.0
    2.37-GCCcore-11.2.0
    2.36-GCCcore-10.2.0
    2.36-GCCcore-10.3.0
    2.33-GCCcore-8.2.0
    2.32-GCCcore-7.3.0



wget
---- 



**Description** 


GNU Wget is a free software package for retrieving files using HTTP, HTTPS and FTP, the most widely-used Internet protocols. It is a non-interactive commandline tool, so it may easily be called from scripts, cron jobs, terminals without X-Windows support, etc. 


**More Information** 


https://www.gnu.org/software/wget 


**Available Versions**:: 


    1.20.3-GCCcore-10.2.0



wtdbg2
------ 



**Description** 


Wtdbg2 is a de novo sequence assembler for long noisy reads produced by PacBio or Oxford Nanopore Technologies (ONT). It assembles raw reads without error correction and then builds the consensus from intermediate assembly output. 


**More Information** 


https://github.com/ruanjue/wtdbg2 


**Available Versions**:: 


    2.5-GCCcore-9.3.0



x264
---- 



**Description** 


x264 is a free software library and application for encoding video streams into the H.264/MPEG-4 AVC compression format, and is released under the terms of the GNU GPL. 


**More Information** 


https://www.videolan.org/developers/x264.html 


**Available Versions**:: 


    20190925-GCCcore-8.3.0
    20210414-GCCcore-10.3.0
    20201026-GCCcore-10.2.0
    20191217-GCCcore-9.3.0
    20190413-GCCcore-8.2.0



x265
---- 



**Description** 


x265 is a free software library and application for encoding video streams into the H.265 AVC compression format, and is released under the terms of the GNU GPL. 


**More Information** 


https://x265.org/ 


**Available Versions**:: 


    3.3-GCCcore-10.2.0
    3.0-GCCcore-8.2.0
    3.2-GCCcore-8.3.0
    3.3-GCCcore-9.3.0
    3.5-GCCcore-10.3.0



xorg-macros
----------- 



**Description** 


X.org macros utilities. 


**More Information** 


https://cgit.freedesktop.org/xorg/util/macros 


**Available Versions**:: 


    1.19.2-GCCcore-7.3.0
    1.19.2-GCCcore-10.2.0
    1.19.3-GCCcore-11.3.0
    1.19.3-GCCcore-11.2.0
    1.19.3-GCCcore-10.3.0
    1.19.2-GCCcore-8.3.0
    1.19.2-GCCcore-8.2.0
    1.19.2-GCCcore-9.3.0



xprop
----- 



**Description** 


The xprop utility is for displaying window and font properties in an X server. One window or font is selected using the command line arguments or possibly in the case of a window, by clicking on the desired window. A list of properties is then given, possibly with formatting information. 


**More Information** 


http://www.x.org/wiki/ 


**Available Versions**:: 


    1.2.3-GCCcore-7.3.0
    1.2.4-GCCcore-8.3.0
    1.2.5-GCCcore-10.2.0



xtb
--- 



**Description** 


xtb - An extended tight-binding semi-empirical program package.  


**More Information** 


https://xtb-docs.readthedocs.io 


**Available Versions**:: 


    6.3.3-GCCcore-9.3.0
    6.4.1-intel-2021a



yaff
---- 



**Description** 


Yaff stands for 'Yet another force field'. It is a pythonic force-field code. 


**More Information** 


https://molmod.github.io/yaff/ 


**Available Versions**:: 


    1.6.0-intel-2020a-Python-3.8.2
    1.6.0-foss-2020a-Python-3.8.2



yaml-cpp
-------- 



**Description** 


yaml-cpp is a YAML parser and emitter in C++ matching the YAML 1.2 spec 


**More Information** 


https://github.com/jbeder/yaml-cpp 


**Available Versions**:: 


    0.7.0-GCCcore-10.3.0



zlib
---- 



**Description** 


zlib is designed to be a free, general-purpose, legally unencumbered -- that is, not covered by any patents -- lossless data-compression library for use on virtually any computer hardware and operating system. 


**More Information** 


http://www.zlib.net/ 


**Available Versions**:: 


    1.2.11-GCCcore-8.1.0
    1.2.12-GCCcore-11.3.0
    1.2.11-intel-2020a
    1.2.11-GCCcore-8.2.0
    1.2.11-GCCcore-7.3.0
    1.2.11-GCCcore-9.3.0
    1.2.8-GCC-7.3.0-2.30
    1.2.11-GCCcore-11.2.0
    1.2.12
    1.2.11
    1.2.11-GCCcore-10.3.0
    1.2.11-GCCcore-8.3.0
    1.2.8
    1.2.11-GCCcore-10.2.0



zstd
---- 



**Description** 


Zstandard is a real-time compression algorithm, providing high compression ratios.  It offers a very wide range of compression/speed trade-off, while being backed by a very fast decoder. It also offers a special mode for small data, called dictionary compression, and can create dictionaries from any sample set. 


**More Information** 


https://facebook.github.io/zstd 


**Available Versions**:: 


    1.4.4-GCCcore-9.3.0
    1.4.5-GCCcore-10.2.0
    1.4.4-GCCcore-8.3.0
    1.5.0-GCCcore-11.2.0
    1.4.9-GCCcore-10.3.0



